---
layout: page
title: About
---

<h2>Introduction</h2>

<p>Nick works as an Independent Research Fellow in the <a href="http://www.birmingham.ac.uk/research/activity/microbiology-infection/index.aspx">Institute for Microbiology and Infection</a> at the <a href="http://www.birmingham.ac.uk">University of Birmingham</a>, sponsored by an MRC Special Training Fellowship in Biomedical Informatics. His research explores the use of cutting-edge genomics and metagenomics approaches to the diagnosis, treatment and surveillance of infectious disease. Nick has so far used high-throughput sequencing to investigate outbreaks of important Gram-negative multi-drug resistant pathogens such as <em>Pseudomonas aeruginosa</em>, <em>Acinetobacter baumannii</em> and Shiga-toxin producing <em>Escherichia coli</em>. His current work focuses on the genetic diversity of <em>Pseudomonas aeruginosa infection</em> in chronic lung diseases such as cystic fibrosis as a diagnostic and prognostic marker.  A more general aim is to develop bioinformatics tools to aid the interpretation of genome and metagenome-scale data in routine clinical practice in collaboration with mentor <a href="http://userweb.eng.gla.ac.uk/christopher.quince/">Chris Quince</a> at the University of Glasgow.</p>

<h2>MicrobesNG</h2>

MicrobesNG is a 5-year, 1.2m BBSRC-funded project to produce a world-leading microbial genome sequencing service and strain archiving resource, launching to the public in Q1 2014.

<h2>CLIMB</h2>

The Cloud Infrastructure for Microbial Genomics (CLIMB) is an 8.4m MRC-funded project to invest in bioinformatics and "big data" capacity building for the UK microbial genomics community. At heart it is four-site (Birmingham, Warwick, Cardiff and Swansea) connected virtual computing infrastructure. It is optimised for specific applications such as microbial genome alignment, de novo assembly and metagenomics. 

<h2>Group</h2>

<p>With Tim Mitchell, I co-supervise a PhD student, Josh Quick. Josh is investigating the role of microbial communities in wound healing in burns and trauma through the <a href="http://www.nihr.ac.uk/research/Pages/CentreforSurgicalReconstructionandMicrobiology.aspx">NIHR Centre for Surgical Reconstruction and Microbiology</a>, as well as the spread of nosocomial pathogens in our local patient popluation. Szymon Calus is helping develop protocols for culture-independent diagnosis of bacterial pathogens from low-input clinical samples.</p>
		
<h2>The E. coli O104:H4 outbreak in Germany</h2>

<p>I was involved with the crowd-sourced analysis of genomics data from the Shiga-toxin producing <em>E. coli</em> O104:H4 outbreak in Germany. This outbreakt caused more than 4,000 cases of disease and 50 deaths. I performed an <a href="http://pathogenomics.bham.ac.uk/blog/2011/06/ehec-genome-assembly/">initial assembly</a> of data from the BGI on the 2nd of June 2011 which helped spark a flurry of <a href="https://github.com/ehec-outbreak-crowdsourced/BGI-data-analysis/wiki">crowd-sourced analysis</a>, documented on the Github Wiki. Recently, the BBSRC made a video about this project which can be viewed below.</p>
<iframe width="560" height="315" src="http://www.youtube.com/embed/ttMnQIE-P-s" frameborder="0" allowfullscreen></iframe>

<h2>Bioinformatics projects</h2>

<p>In my previous role, I was responsible for the maintenance and development of the comparative bacterial genomics database, <a href="http://xbase.ac.uk">xBASE</a>. Although funding for this project has ceased, I still continue to develop the high-throughput sequencing pipeline <a href="http://ng.xbase.ac.uk">xBASE-NG</a> to support our local sequencing efforts, as well as the rapid bacterial genome annotation service <a href="http://xbase.ac.uk/annotation/">xBASE Annotation</a>.</p>

<p>In collaboration with Lex Nederbragt, we are developing an open-source repository of sequencing and bioinformatics benchmarking datasets entitled <a href="http://seqbench.org">Seqbench</a>.

<p>I aim to publish accompanying Github repositories with manuscripts as an aid to extensible and reproducible research, please see <a href="https://github.com/nickloman">my Github page for details</a>.</p>
		
<p>With James Hadfield, I built <a href="http://omicsmaps.com/">Omicsmaps.com</a>, a map of high-througput sequencing instruments.</p>

