---
layout: page
title: News
---

	<p><b>July, 2015.</b>. Our paper on nanopore assembly was published in Nature Methods. It got lots of attention from the press and was featured as a <a href="http://www.nature.com/nrmicro/journal/v13/n8/full/nrmicro3536.html">Nature Reviews Microbiology</a> highlight.</p>

	<p><b>April, 2015</b>. We have been collaborating with Jared Simpson in Ontario on a new method for nanopore-only de novo assembly and polishing. We found it was possible to assemble <em>E. coli</em> into a single contig using nanopore data alone. The method is described in a preprint which we have deposited in <a href="http://biorxiv.org/content/early/2015/03/11/015552">bioRxiv</a>. Our work on nanopore sequencing in outbreaks was recently featured in <a href="http://www.wired.co.uk/magazine/archive/2015/04/features/usb-gene-sequence/viewall">Wired UK</a>. Nick wrote a News and Reviews article for <a href="http://www.readcube.com/articles/10.1038%2Fnmeth.3327?shared_access_token=JZW2qpH40Q-3HIW5mMl9D9RgN0jAjWel9jnR3ZoTv0NOap72XcayK9iwtwkB44SHBJm8mPBxffWsR35eNkJhf_0wSNdqj6GsOhYNbIPJorSRPS-OAvGAfxN7-eCCaeqU8lVHOSGydOhohE2YCP67sFpJpRx_Mfi1x2NrIjRZE4ktdbAPrlFRfEvp9br5IA0q5gtTlR-7w7pl-vUTONTrCg%3D%3D">Nature Methods</a> in collaboration with Mick Watson about the recent exciting progress with nanopore sequencing. Nick did <a href="http://biome.biomedcentral.com/nanopore-based-genomic-sequencing-minion/">an interview with BioMedCentral's Biome magazine</a> about nanopore sequencing. A manuscript from Del Besra's team <a href="http://dx.doi.org/10.1371/journal.pone.0121492">was recently published</a> on drug discovery in tuberculosis. Our group performed whole-genome sequencing and mutation detection in order to confirm the predicted drug target. We are planning a <a href="/balti/">Balti and Bioinformatics</a> meeting on the 5th of May in Birmingham, and a <a href="http://www.climb.ac.uk/">CLIMB</a> Hackathon on Public Health Microbiology the following weekend. We launched our new BBSRC-funded microbial genome sequencing and strain archiving service, <a href="http://microbesng.uk/">MicrobesNG</a>, at the SGM meeting in Birmingham.</p>

	<p><b>January, 2015</b>. Happy New Year! Some recent papers have been featured as research highlights: see <a href="http://genomebiology.com/2014/15/12/564">Rob Knight's piece at Genome Biology on kit contamination</a>, <a href="http://www.nature.com/nmeth/journal/v12/n1/full/nmeth.3244.html">Nature Method's</a> article on poretools and our recent nanopore data release</a> and <a href="http://www.nature.com/nrmicro/journal/vaop/ncurrent/full/nrmicro3422.html">Susannah Salter writing in Nature Reviews Microbiology</a> on our recent <em>P. aeruginosa</em> in hospital water study. Our nanopore read on FigShare was the <a href="http://figshare.com/blog/The_figshare_top_10_of_2014_according_to_altmetric/142">8th most popular post in 2014</a>. Aaron Krol from Bio-IT World did a <a href="http://www.bio-itworld.com/2014/12/22/nanopore-sequencing-here-stay.html">cracking write-up</a> of the recent nanopore buzz.</p> 
	<p>So far this year I have been giving the <a href="http://omicsmaps.com">Omicsmaps</a> a lick of paint.</p>

	<p><b>November, 2014</b>. Our paper on the <em>kitome</em> is published today in <a href="http://www.biomedcentral.com/1741-7007/12/87">BMC Biology</a>. It has had great coverage from <a href="http://phenomena.nationalgeographic.com/2014/11/11/contaminomics-why-some-microbiome-studies-may-be-wrong/">Ed Yong in National Geographic</a>, <a href="http://www.nature.com/news/microbiome-science-threatened-by-contamination-1.16327">Nature News</a>, <a href="http://news.sciencemag.org/biology/2014/11/contamination-plagues-some-microbiome-studies?rss=1">Science</a>, <a href="http://www.scientificamerican.com/podcast/episode/microbiome-studies-contaminated-by-sequencing-supplies/">Scientific American's 60-second science podcast</a>, <a href="http://www.the-scientist.com/?articles.view/articleNo/41421/title/DNA-Extraction-Kits-Contaminated/">The Scientist</a> and <a href="http://blogs.biomedcentral.com/bmcblog/2014/11/12/keeping-it-clean-spotlight-on-contamination-in-microbiome-studies/">BioMed Central Blog</a>. It has even been covered on <a href="http://thescienceweb.wordpress.com/2014/11/12/real-science-found-to-be-contaminating-microbiome-studies/">The Science Web</a>, a true metric of impact!</p>
	<p>Our paper, tracking the spread of <em>Pseudomonas aeruginosa</em> from hospital water outlets to patients, using genomics and metagenomics sequencing is now out in <a href="http://bmjopen.bmj.com/content/4/11/e006278.full">BMJ Open</a>. It has been covered by the <a href="http://www.phgfoundation.org/blog/16407/">PHG Foundation think tank</a>.</p>

	<p><b>October, 2014</b>. The first manuscript describing a genome sequenced by nanopore technology, in this case E. coli K-12, is <a href="http://www.gigasciencejournal.com/content/3/1/22/abstract">published in GigaScience</a>. The poretools manuscript is <a href="http://bioinformatics.oxfordjournals.org/content/early/2014/08/19/bioinformatics.btu555.abstract">published in Bioinformatics</a>. The <a href="http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.3103.html">CONCOCT manuscript is publised in Nature Methods</a>. Work has begun on unifying my blog, lab notebook and this lab website in Github. There may be bugs and gotchas for a while as this is a slow process.</p>

        <p><b>July, 2014</b>. Two new preprints were deposited in bioRxiv. The first looks at the vexing problem of <a href="biorxiv.org/content/early/2014/07/16/007187">DNA contamination in commonly used extraction kits</a>. We find that when looking at low biomass samples this contamination has the potential to affect results of microbiome studies. The second details a <a href="http://biorxiv.org/content/early/2014/07/23/007401">toolkit for analysis of Oxford Nanopore MinION data</a>. The accompanying <a href="github.com/arq5x/poretools">Github repository is here</a>. Finally, a new paper in <a href="http://onlinelibrary.wiley.com/doi/10.1111/mmi.12704/abstract">Molecular Microbiology</a> looking at in-vitro evolution for acid resistance in <em>E. coli</em> in collaboration with Pete Lund in Biosciences.</P>

	<p><b>June, 2014</b>. We got up and running with the Oxford Nanopore MinION and were the first group to <a href="http://figshare.com/articles/A_P_aeruginosa_serotype_defining_single_read_from_our_first_Oxford_Nanopore_run/1052996">publish a read</a> from this instrument, see press coverage at <a href="http://www.bio-itworld.com/BioIT_Article.aspx?id=139007&terms=figshare">Bio-IT World</a> and <a href="http://www.genomeweb.com/sequencing/first-oxford-nanopore-users-comment-experience-minion-start-posting-data">GenomeWeb</a>. Lots of exciting samples to try out now!</a></p>

	<p><b>May, 2014</b>. We received our Oxford Nanopore <a href="https://twitter.com/pathogenomenick/statuses/454555293759057920">MinIONs</a>!. New <a href="http://genomea.asm.org/content/2/3/e00355-14.full.pdf">Genome Announcement of Elizabethkingia meningoseptica</a>. The <a href="http://www.climb.ac.uk">MRC CLIMB website</a> for cloud computing in microbial genomics is now live. We received notification that our <a href="http://www.bbsrc.ac.uk/funding/opportunities/2013/2013-bioinformatics-biological-resources-fund.aspx">grant proposal to the BBSRC for our MicrobesNG service</a> was funded. I gave an interview to Keith Bradnam's series on <a href="http://www.acgt.me/blog/2014/5/18/101-questions-with-a-bioinformatician-8-nick-loman">interviews with bioinformaticians</a>.</p>
		
	<p><b>February, 2014</b>. I was proud to be part of the awesome Evomics 2014 course held in Cesky Krumlov. There are some <a href="http://evomics.org/workshops/workshop-on-genomics/2014-workshop-on-genomics-cesky-krumlov/">great presentations and tutorials available</a> on their site.</p>

	<p>We have been accepted onto the <a href="https://www.nanoporetech.com/technology/the-minion-device-a-miniaturised-sensing-system/minion-access-programme">first wave</a> of the Oxford Nanopore MinION access programme!</p>

	<p>The Medical Research Council announced that <a href="http://www.mrc.ac.uk/Newspublications/News/MRC009720">a consortium for medical microbial bioinformatics</a> consisting of the Universities of Warwick, Birmingham, Swansea and Cardiff will be funded! This will provide an investment of nearly >Â£4m in server hardware and storage in Birmingham, as well as a five-year fellowship for me.</a>

	<p><b>January, 2013</b>. I had a lot of fun instructing and learning at C. Titus Brown's "Assembly Masterclass". If you are interested in assembly, there are lots of useful new tutorials and guides posted on the <a href="http://davis-assembly-masterclass-2013.readthedocs.org/en/latest/">course website</a>.</p>
		<p>A collaboration with Chris Quince (Glasgow) and Anders Andersson (SciLifeLab, Sweden) on a novel algorithm for genome binning we call CONCOCT has reached maturity, and a <a href="http://arxiv.org/abs/1312.4038">preprint is now available in the arXiv.</a></p>
		<p>A new paper in PLoS ONE from a collaboration with Waltham Petcare (part of Mars) was published on the <a href="http://www.ncbi.nlm.nih.gov/pubmed/24349448">oral microbiota of dogs.</a></p>

		<p><b>November, 2013</b>. <a href="http://biomickwatson.wordpress.com/">Mick Watson</a> and I have penned a commentary in Nature Biotechnology which may be of interest entitled: <a href="http://www.nature.com/nbt/journal/v31/n11/full/nbt.2740.html">So you want to be a computational biologist?</a>. I've also updated this page to add some details of upcoming <a href="#training">training courses</a>.

