---
layout: post
status: publish
published: true
title: xBASE 2 - Launched!
author:
  display_name: Nick Loman
  login: nick
  email: n.j.loman@bham.ac.uk
  url: http://xbase.bham.ac.uk
author_login: nick
author_email: n.j.loman@bham.ac.uk
author_url: http://xbase.bham.ac.uk
wordpress_id: 14
wordpress_url: http://robson.bham.ac.uk:8001/blog/?p=14
date: '2007-09-14 18:32:26 +0100'
date_gmt: '2007-09-14 17:32:26 +0100'
categories:
- Uncategorized
tags: []
comments: []
---
<p>We're very proud to announce the release of xBASE2. The site has numerous enhancements to the previous versions which we will describe in detail. The most important change to note is that:</p>
<ol>
<li>all of the previous xBASE sites - campyDB, colibase etc. are now part of the unified xBASE database, and so for example colibase.bham.ac.uk now redirects to <a href="http://xbase.bham.ac.uk/colibase">xbase.bham.ac.uk/colibase</a></li>
<li>we have imported all complete bacterial genomes available from NCBI, and a good proportion of the incomplete genomes</li>
<li>coliBASE, campyDB et al. are purely labels for particular points on the taxonomy, e.g. Enterobacteriacae for coliBASE.</li>
</ol>
<p>There are a large number of new features including:</p>
<ol>
<li>ability to stack multiple whole genome alignments (using promer or nucmer components of MUMmer)</li>
<li>a completely rewritten search engine</li>
<li>BLAST using the entire set of xBASE genes, proteins or genome sets</li>
<li>graphing function</li>
</ol>
<p>We will describe these in more detail in subsequent blog entries.</p>
<p>Please send us feedback and reports of any bugs you may encounter: this can be either via the comments section of this blog or direct to <a href="mailto:n.j.loman@bham.ac.uk">n.j.loman@bham.ac.uk</a></p>
