---
layout: post
status: publish
published: true
title: New Google Maps for Next-Gen Sequencers
author:
  display_name: Nick Loman
  login: nick
  email: n.j.loman@bham.ac.uk
  url: http://xbase.bham.ac.uk
author_login: nick
author_email: n.j.loman@bham.ac.uk
author_url: http://xbase.bham.ac.uk
wordpress_id: 191
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=191
date: '2009-11-20 17:27:58 +0000'
date_gmt: '2009-11-20 16:27:58 +0000'
categories:
- High-throughput sequencing
tags:
- solid
- '454'
- google map
- james hadfield
- next-generation sequencers
- solexa
- heliscope
- genomics
comments:
- id: 59495
  author: Bevan Weir
  author_email: weirb@landcareresearch.co.nz
  author_url: http://www.rhizobia.co.nz/
  date: '2010-06-02 23:55:53 +0100'
  date_gmt: '2010-06-02 22:55:53 +0100'
  content: Nice map. Do you want the locations of the 454 The GS Junior machines too?
    The labs that have these would not necessary be "High-throughput 'Next-Generation'
    Sequencing Facilities"
- id: 59496
  author: Nick Loman
  author_email: n.j.loman@bham.ac.uk
  author_url: http://xbase.bham.ac.uk
  date: '2010-06-03 08:59:00 +0100'
  date_gmt: '2010-06-03 07:59:00 +0100'
  content: "<a href=\"#comment-59495\" rel=\"nofollow\">@Bevan Weir </a> \r\nHmm.
    I support it might be best to concentrate on centres offering sequencing as a
    service (academic collaboration or commercial). I suspect those with GS Junior
    / SOLiD PI machines will be keeping them for exclusive use in their labs."
---
<p>A few months back I <a href="http://pathogenomics.bham.ac.uk/blog/2009/08/sequencing-in-the-u-k/">blogged about James Hadfield's excellent Google Map</a> for locating next-generation sequencing facilities. Some time has passed and the map has become very popular with over 100,000 views registered. Keen to create this into a really great community resource, James and I have been collaborating in our spare time on <a href="http://pathogenomics.bham.ac.uk/hts/">a revamped version</a>. There are a couple of major improvements over the first version;</p>
<ul>
<li>you can now filter down to locate and compare sequencing platforms</li>
<li>the placemarkers are now clustered to make the map faster and to make it easier to visualise hotspots of sequencing machines</li>
<li>it is now easy to add new genomics platforms (including microarrays) as they become available</li>
<li>you can easily add and edit facilities, which is subject to moderation to prevent spam</li>
</ul>
<p>Perhaps most excitingly for stats-nerds, this is all database-driven so the long awaited league tables of sequencing platforms can be calculated automatically. However, mindful that the information could be made more accurate, I have decided to leave the league tables off until people have had a bit of time to check the map for errors, and add any missing facilities.</p>
<p>We're really keen to gather feedback so either post a comment on the blog, or <a href="mailto:n.j.loman@bham.ac.uk">get in touch directly</a> if you have ideas you want to discuss. We were also thinking about extending it for other non-genomics utilities, for example next-generation proteomics. If you are working in such a field and would be interested in collaborating, again get in contact.</p>
<p>Well, what are you waiting for? <a href="http://pathogenomics.bham.ac.uk/hts/">Go check out the map right now</a>!</p>
