---
layout: post
status: publish
published: true
title: 'Evidence for an early prokaryotic endosymbiosis: I don''t believe it!'
author:
  display_name: Mark Pallen
  login: mpallen
  email: m.pallen@bham.ac.uk
  url: http://roughguidetoevolution.blogspot.com/
author_login: mpallen
author_email: m.pallen@bham.ac.uk
author_url: http://roughguidetoevolution.blogspot.com/
wordpress_id: 121
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=121
date: '2009-08-23 10:52:49 +0100'
date_gmt: '2009-08-23 09:52:49 +0100'
categories:
- Uncategorized
tags:
- James Lake
- Endosymbiosis
- Nature
- Sequence Analysis
- Phylogeny
- Bacterial Evolution
- Victor Meldrew
comments:
- id: 59219
  author: herm
  author_email: herman@graduate.hku.hk
  author_url: ''
  date: '2009-08-24 06:14:38 +0100'
  date_gmt: '2009-08-24 05:14:38 +0100'
  content: "Hi, I quite agree with your views and analysis on this paper. The split-up
    of the Firmicutes is especially jarring. It should be noted that the paper is
    categorized as 'Hypothesis', which to my impression appears to be less stringently
    reviewed (in terms of scientific evidence/ quality). I wouldn't have objected
    to this paper being published in a journal dedicated to discussion of important
    hypotheses, but many Nature readers may not appreciate the qualitative difference
    between this 'Hypothesis' article and the regular articles/ letters. I have no
    doubt that we'll be seeing many papers citing this as 'proof' of the early prokaryotic
    endosymbiosis idea.\r\n\r\nP.S. BTW, what's happening at the Nature editorial
    office? The author JA Lake has 4 affiliations listed, however they are actually
    fragments of some longer affiliation(s) at UCLA."
- id: 59226
  author: kswithers
  author_email: kristen.swithers@uconn.edu
  author_url: ''
  date: '2009-09-05 17:53:41 +0100'
  date_gmt: '2009-09-05 16:53:41 +0100'
  content: Also on his taxonomic categories. There are actinobacteria in his clostridia
    group! So of course his rings are going to connect the Actinobacteria and Clostridia
- id: 59227
  author: ppgardne
  author_email: pg5@sanger.ac.uk
  author_url: ''
  date: '2009-09-05 19:03:40 +0100'
  date_gmt: '2009-09-05 18:03:40 +0100'
  content: For an article based almost entirely upon Pfam data there is a remarkable
    lack of references to this resource!
- id: 59641
  author: Reassessing the Lineage Fusion Hypothesis for the Origin of Double Membrane
    Bacteria
  author_email: ''
  author_url: http://pathogenomics.bham.ac.uk/blog/2011/03/reassessing-the-lineage-fusion-hypothesis-for-the-origin-of-double-membrane-bacteria/
  date: '2011-03-31 15:27:21 +0100'
  date_gmt: '2011-03-31 15:27:21 +0100'
  content: '[...] http://pathogenomics.bham.ac.uk/blog/2009/08/evidence-for-an-early-prokaryotic-endosymbiosis-i-dont-...
    [...]'
---
<p>I am thankful to Nick  for <a href="http://pathogenomics.bham.ac.uk/blog/2009/08/how-to-publish-a-comment-in-1-2-3-easy-steps/">the previous post</a>, which eerily mirrors our own experience a year or two ago when trying to get a comment published on a <a href="http://www.pnas.org/content/104/17/7116">problematic paper on flagellar evolution</a>. Our experience—and his post—explains Nick's negative response the other day, when I suggested we marshall evidence for a "comment" on a paper by James Lake published this week in <em>Nature</em>!! So, let us be thankful for the power of blogging to get comments out (<a href="http://pandasthumb.org/archives/2007/06/correction-to-l.html">see Matzke on the flagellar paper</a>) without having to negotiate a barrage of editorial and other hurdles. Here I outline a few problems I have with Lake's paper, in the hope that others flesh them out by adding comments here and even take up the challenge of submitting a formal response to the paper to <em>Nature </em>(I don't have time or energy for this).</p>
<p>The paper in question is <em><a href="http://www.nature.com/nature/journal/v460/n7258/full/nature08183.html">Evidence for an early prokaryotic endosymbiosis</a>.</em> In this paper ,Lake claims to have evidence for an ancient endosymbiotic origin of Gram-negative bacteria from fusion of two Gram-positive lineages, the Clostridia and the Actinobacteria.</p>
<p>Lake starts his argument by lining up a list of what he sees as five "natural taxa": the Archaea (R), the Actinobacteria (A), the Bacilli and relatives (B), the Clostridia and relatives (C), and the double-membrane prokaryotes (D). To me, this choice of high-level taxa seems anything but "natural", because to the jobbing microbiologist, Clostridia and Bacilli belong together as spore-forming Gram-positives and according to traditional taxonomies belong together in the phylum <a href="http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=1239">Firmicutes</a>. In a quick and dirty look, I cannot find any published work evaluating in detail whether the sporulation apparatus of these two groups is likely to date from from their common ancestor or whether one group acquired it by horizontal gene transfer, but my intuition would be for monophyly of both taxa and apparatus, with Firmicutes as a "natural taxon".</p>
<p>In one confusing paragraph, Lake makes much of the similarities between the inner membrane of Gram-negatives and what he calls "the outer membrane" of Gram-positives. In fact, given that Gram-positives have only one membrane, calling it an "outer membrane" is misleading. It is more usually called the "cell membrane" or "plasma membrane". And no one doubts that this Gram-positive cell membrane is homologous to the inner membrane from Gram-negatives—it is the outer membrane of Gram-negatives that needs explaining!</p>
<p>Although he seems rather obscure on this point, it seems that Lake is claiming that the outer membrane originates from the cell membrane of one or other of his two symbiotic partners, even though there is no evidence that the Gram-negative outer membrane (or its proteins) resembles the cell membranes from either of these  Gram-positive taxa (where is the LPS in the cell membranes of Actinobacteria or Clostridia?).</p>
<p>But it is the evidence from analysis of sequences that Lake puts forward for his hypothesis that troubles me most. Rather than look at individual protein sequences in individual genomes, Lake chooses to look simply at the presence or absence of protein domains within his chosen taxa, gleaning his data from the Pfam database. He uses seductively simple binary patterns of presence (+) and absence (-) as the basis for his phylogenetic analyses.</p>
<p>But even a cursory glance at the <a href="http://www.nature.com/nature/journal/v460/n7258/extref/nature08183-s1.pdf">Supplementary Data</a> reveals two problems:</p>
<ol>
<li><strong>Double counting of some domains. </strong>Treating presence or absence of domains as independent character states might be acceptable if there were no stable links between domains, but when two domains commonly occur together in the same protein or pathway or macromolecular complex, it strikes me that this should be treated as one piece of data not two, e.g. <em>MAAL_C </em><a href="http://pfam.janelia.org//family/PF07476"><em>PF07476</em></a><em> Methylaspartate ammonia-lyase C-terminus</em> and <em>MAAL_N </em><a href="http://pfam.janelia.org//family/PF05034"><em>PF05034</em></a><em> Methylaspartate ammonia-lyase N-terminus </em>which are listed in the first table in the Supplementary Data among "the 99 Pfams selected for (A,B,C,D,R) equal (-,-,+,+,+)."  A quick glance suggests that there are many such examples in the tables, but whether this alone is enough to undermine the conclusions is unclear to me.</li>
<li><strong>Asymmetry in what counts as presence in one taxon versus what counts as presence in another</strong>—not all + signs are equal! Let's take a quick look at the first entries in each of Lake's tables.<br />
<strong> Table S2A. The 99 Pfams selected for (A,B,C,D,R) equal (-,-,+,+,+). </strong><em>A2M_N <a href="http://pfam.janelia.org//family/PF01835">PF01835</a> Alpha-2-macroglobulin family N-terminal region</em>. Click on the link to access the PFAM entry and then click on the "species" link in the menu on the left hand side to see the phylogenetic distribution of the domain:  hundreds species of double-membraned bacteria have the domain, but only two species of Archaea and two species from Clostridia (in Lake's <em>sensu latu</em>).<br />
<strong> Table S2B. The15 Pfams selected for (A,B,C,D,R) equals (-,+,-,+,+)</strong>. <em>Dna2 </em><a href="http://pfam.janelia.org//family/PF08696"><em> PF08696 </em></a><em>DNA replication factor Dna2.</em> Only one example of this domain in the Bacteria (in <em>Pseudoalteromonas haloplanktis</em> strain TAC 125) ! And four in a halophilic group of Archaea,but none among the Bacilli so cannot work out how Lake arrives at his -,+,-,+,+ designation!<br />
<strong> Table S2C. The 8 Pfams selected for (A,B,C,D,R) equals (-,+,+,-,+)</strong>. <em>DUF1002 </em><a href="http://pfam.janelia.org//family/PF06207"><em>PF06207</em></a><em> Protein of unknown function (DUF1002).</em> A couple of dozen examples each in Bacillus and Clostridia, but just four in the Archaea.<br />
<strong> Table S2D. The 174 Pfams selected for (A,B,C,D,R) equals (-,+,+,+,-).</strong> <em>3D </em><a href="http://pfam.janelia.org//family/PF06725"><em> PF06725</em></a><em> 3D domain</em>. Does indeed have dozens of examples in each of the three phyla.<br />
<strong> Table S2E. The 62 Pfams selected for (A,B,C,D,R) equals (+,-,-,+,+)</strong>. <em>Adenosine_kin </em><a href="http://pfam.janelia.org//family/PF04008"><em> PF04008</em></a><em> Adenosine specific kinase</em>. Dozens of examples in D and R, but just a handful in A, largely confined to <em>Mycobacterium tuberculosis </em>complex.<br />
<strong> Table S2G. The 73 Pfams selected for (A,B,C,D,R) equals (+,-,+,+,-). </strong><em>ABC_membrane_2 </em><a href="http://pfam.janelia.org//family/PF06472"><em>PF06472</em></a><em> ABC transporter transmembrane region 2</em> Hundreds in D, dozens in A, but only one in C!</li>
</ol>
<p>OK, so, I haven't got to the end of the tables, but already it is clear that these tables contain mistakes and gross asymmetries in domain distributions.</p>
<p>Why do these asymmetries matter? Well, Lake uses his patterns of + and - designations to build his hypothesis. But inherent in Lake's + (present) and - (absent) designations for domains in taxa is the assumption that these patterns represent the ancestral state for the taxa, i.e. more or less reflect the domain distributions in the last common ancestor of each taxon.</p>
<p>Yet descent from that common ancestor is not the only explanation for the presence of a domain in a taxon—horizontal gene transfer might also account for the presence of the domain.</p>
<p>Now, I guess a working assumption of vertical descent might be justified if each taxon that had a domain, contained large numbers of that domain scattered across many species. BUT this is not true with the data Lake is presenting here--in some cases, just one example of a domain occurs in one protein in just one one species in a whole "natural taxon". In such circumstances, one's working assumption has to be that this domain has been acquired by that one species through horizontal gene transfer, rather than acquired from the whole taxon's last common ancestor and then lost by all species in it bar one!</p>
<p>And, as with many broad-reaching sequence-based analyses, there is a house-of-cards structure to the argument here, so if these tables in the supplementary data cannot be trusted, nor can the final conclusions, which now have to be regarded as "unsafe".</p>
<p>Anyhow, the above counts as just a quick dirty look at this paper and I may have missed some important points or got my wires crossed somewhere along the line. But I hope that others will now take an even closer look at it.</p>
<p>But for me, what I have seen already evokes a well-known response from my grumpy alter ego, <a href="http://en.wikipedia.org/wiki/Victor_Meldrew">Victor Meldrew</a>: "I don't believe it!", even though it has supposedly passed peer review in <em>Nature</em>!</p>
