---
layout: post
status: publish
published: true
title: The Mysterious Case of the Russian Emperor
author:
  display_name: Nick Loman
  login: nick
  email: n.j.loman@bham.ac.uk
  url: http://xbase.bham.ac.uk
author_login: nick
author_email: n.j.loman@bham.ac.uk
author_url: http://xbase.bham.ac.uk
wordpress_id: 227
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=227
date: '2009-12-11 01:24:27 +0000'
date_gmt: '2009-12-11 00:24:27 +0000'
categories:
- Uncategorized
tags: []
comments:
- id: 59269
  author: krobison
  author_email: keith.e.robison@gmail.com
  author_url: http://omicsomics.blogspot.com
  date: '2009-12-11 21:42:20 +0000'
  date_gmt: '2009-12-11 20:42:20 +0000'
  content: "In my opinion the mate-pair vs. paired end distinction is useful.  Mate
    pair information is captured during library construction; paired end information
    off the sequencer.  They will probably have different quality issues.  Some sample
    types may not be amenable to mate pairs but still can read paired ends.\r\n\r\nOf
    course, get ready for more.  SOLiD is reputedly going to have paired ends sometime
    in the not too distant future.  Both PacBio &amp; Helicos have \"dark fill\" strategies
    for getting such information, perhaps in multiple islands.  Complete Genomics
    has something closer to a mate quartet (and perhaps higher) scheme."
---
<p>At yesterday's IDRN meeting, Neil Ward gave an excellent summary of the Illumina sequencing platform, which really does seem to be going from strength to strength. They are pushing throughput hard - 35Gb per run routinely, with 50Gb targeted in the next few months. They think 95Gb per run should be achievable with the current technology, phenomenal stuff. 100 base-pair reads and the new 3kb mate-pairs (annoying distinction from paired-ends - stop confusing me!) means <em>de novo</em> applications will really become much easier on this platform.<img class="alignright size-medium wp-image-228" title="publications_overview_hero" src="http://pathogenomics.bham.ac.uk/blog/wp-content/uploads/publications_overview_hero-300x120.jpg" alt="publications_overview_hero" width="300" height="120" /></p>
<p>But a strange thing happened during the presentation, Neil mentioned an interesting sounding Illumina case-study, the confirmation of the identity of Russian Emperor <a href="http://en.wikipedia.org/wiki/Nicholas_II_of_Russia">Czar Nicholas II</a> and his family by sequencing STRs and mtDNA from DNA recovered from grave sites. This case-study is featured prominently on the <a href="http://www.illumina.com/publications/overview.ilmn">Illumina publications page</a>. So imagine my surprise when I clicked through to<a href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B8JHP-4XC41T8-K&amp;_user=10&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;_docanchor=&amp;view=c&amp;_acct=C000050221&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=f3d32278d10c0ec59c8cf2afbc40734e"> find the article in <em>Forensic Science International </em>had been withdrawn</a>!</p>
<p>What happened I wonder? Some of the printable suggestions from conference delegates included 1) the authors sequenced their own DNA by accident, 2) a problem with informed consent for the samples - perhaps Nicholas' living relatives were not happy or even 3) they dug up the wrong grave!</p>
<p>Of course it is probably unfair to speculate in such a case, and I don't want to annoy anyone by posting about it, but its such a tantalising story its difficult not to! It is certainly unusual to see aÂ  paper withdrawn (rather than getting addendum after addendum of corrections). The truth of course is likely to be more prosaic than my suggestions. But I would expect it to disappear off Illumina's website fairly soon.</p>
<p>A <a href="http://www.pnas.org/content/106/13/5258.full">previous genomic study</a> was published in PNAS in October 2008.</p>
