---
layout: post
status: publish
published: true
title: High-throughput sequencing in pathogen discovery and epidemiology, plus a job
  opportunity
author:
  display_name: Mark Pallen
  login: mpallen
  email: m.pallen@bham.ac.uk
  url: http://roughguidetoevolution.blogspot.com/
author_login: mpallen
author_email: m.pallen@bham.ac.uk
author_url: http://roughguidetoevolution.blogspot.com/
wordpress_id: 315
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=315
date: '2010-06-10 16:50:51 +0100'
date_gmt: '2010-06-10 15:50:51 +0100'
categories:
- Uncategorized
tags: []
comments: []
---
<p>[psst: want an exciting job? read on!]</p>
<p>During a decade of progress, bacterial genomics has already primed the development of new vaccines, novel anti-bacterial agents and innovations in identifying, culturing and tracking the spread and evolution of bacterial pathogens. Yet, despite its success as a research discovery platform, genome sequencing has, until now, proven too cumbersome and expensive to be used as a routine diagnostic or epidemiological tool in clinical bacteriology.  But now, with the arrival of <strong>“<span style="font-weight: normal;">next-generation” high-throughput sequencing (HTS) technologies</span></strong> it may be possible to deliver sequence data quickly and easily enough (i.e. within days or weeks) for it to have a real-time impact in the clinical arena. </p>
<p>Several pioneering efforts have established the potential of high-throughput sequencing in pathogen discovery, albeit primarily in virology. Unbiased 454 sequencing of cDNA from three Australian transplant recipients with fatal infections yielded &gt;100,000 sequences, of which 14 resembled sequences from lymphocytic choriomeningitis virus (<a href="http://content.nejm.org/cgi/content/abstract/358/10/991">Palacios et al., 2008, N Engl J Med, 358, 991-8</a>). Specific PCR and serological assays were then used to confirm the infection. Similar approaches made it possible to identify Lujo virus, a novel arenavirus responsible for a nosocomial outbreak of haemorrhagic fever in South Africa, within 72 hours of sample receipt (B<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2680969/pdf/ppat.1000455.pdf">riese et al., 2009, PLoS Pathog, 5, e1000455</a>) and to obtain 70% of the genome of a novel species of Ebola virus (Bundibugyo ebolavirus) from Uganda in less than 10 days (<a href="http://www.plospathogens.org/article/info%3Adoi%2F10.1371%2Fjournal.ppat.1000212">Towner et al., 2008, PLoS Pathog, 4, e1000212</a>).</p>
<p>A metagenomics survey of faeces has shown the feasibility of detecting bacterial pathogens such as <em>Campylobacter</em> through unbiased high-throughput sequencing of microbial DNA (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2630750/?tool=pubmed">Nakamura et al., 2008, Emerg Infect Dis, 14, 1784-6</a>), while  several publications have shown that high-throughput sequencing can be applied to the analysis of amplified molecular bar-codes, such as 16S rDNA sequences (a topic to be covered in a later posting). However, at present, the most exciting application of high-throughout sequencing in clinical bacteriology is in unravelling the fine-grained epidemiology of pathogen spread. In this context, genome sequencing represents the ultimate epidemiological typing method—a universally applicable, digital, "library" typing method, portable internationally and across time; capable of distinguishing strains that differ by as little as a millionth of a genome.  In detecting differences between strains that appear indistinguishable by current methods, high-throughput sequencing can certainly augment existing typing approaches and might give us the chance to unrvael chains of transmission of pathogens. A more provocative thought is that high-throughput whole-genome sequencing might soon consign all existing typing methods to the dustbin of history!</p>
<p>In a recent study published in Science, Harris et al (<a href="http://www.sciencemag.org/cgi/content/abstract/327/5964/469">Harris et al., 2010, Science, 327, 469-74</a>) used Illumina sequencing of 63 isolates, twinned with alignment against a reference, to obtain a high-resolution view of the genomic epidemiology of an epidemic strain of methicillin-resistant <em>Staphylococcus aureus</em> (MRSA). The resulting SNP-based phylogeny revealed a geographic structuring within the lineage spanning five continents and person-to-person transmission within the hospital environment. Beres et al, in a <a href="http://www.pnas.org/content/107/9/4371.full.pdf">PNAS paper</a>, used a similar approach to unravel the genomic epidemiology of invasive <em>Streptococcus pyogenes</em> infections in Ontario, Canada. They sequenced the genomes of 95 strains from three epidemics, which coupled with analysis of 280 SNPs in 344 strains, revealed a complex population structure and dynamic mixture of clonal complexes. They were also able to identify genes with increased rates of nonsynonymous SNPS, which were probably the result of adaptive changes. </p>
<p>Recently, we have also turned our attention to the potential of high-throughput genome sequencing in the epidemiology of bacterial pathogens. <em><a href="http://cmr.asm.org/cgi/content/full/21/3/538">Acinetobacter baumannii</a></em> is an important cause of healthcare-associated infection, particularly ventilator-associated pneumonia and bloodstream infections in critically ill patients. Two features of the organism stand out as particularly alarming: firstly, its <strong><span style="font-weight: normal;">propensity to cause outbreaks</span></strong> in the hospital setting, particularly in intensive care units; and secondly, its ability to <strong><span style="font-weight: normal;">acquire resistance to multiple antibiotics</span></strong>. Indeed, this bacterium is frontrunner in the progression towards what has been called the “post-antibiotic apocalypse”, where pathogens become resistant to all available antibiotics. <em>A. baumannii</em> infections and outbreaks have proven a problem in hospitals across the developed world, with three “international clones” (European Clones I, II and III) spreading widely. <em>Acinetobacter</em> infection has also emerged as a <a href="http://www.ncbi.nlm.nih.gov/pubmed/16825400">threat to British and American casualties of the Iraq and Afghanistan wars</a>, prompting high-level coverage in the news media and even on a patient advocacy site (<a href="http://www.acinetobacter.org">www.acinetobacter.org</a>). Our own experience in Birmingham, where <a href="http://www.uhb.nhs.uk/rcdm.htm">UK military casualties are treated</a>, confirms that the problem extends beyond the theatre of war, with strains introduced by returning casualties <a href="http://www.ncbi.nlm.nih.gov/pubmed/16728314">cross-infecting</a> other military or civilian patients. </p>
<p>In a modestly priced survey (~£5000 in sequencing costs, kindly funded by the <a href="http://www.his.org.uk/">Hospital Infection Society</a>), we applied 454 whole-genome sequencing to six isolates of <em>Acientobacter baumannii</em> that were indistinguishable using standard typing techniques. The identification of SNPs in three loci facilitated discrimination between alternative epidemiological hypotheses and provided evidence of transmission between a military and civilian patient.</p>
<p>This small scale study, which was <a href="http://www.journalofhospitalinfection.com/article/S0195-6701(10)00028-9/abstract">published recently in the <em>Journal of Hospital Infection</em></a>, primed a successful proposal to the UK Medical Research Council to use <strong><span style="font-weight: normal;">high-throughput bacterial</span><span style="font-weight: normal;"> genome sequencing</span></strong> to advance our understanding of the biology, epidemiology and evolution of <strong><span style="font-weight: normal;">multi-drug resistant </span><em><span style="font-weight: normal;">Acinetobacter baumannii</span></em></strong> and to explore the potential of genome sequencing as a <strong><span style="font-weight: normal;">tool in clinical microbiology and infection control. To do this, we will be working with partners from<strong> <span style="font-weight: normal;">the National Health Service </span></strong>and<strong><span style="font-weight: normal;"> the Health Protection Agency</span><span style="font-weight: normal; ">. </span></strong></span></strong></p>
<p><strong><span style="font-weight: normal;"><strong><span style="font-weight: normal; ">We have ambitious plans—we aim to genome-sequence at least 180 <em>Acinetobacter</em> isolates over the next three years and perform high-throughput phenotypic assays, pioneering an approach that could be applied to a wide range of human pathogens...</span></strong></span></strong></p>
<p><strong><span style="font-weight: normal;"><strong><span style="font-weight: normal; ">And so we are looking to recruit an ambitious, energetic and successful Research Fellow in Bacterial Pathogenomics to bring biological, epidemiological and/or clinical expertise to our high-throughput genotypic and phenotypic analyses of <em>Acinetobacter</em> isolates. If you think you fit the bill, take a look at the <a href="http://www.download.bham.ac.uk/vacancies/jd/47223.pdf">Job Description</a> and <a href="https://atsv7.wcn.co.uk/search_engine/jobs.cgi?SID=amNvZGU9MTEzODQ1NCZ2dF90ZW1wbGF0ZT03Njcmb3duZXI9NTAzMjUyMSZvd25lcnR5cGU9ZmFpciZicmFuZF9pZD0wJnZhY2Zpcm0udmFjdGl0bGU9NDcyMjMmcG9zdGluZ19jb2RlPTExNw==">follow this link</a> for instructions on how to apply. Closing Date: 9 Jul 2010. Interview Date:  22 July 2010. For informal enquiries please e-mail me on <a href="mailto:m.pallen@bham.ac.uk">m.pallen@bham.ac.uk</a>. Feel free to forward this blog post to anyone who might be interested.</span></strong></span></strong></p>
