---
layout: post
status: publish
published: true
title: A triumph for Old World genomics
author:
  display_name: Mark Pallen
  login: mpallen
  email: m.pallen@bham.ac.uk
  url: http://roughguidetoevolution.blogspot.com/
author_login: mpallen
author_email: m.pallen@bham.ac.uk
author_url: http://roughguidetoevolution.blogspot.com/
wordpress_id: 255
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=255
date: '2010-03-08 18:26:53 +0000'
date_gmt: '2010-03-08 17:26:53 +0000'
categories:
- Uncategorized
tags: []
comments: []
---
<p>This week's <em>Nature</em> sees publication of <a href="http://www.nature.com/nature/journal/v464/n7285/full/nature08821.html">a metagenomics survey</a> of the human gut microbiome that is astonishing in scale. Nearly 600 gigabase pairs of raw data from stool samples from 33 Europeans processed to give over three million protein-coding genes! This is around 200 times larger than the most recent comparable study!</p>
<p>Also remarkable is the provenance of the study. If one had been taking bets on who would be the first to make the front cover of <em>Nature</em> with a study like this, the US-led <a href="http://nihroadmap.nih.gov/hmp/">Human Microbiome Project</a> would have been most people’s clear favourites! Instead, this study is the fruit of a collaboration between two ends of the Old World, the European <a href="http://www.metahit.eu/">MetaHIT</a> consortium and the world's largest genomics institute, China's <a href="http://www.genomics.cn/">BGI</a>. There is life in the Old World yet!</p>
<p>But this is far from the last word on the human gut microbiome. 600 gigabase pairs sounds like a lot, but this could be completed in just three runs of the <a href="http://pathogenomics.bham.ac.uk/blog/2010/02/landed-first-illumina-hiseq-machines-advertised/">new Illumina Hi-Seq instrument</a>! It makes you wonder whether future studies will need quite so many authors!</p>
<p>Also questionable is the use of a short-read sequencing approach for this problem. The scale of the assembly achieved in this study is stupendous. But, even so, instead of complete genomes or even gene clusters, we just end up with lots of genes, often unlinked to each other or to any organism. And the lack of any assembled 16S rRNA genes makes it impossible to compare this study with the large body of existing literature on phylogenetic profiling of the gut. Only when we have complete assembled genomes for all the inhabitants of the bowel can we consider the gut microbiome properly sequenced. In fact, it is interesting to speculate how long it will be before today's obsession with short-read technologies, and the analytical problems they bring, will be swept away by the tide of progress towards technologies that combine high throughput with long read lengths.</p>
<p>Still, I am being a bit churlish here. Three million protein sequences cover a huge swathe of protein space and represent a huge resource for future studies. I, for one, look forward to discovering how many flagellar genes or type III secretion genes are hiding in the human gut microbiome! And if a <a href="http://www.nature.com/news/2010/100303/full/464022a.html">news piece</a> in this week's <em>Nature</em> is anything to go by, we ain't seen nothing yet from the BGI, which now has 128 of the Illumina HiSeq instruments on order—putting the Old World on track to surpass the New World's sequencing capacity by a considerable margin! 中国人从此站立起来了!</p>
