---
layout: post
status: publish
published: true
title: EHEC Genome Assembly
author:
  display_name: Nick Loman
  login: nick
  email: n.j.loman@bham.ac.uk
  url: http://xbase.bham.ac.uk
author_login: nick
author_email: n.j.loman@bham.ac.uk
author_url: http://xbase.bham.ac.uk
wordpress_id: 662
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=662
date: '2011-06-02 15:31:26 +0100'
date_gmt: '2011-06-02 15:31:26 +0100'
categories:
- Uncategorized
tags: []
comments:
- id: 59674
  author: BioInfo
  author_email: jhjohnsn@gmail.com
  author_url: ''
  date: '2011-06-02 17:38:06 +0100'
  date_gmt: '2011-06-02 17:38:06 +0100'
  content: "Hi Nick,\r\n\r\nI just completed a quick first past analysis of the data
    if you are interested: http://www.edgebio.com/blog/?p=209 - Analysis data is found
    here for download: http://www.edgebio.com/data/ion/ecoli_bgi/"
- id: 59675
  author: 'News: Nick Loman draft assembly of the E. Coli sequence from BGI | NGS
    bioinformatics'
  author_email: ''
  author_url: http://ngsbioinformatics.com/2011/06/news-nick-loman-draft-assembly-of-the-e-coli-sequence-from-bgi/
  date: '2011-06-03 00:37:56 +0100'
  date_gmt: '2011-06-03 00:37:56 +0100'
  content: '[...] can find in the blog of Nick Loman a draft assembly of the sequences
    from BGI. He has obtained it with [...]'
- id: 59676
  author: 'News: Era7 Bioinformatics annotates E. Coli EHEC O104 starin with BG7 Pipeline
    | NGS bioinformatics'
  author_email: ''
  author_url: http://ngsbioinformatics.com/2011/06/news-era7-bioinformatics-annotates-e-coli-ehec-o104-starin-with-bg7-pipeline/
  date: '2011-06-03 04:19:51 +0100'
  date_gmt: '2011-06-03 04:19:51 +0100'
  content: '[...] They have worked with the assembly from Nick Loman: http://pathogenomics.bham.ac.uk/blog/2011/06/ehec-genome-assembly/
    [...]'
- id: 59679
  author: EHEC genomes &laquo; bacpathgenomics
  author_email: ''
  author_url: http://bacpathgenomics.wordpress.com/2011/06/04/ehec-genomes/
  date: '2011-06-04 07:51:54 +0100'
  date_gmt: '2011-06-04 07:51:54 +0100'
  content: '[...] http://pathogenomics.bham.ac.uk/blog/2011/06/ehec-genome-assembly/
    [...]'
- id: 59680
  author: kat
  author_email: kholt@unimelb.edu.au
  author_url: http://bacpathgenomics.wordpress.com
  date: '2011-06-04 07:57:00 +0100'
  date_gmt: '2011-06-04 07:57:00 +0100'
  content: "Hi Nick,\r\n\r\nI've done some really preliminary analysis to look at
    SNPs &amp; phylogeny, posted here:\r\n\r\nhttp://bacpathgenomics.wordpress.com/2011/06/04/ehec-genomes/\r\n\r\nKat
    Holt"
- id: 59681
  author: My contribution to the &#8216;HUSEC41-strains-are-not-that-new&#8217; debate
    &laquo; The Alignment Gap
  author_email: ''
  author_url: http://phylogeo.wordpress.com/2011/06/04/7-2/
  date: '2011-06-04 09:38:13 +0100'
  date_gmt: '2011-06-04 09:38:13 +0100'
  content: '[...] ECOR concatenated MLST sequences (using this scheme) and extracted/concatenated
    MLST sequences from Nick Loman&#8217;s assembly of strain TY2482 genome (the groups
    I circled are not very accurate, I did it [...]'
- id: 59682
  author: 'Batterio killer, continuano le analisi: potrebbe essere resistente a 8
    antibiotici &laquo; my GenomiX'
  author_email: ''
  author_url: http://mygenomix.wordpress.com/2011/06/04/batterio-killer-continuano-le-analisi-potrebbe-essere-resistente-a-8-antibiotici/
  date: '2011-06-04 10:14:43 +0100'
  date_gmt: '2011-06-04 10:14:43 +0100'
  content: '[...] e proponendo assemblaggi diversi che possono essere visualizzati
    e confrontati seguendo questo link. Nel frattempo, al BGI hanno proseguito le
    analisi scoprendo che il ceppo in questione porterebbe [...]'
- id: 59683
  author: Nick Loman
  author_email: n.j.loman@bham.ac.uk
  author_url: http://xbase.bham.ac.uk
  date: '2011-06-04 10:53:16 +0100'
  date_gmt: '2011-06-04 10:53:16 +0100'
  content: Kat, that's fantastic. I also think some SNPs will be false positive because
    of a) low coverage of BGI dataset (a few more runs are available now) b) hybrid
    approach of the Life Tech assembly c) possibly homopolymers in Ion Torrent data
    d) potential misassemblies due to short read data. This is a great first step
    in getting some whole-genome phylogenies going on.
- id: 59684
  author: RickLStevens
  author_email: rick.stevens@mac.com
  author_url: ''
  date: '2011-06-04 14:44:09 +0100'
  date_gmt: '2011-06-04 14:44:09 +0100'
  content: "Hi Nick\r\n\r\nI've submitted your Mira assembly to the RAST microbial
    genome annotation server and shared it with the \"guest\" user, making it available
    to everyone.\r\n\r\nIt should be finished annotating in an hour or so.\r\n\r\nTo
    access it go to http://rast.nmpdr.org \r\n\r\nlogin is a \"guest\" with password
    \"guest\" it is job #26553\r\n\r\nThere are two other assemblies there that have
    also been shared with \"guest\" that might be of interest, \r\n\r\njob #26503
    which is the second BGI assembly with 419 contigs and #26498 which is the first
    assembly from BGI with 1218 contigs.\r\n\r\n--Rick"
- id: 59690
  author: fangfang
  author_email: fangfangxia@gmail.com
  author_url: ''
  date: '2011-06-05 14:13:21 +0100'
  date_gmt: '2011-06-05 14:13:21 +0100'
  content: "Hi Nick,\r\n\r\nWe have computed alignments, trees, evolutionary distances,
    etc for all the proteins between the two RAST annotated genomes and all the E.
    coli strains we have in the public SEED.\r\n\r\nhttp://bioseed.mcs.anl.gov/~fangfang/EHEC/LB.html\r\n\r\nThis
    is a large, sortable table. You need to log in to the guest account (guest/guest)
    for the links to the two RAST genomes to work.\r\n\r\nFangfang"
- id: 59691
  author: EAEC / STEC genomes &laquo; bacpathgenomics
  author_email: ''
  author_url: http://bacpathgenomics.wordpress.com/2011/06/06/eaec-stec-genomes/
  date: '2011-06-05 15:45:02 +0100'
  date_gmt: '2011-06-05 15:45:02 +0100'
  content: '[...] a few fimbrial genes annotated (in both the BG7 annotation and Torsten
    Seeman&#8217;s annotation of Nick Loman&#8217;s MIRA assembly of BGI&#8217;s TY2428
    data) but they are currently each in their own contigs, so it&#8217;s not really
    possible to get an idea [...]'
- id: 59692
  author: mshukla
  author_email: mshukla@vbi.vt.edu
  author_url: ''
  date: '2011-06-05 17:47:21 +0100'
  date_gmt: '2011-06-05 17:47:21 +0100'
  content: "Hi Nick,\r\n\r\nRAST annotations for Escherichia coli TY-2482 and Escherichia
    coli O104:H4 str. LB226692 genomes are available for download in various file
    formats (genbank, gff3, gtf, faa, fna) from PATRIC website:\r\n\r\nhttp://brcdownloads.vbi.vt.edu/patric2/EHEC/\r\n\r\n-Maulik"
- id: 59693
  author: Talks, Genomes, Reads and Annotations | the oh no sequences! blog
  author_email: ''
  author_url: http://blog.ohnosequences.com/2011/06/talks-genomes-reads-and-annotations/
  date: '2011-06-05 18:01:54 +0100'
  date_gmt: '2011-06-05 18:01:54 +0100'
  content: '[...] Loman (@pathogenomenick) published a de novo assembly of the reads
    with MIRA in his blog (see post here). And then, some hours later (in the morning
    of the 3rd of June) we published the annotation of the [...]'
- id: 59694
  author: the oh no sequences! blog
  author_email: ''
  author_url: http://blog.ohnosequences.com/2011/06/275/
  date: '2011-06-05 18:28:20 +0100'
  date_gmt: '2011-06-05 18:28:20 +0100'
  content: '[...] http://www.bgisequence.com/eu/index.php?cID=194 ) and assembled
    with MIRA by Nick Loman (6-2-2011  http://pathogenomics.bham.ac.uk/blog/2011/06/ehec-genome-assembly/
    [...]'
- id: 59695
  author: Gerhard Thallinger
  author_email: Gerhard.Thallinger@tugraz.at
  author_url: http://genome.tugraz.at/people.shtml
  date: '2011-06-06 12:19:17 +0100'
  date_gmt: '2011-06-06 12:19:17 +0100'
  content: "Hi Nick,\r\n\r\n&gt; BGI just released two more 314 chips worth of data
    and their own assembly of TY2482. \r\n&gt; I don’t have any details on program
    used or parameters just yet but I’ve enquired.\r\n\r\n    It seems to be an assembly
    based on newbler (judging from the contig names) with some \r\n    post processing.\r\n\r\n
    \  I performed a newbler assembly (2.5.3) myself using the 7 IonTorrent runs currently
    available.\r\n   This results in 1,398 contigs (&gt;100 bp) with an N50 of 9,107
    and a total of 5,228,942 bases.\r\n   The largest contig is 47,177 bps.\r\n\r\n
    \  If there is interest I can upload it to the github repository.\r\n\r\n   Does
    anybody know whether the raw data from the LB226692 is somewhere available ?\r\n\r\nGerhard"
- id: 59697
  author: Nick Loman
  author_email: n.j.loman@bham.ac.uk
  author_url: http://xbase.bham.ac.uk
  date: '2011-06-07 12:19:07 +0100'
  date_gmt: '2011-06-07 12:19:07 +0100'
  content: "@Gerhard\r\n\r\nThanks for stopping by. BGI have released details of their
    assembly methods now here: http://www.ncbi.nlm.nih.gov/nuccore/AFOG00000000\r\n\r\nYou
    could upload your Newbler assembly if you like to Github, although perhaps this
    may not add anything.\r\n\r\nAs far as I know the raw data for LB226692 is not
    available although a Life Tech representative told me they wanted to release it."
- id: 59698
  author: New German STEC/EHEC data from BGI &laquo; bacpathgenomics
  author_email: ''
  author_url: http://bacpathgenomics.wordpress.com/2011/06/07/new-german-stecehec-data-from-bgi/
  date: '2011-06-09 13:14:07 +0100'
  date_gmt: '2011-06-09 13:14:07 +0100'
  content: '[...] from BGI implies that all prior assemblies were reference based
    and not de novo. However the MIRA assembly of runs 1-5 of BGI reads, which has
    been annotated by ERA7 and analysed extensively, was a de novo [...]'
- id: 59699
  author: smithjr
  author_email: jason.smith@lifetech.com
  author_url: ''
  date: '2011-06-09 17:46:16 +0100'
  date_gmt: '2011-06-09 17:46:16 +0100'
  content: "Data generated by the Life Technologies FALCON team in Europe are now
    publicly available for download.\r\nYou can go to http://www.iontorrent.com/applications-speed/
    and start analyzing the data!"
- id: 59700
  author: john_doe
  author_email: python@nurfuerspam.de
  author_url: ''
  date: '2011-06-15 13:24:08 +0100'
  date_gmt: '2011-06-15 13:24:08 +0100'
  content: "Hi Nick,\r\n\r\ngreat blog, we always love to stop by!\r\nThe G2L team
    just released 454 data from two isolates from the German E. coli O104:H4 outbreak.
    You can find it on our website (which unfortunately is in German):\r\nhttp://www.g2l.bio.uni-goettingen.de/\r\nThe
    link to the ftp server is ftp://134.76.70.117/\r\nUser name and password are 'EAHEC_GOS'.\r\nWould
    be great if you could post it on the github E. coli O104:H4 Genome Analysis Crowdsourcing
    (however, we already posted it also to Kat Holt ...)\r\nThanks and keep up the
    good work!\r\nCheers,\r\nG2L team"
- id: 59702
  author: Offene EHEC-Forschung | wisspub.net
  author_email: ''
  author_url: http://wisspub.net/2011/06/19/offene-ehec-forschung/
  date: '2011-06-19 18:18:23 +0100'
  date_gmt: '2011-06-19 18:18:23 +0100'
  content: '[...] und niedrigschwellige Kommunikation fördern, lässt sich z.B. an
    Blogposts von Nick Loman (Pathogens: Genes and Genomes) und Kat Holt (bacpathgenomics)
    und den dort erwähnten Referenzen [...]'
- id: 59707
  author: E-gob.pe | Gobierno Electrónico &#124; Blog &#124; La ciencia 2.0 mató a
    la bacteria &#039;E.Coli&#039;
  author_email: ''
  author_url: http://e-gob.pe/2011/07/18/la-ciencia-2-0-mato-a-la-bacteria-e-coli/
  date: '2011-07-18 02:48:28 +0100'
  date_gmt: '2011-07-18 02:48:28 +0100'
  content: '[...] de Sistemas de la Universidad de Birmingham. Como los chinos del
    BGI, Loman puso en internet su primer ensamblaje preliminar . Segundo [...]'
- id: 59795
  author: '2011: Review of a Remarkable Year'
  author_email: ''
  author_url: http://pathogenomics.bham.ac.uk/blog/2011/12/2011-review-of-a-remarkable-year/
  date: '2011-12-31 09:36:39 +0000'
  date_gmt: '2011-12-31 09:36:39 +0000'
  content: '[...] started with Nick Loman filling me in on the crowdsourcing efforts
    that he had jump-started with his assembly of the BGI&#8217;s Ion Torrent data.
    I was initially dismissive of all this activity on social media, in the absence
    of any plans for a [...]'
- id: 59802
  author: Some More Thoughts About the German E. coli Outbreak | Mike the Mad Biologist
  author_email: ''
  author_url: http://mikethemadbiologist.com/2011/06/06/some_more_thoughts_about_the_g/
  date: '2012-02-03 18:57:17 +0000'
  date_gmt: '2012-02-03 18:57:17 +0000'
  content: '[...] sequenced E. coli genomes E. coli update: sprouts as the culprit?
    ehec-outbreak-crowdsourced EHEC Genome Assembly E. coli TY2482: strain-specific
    genes The reason why this deadly E coli makes doctors shudder Share [...]'
- id: 59876
  author: 'Dos pepinos espanhóis ao genoma nos blogs: E. coli patogênica na Alemanha
    | Rainha Vermelha'
  author_email: ''
  author_url: http://scienceblogs.com.br/rainha/2011/06/dos_pepinos_espanhis_ao_genoma/
  date: '2012-03-07 08:24:12 +0000'
  date_gmt: '2012-03-07 08:24:12 +0000'
  content: '[...] Pathogens: Genes and Genomes &#8211; EHEC Genome Assembly [...]'
- id: 59970
  author: 'E. coli TY2482: strain-specific genes | ExCePD'
  author_email: ''
  author_url: http://bio-engawa.ex.ac.uk/djs217/excepd/?p=261
  date: '2013-02-02 19:24:07 +0000'
  date_gmt: '2013-02-02 19:24:07 +0000'
  content: '[...] data generated using Ion Torrent. With equally astonishing speed,
    Nick Loman has generated a preliminary de novo sequence assembly and Marina Manrique
    generated a preliminary annotation of that [...]'
- id: 60134
  author: Our Modern Day John Snow teaches us about cool crap | Evolution and Genomics
  author_email: ''
  author_url: http://evomics.org/2014/01/modern-day-john-snow/
  date: '2014-01-23 18:44:58 +0000'
  date_gmt: '2014-01-23 18:44:58 +0000'
  content: '[&#8230;] posted a blog linking to Ion Torrent data and asked crowd sourcing
    the analysis of [&#8230;]'
---
<p><strong>Keep track of the genomic analysis of the EHEC strains on our <a href="https://github.com/ehec-outbreak-crowdsourced/BGI-data-analysis/wiki">Github Wiki</a>.</strong></p>
<p>BGI have released 5<a href="http://www.bgisequence.com/eu/index.php?cID=194&amp;sms_ss=twitter&amp;at_xt=4de74702e845e3a2,0"> runs of Ion Torrent data</a> for the German EHEC/VTEC outbreak strain. I hope it is released with no specific restrictions on use for the benefit of the entire community, but the site doesn't make that entirely clear. Thanks to the BGI for putting it up!</p>
<p>Shall we crowd source some analysis? This comes at a very timely moment as I am currently help organise the Applied Bioinformatics &amp; Public Health conference in Hinxton (<a href="http://search.twitter.com/search?q=ABPH11">#ABPH11</a>), where we are discussing the use of whole-genome sequencing in epidemiology. The problem is I don't have much time to dig into the data.</p>
<p>But I've put a first-pass de novo assembly up using MIRA (3.2.1.17_dev) <a href="http://static.xbase.ac.uk/files/results/nick/TY2482/TY2482.fasta.txt">here</a>. 3,057 contigs, total bases: 5,491,032, N50 3,675. If you want the alignment files etc. get the big file <a href="http://static.xbase.ac.uk/files/results/nick/TY2482/TY2482.all.tgz">here</a> (282Mb).</p>
<p>Parameters are: mira --job=denovo,genome,accurate,iontor -GE:not=1</p>
<p><strong>Update 3/6/11 09:15 GMT+1</strong></p>
<p>Marina Manrique has run the assembly through their <a href="http://ngsbioinformatics.com/">BG7 bacterial genome annotation pipeline</a>, results are <a href="http://www.era7bioinformatics.com/en/E_Coli_EHEC_O104_STRAIN_EU_OUTBREAK_era7bioinformatics.html">here</a>.</p>
<p>Torsten Seemann and Simon Gladman from the <a href="http://www.vicbioinformatics.com/">Victorian Bioinformatics Consortium</a> have sent me the results of their in-house annotation pipeline. Results are available: contigs reordered according to <em>E. coli</em> <a href="http://static.xbase.ac.uk/files/results/nick/TY2482/German_vs_55989.gbk">EAEC 55989</a> and <a href="http://static.xbase.ac.uk/files/results/nick/TY2482/German_vs_TWEC.gbk">TWEC.</a></p>
<p>NCBI have also posted a <a href="http://www.ncbi.nlm.nih.gov/nuccore/AFOB00000000">preliminary assembly</a> (of a different isolate - LB226692) - although it is not a true de novo assembly. The approach is a bit different. "Reads were mapped with TMAP against the publicly available E. coli 55989 chromosome (CU928145.2) and the derived consensus was split into contigs at zero-coverage regions. These contigs were used as a 'backbone' for mapping of reads, followed by de novo assembly of unmapped reads with the MIRA assembler (v 3.2.1). A small number of de novo and consensus contigs were merged using CAP3."</p>
<p><strong>Update 3/6/11 16:50 GMT+1</strong></p>
<p>There are two O104 isolates sequenced from this outbreak now. This first - named TY2482 - was done by BGI in collaboration with University Medical Centre Hamburg-Eppendorf and the second was done by <a href="http://www.lifetech.com/news-gallery/press-releases/2011/dna-sequencing-data-reveals-new-hybrid-e.html">Life Tech in-house in collaboration with University of Muenster</a> - this is called LB226692. So opportunities for comparison exist now.</p>
<p>In summary: <a href="http://static.xbase.ac.uk/files/results/nick/TY2482/TY2482.fasta.txt">TY2482 assembly (BGI reads, my assembly)</a>, <a href="http://www.ncbi.nlm.nih.gov/nuccore/AFOB00000000">LB226692 assembly (Life Tech reads, assembly)</a>.</p>
<p><a href="http://scienceblogs.com/mikethemadbiologist/2011/06/i_dont_think_the_german_e_coli.php">Mike the Mad Biologist has looked at the TY2482 assembly</a> and concludes it is ST678 (or closely related) which agrees with the original molecular typing release from the <a href="http://www.rki.de/cln_169/nn_217400/EN/Home/EHEC__O104__H4,templateId=raw,property=publicationFile.pdf/EHEC_O104_H4.pdf">Robert Koch Institute</a>.</p>
<p>I've heard from another group they are planning on sequencing another isolate. I am going to try and find a place where the latest information can be collated to aid in further crowd-sourcing analysis.</p>
<p><strong>Update 3/6/11 19:50 GMT+1</strong></p>
<p>BGI just released <a href="ftp://ftp.genomics.org.cn/pub/Ecoli_TY-2482/">two more 314 chips</a> worth of data and their <a href="ftp://ftp.genomics.org.cn/pub/Ecoli_TY-2482/Escherichia_coli_TY-2482.contig.fa.gz">own assembly of TY2482</a>. I don't have any details on program used or parameters just yet but I've enquired.</p>
<p>Who will take on the challenge of building a whole-genome phylogeny?</p>
<p><strong>Update 4/6/11 16:15 GMT+1</strong></p>
<p>A few notable updates.</p>
<p>Kat Holt has picked up the gauntlet of doing some whole-genome SNP comparisons of the strains. <a href="http://bacpathgenomics.wordpress.com/2011/06/04/ehec-genomes/">Results here</a>.</p>
<p>David Studholme has looked for <a href="http://www.genomic.org.uk/blog/?p=261">strain-specific genes</a> in TY2482 and found some, including a class A beta-lacatamase.</p>
<p>BGI have <a href="http://www.bgisequence.com/eu/newsandevents/news/further-analysis-on-improved-genome-assembly-indicates-the-outbr/">published some more analysis</a> of the genomes and have suggested people use their assembly for further comparison. However I still don't have any details on how that assembly was done (I have asked), which seems important.</p>
<p>Some more useful discussion from <a href="http://phylogeo.wordpress.com/2011/06/04/7-2/">Phylogeo about the novelty of this strain</a>. I think the consensus is now that this strain has been seen and subsequently typed in the past (hence ST678 - not a new sequence type), but before now we did not have a genome sequence for this particular strain. More discussion over at <a href="http://scienceblogs.com/aetiology/2011/06/e_coli_o104h4_in_europe--is_it.php?utm_source=feedburner&amp;utm_medium=feed&amp;utm_campaign=Feed%3A+scienceblogs%2Faetiology+%28Aetiology%29">Aetiology</a>.</p>
<p>Marina Manrique has set up a Github <a href="https://github.com/ehec-outbreak-crowdsourced/BGI-data-analysis">repository</a> and <a href="https://github.com/ehec-outbreak-crowdsourced/BGI-data-analysis/wiki">Wiki</a> for this EHEC crowd-sourcing project. I am going to have a play around with this and hopefully we can start keeping all our crowd-sourced data here in a logical format.</p>
<p>Some RAST annotations are available, see the comments thread.</p>
<p><strong>Update 6/6/11 10:54 GMT+1</strong></p>
<p>Keep track of the genomic analysis of the EHEC strains on our <a href="https://github.com/ehec-outbreak-crowdsourced/BGI-data-analysis/wiki">Github Wiki</a>.</p>
