---
layout: post
status: publish
published: true
title: All the tweets from ABPH11
author:
  display_name: Nick Loman
  login: nick
  email: n.j.loman@bham.ac.uk
  url: http://xbase.bham.ac.uk
author_login: nick
author_email: n.j.loman@bham.ac.uk
author_url: http://xbase.bham.ac.uk
wordpress_id: 674
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=674
date: '2011-06-03 10:13:20 +0100'
date_gmt: '2011-06-03 10:13:20 +0100'
categories:
- Uncategorized
tags: []
comments: []
---
<p>The tweets from Applied Bioinformatics & Public Health 2011- preserved for posterity.</p>
<p><em>pathogenomenick</em>: Started on my train journey through the bucolic scenery of middle England to Hinxton, nr. Cambs for #ABPH11 (invented hashtag)<br />
<em>attilacsordas</em>: 'Applications of 3rd Gen Sequencing in Public Health Microbiology Andrew Kasarskis' @PacBio UK' #ABPH11 he was working 4 @Sagebio before<br />
<em>pathogenomenick</em>: Checked-in at Hinxton Hall for #ABPH11 - view from my window. Weather is gorgeous!http://yfrog.com/h26h8apxj <a href="http://yfrog.com/h26h8apxj">http://yfrog.com/h26h8apxj</a><br />
<em>pathogenomenick</em>: OK time to kick off #ABPH11. Looking forward to talks from Roche, Illumina, Ion Torrent and PacBio shortly. Hoping for some juicy nuggets.<br />
<em>pjacock</em>: @pathogenomenick What's the full name of #ABPH11 and is there a website for the meeting? Ta!<br />
<em>pathogenomenick</em>: I'm not going tweet crazy this meeting to save your timelines. But I will tweet things I think are new / particularly interesting #ABPH11<br />
<em>jacarrico</em>: @pathogenomenick So lets keeps #ABPH11 as oficial tag ?<br />
<em>jacarrico</em>: &quot;Mouth is a whole collection of microhabitats&quot; -William Wade #ABPH11<br />
<em>pathogenomenick</em>: William Wade is describing the process of trying to culture an &quot;uncultivable&quot; isolate of Synergistetes http://bit.ly/jwDC4Z #ABPH11 <a href="http://bit.ly/jwDC4Z">http://bit.ly/jwDC4Z</a><br />
<em>pathogenomenick</em>: It took 4 months of laborious co-culturing and passage to cultivate P. micra #ABPH11<br />
<em>jacarrico</em>: http://t.co/tK3GMqb - Human Oral Microbiome DB #ABPH11 <a href="http://t.co/tK3GMqb">http://t.co/tK3GMqb</a><br />
<em>pjacock</em>: Thanks @pathogenomenick #ABPH11 is Applied Bioinformatics &amp; Public Health Microbiology 2011, at Sanger Inst, Hinxton http://bit.ly/e6Dipx <a href="http://bit.ly/e6Dipx">http://bit.ly/e6Dipx</a><br />
<em>pathogenomenick</em>: William Wade has a very nice database www.homd.org - 619 bact &quot;species&quot; represented, 66% of those cultured. 113 un-named. #ABPH11<br />
<em>pathogenomenick</em>: My 'p' key is broken. This is going to make life difficult. #ABpH11<br />
<em>pathogenomenick</em>: You will struggle to identify many species by 16S - particularly Streptococcus - even with full-length sequences #ABpH11<br />
<em>pathogenomenick</em>: Actinobacteria are  underrepresented in 16S clone libraries: why - lysis? high GC? primers turned out to be the major problem #ABpH11<br />
<em>aunderwo</em>: William Wade: Both bacterial culture and PCR amplification of 16S rRNA gene introduce their own biases when examining microbiomes #ABPH11<br />
<em>jacarrico</em>: WilliamWade -&quot;Mouth is sterile at birth and gets colonized in the first 18 h!&quot; #ABPH11<br />
<em>aunderwo</em>: William Wade: By 16S rDNA sequencing found 50% of oral microbiome is unculturable #ABPH11<br />
<em>pathogenomenick</em>: OK time for some instrument talks - Jason Myers up first talking Ion Torrent - &quot;democratising sequencing&quot; #ABPH11<br />
<em>iddux</em>: @pathogenomenick: My 'p' key is broken. This is going to make life difficult. #ABpH11 &lt;- hard to 'p'<br />
<em>pathogenomenick</em>: Not sure exactly the content of this talk but suspect it will have overlap with these slides from AACC http://bit.ly/ieQ81s #ABPH11 <a href="http://bit.ly/ieQ81s">http://bit.ly/ieQ81s</a><br />
<em>avilella</em>: Ion Torrent guy showing a slide that shows 1Gb and 200-400bp reads for 2012 #ABPH11<br />
<em>aunderwo</em>: Ion Torrent is to sequencing technologies as PCs are to computing - everybody can potentially sequence a genome #ABPH11<br />
<em>aunderwo</em>: Next year Ion Torrent should yield 400bp reads and 1Gb of sequence #ABPH11<br />
<em>fionabrinkman</em>: At Applied Bioinformatics &amp; Public Health Microbiology 2011 meeting and now have wifi. Agenda: http://goo.gl/hGVPM #ABPH11 <a href="http://goo.gl/hGVPM">http://goo.gl/hGVPM</a><br />
<em>jacarrico</em>: Ion Torrent - Jasom Myers - PGM as a &quot;disruptive technology &quot; as PCs were #ABPH11<br />
<em>jennifergardy</em>: Follow @pathogenomenick and I for scintillating back-and-forth genomics gossip and commentary live from #ABPH11 at Hinxton!<br />
<em>pathogenomenick</em>: Ion Torrent guy is predicting linear read length improvement but his Y axis is all funny so not sure exactly when. #ABPH11<br />
<em>jacarrico</em>: Linear read length predictions for PGM seem a little forced after those bumps on the line .. #ABPH11<br />
<em>pathogenomenick</em>: I may give an impromptu talk tomorrow on Ion Torrent data analysis to make up for a missing speaker #ABPH11<br />
<em>aunderwo</em>: Whole transcriptome RNA sequencing will be possible with Ion Torrent 'soon' according to their rep #ABPH11<br />
<em>pathogenomenick</em>: Ion Torrent claim good consensus accuracy for 8/9-mer homopolymers, using a 36x coverage assembly example in H. pylori #ABPH11<br />
<em>aunderwo</em>: H.pylori de novo assembly resulted in 150 contigs with Ion Torrent (data from Penn State U) #ABPH11<br />
<em>pathogenomenick</em>: The Broad have done some early experiments with mate-pair sequencing on Ion Torrent using insert lengths of around 1.5kb #ABPH11<br />
<em>aunderwo</em>: Mate paired libraries with 1.5kb inserts have been achieved with Ion Torrent PGM #ABPH11<br />
<em>fionabrinkman</em>: Now doing Sequencing Tech talks (getting caught up...) #ABPH11<br />
<em>pathogenomenick</em>: We asked each company to talk directly about public health applications. This is a little bit generic but early days I guess #ABPH11<br />
<em>peterdilaura</em>: @pathogenomenick thanks in advance for the #ABPH11 tweets.<br />
<em>phylogenomics</em>: RT @jennifergardy: Follow @pathogenomenick and I for scintillating back-and-forth genomics gossip and commentary live from #ABPH11 at Hinxton!<br />
<em>pathogenomenick</em>: I asked why the 316 &amp; 318 chips aren't out yet. The issue seems to relate to fluidics rather than the semiconductor design or mfctr #ABPH11<br />
<em>pathogenomenick</em>: Geoff Smith talking about MiSeq - he's a very clever man in fact. Solexa started down the road from Hinxton in Gt. Chesterford #ABPH11<br />
<em>aunderwo</em>: At volume it is now possible to sequence a human genome for $4k using Illumina HiSeq #ABPH11<br />
<em>pathogenomenick</em>: HiSeq 2000 - 8 human genomes per Tb or 8000  bacterial genomes per run. I know what I'd prefer! #ABPH11<br />
<em>avilella</em>: Illumina HiSeq now: 600Gb per run. Latest R&amp;D number: more than 1Tb per run #ABPH11<br />
<em>pathogenomenick</em>: MiSeq data more or less equivalent to HiSeq data, but less of it. More about this here: http://bit.ly/lB6Y9U #ABPH11 <a href="http://bit.ly/lB6Y9U">http://bit.ly/lB6Y9U</a><br />
<em>aunderwo</em>: All methods on HiSeq are interchangeable with MiSeq - equivalent base accuracy etc also #ABPH11<br />
<em>pathogenomenick</em>: MiSeq has a single well for sample loading. Does cluster gen on-board. So there are no &quot;lane&quot; equivalents as per HiSeq/GA2x #ABPH11<br />
<em>jacarrico</em>: Geoff Smith giving a very convincing talk about MiSeq #ABPH11<br />
<em>pathogenomenick</em>: HiSeq: 19m chemistry, 23m imaging per base. MiSeq: 4m chem, 1m imaging per cycle. I guessed this in Jan http://bit.ly/fkVMyL #ABPH11 <a href="http://bit.ly/fkVMyL">http://bit.ly/fkVMyL</a><br />
<em>aunderwo</em>: Improvements in chemistry and imaging enable the cycle time in the MiSeq to be 5 mins as opposed to 42 mins for HiSeq #ABPH11<br />
<em>pathogenomenick</em>: Wireless network at Hinxton is super-fast compared to ASM. #ABPH11<br />
<em>assemblathon</em>: For great coverage of the #ABPH11 conference, follow @pathogenomenick. He's dropping some tasty morsels of NGS information today.<br />
<em>jacarrico</em>: 6 twitters so far ! Not bad! #ABPH11<br />
<em>pathogenomenick</em>: Super deep sequencing of KRAS allows detection of 1.1% variant frequency using MiSeq. This is going to take over cancer screening. #ABPH11<br />
5<br />
<em>fionabrinkman</em>: #ABPH11 I'd love a MiSeq but my Dept says it needs higher capacity for Euk genomes :( Need to convince them otherwise...<br />
<em>fionabrinkman</em>: Yes RT @assemblathon: For great coverage of #ABPH11 follow @pathogenomenick. He's dropping some tasty morsels of NGS information today.<br />
<em>pathogenomenick</em>: Geoff claims Nextera library prep enables 'sample-to-answer' in a single working day with Miseq. That is a game-changer for ID/dx. #ABPH11<br />
<em>pathogenomenick</em>: And that's an 8 hour working day like in UK not an American working day of 18 hours ;) #ABPH11<br />
<em>aunderwo</em>: Nextera library prep (single tube) and sequencing 2x36bp on MiSeq enables sample to answer in 1 working day #ABPH11<br />
<em>pathogenomenick</em>: Gets N50 of 148kb HiSeq 132kb MiSeq from E. coli assembly (not sure about other params) #ABPH11<br />
<em>aunderwo</em>: Possible to find SNPs involved in drug resistance in TB strains using MiSeq sequencing #ABPH11<br />
<em>jennifergardy</em>: I'm switching my Christmas wish from a pony to an Illumina MiSeq. If they could throw in a pony w/ the machine, that would be great #ABPH11<br />
<em>jacarrico</em>: Very nice examples of use for microbiology using miseq: TB, pseudomonas, ecoli sequencing #ABPH11<br />
<em>pathogenomenick</em>: Presenting CF sputum metagenomics using HiSeq., PA LESB58  came out at 636x depth, plus phage &amp; 7 other genomes 50-86% covered #ABPH11<br />
<em>pathogenomenick</em>: After depleting human DNA, CF sputum DNA is &gt;70% bacterial #ABPH11<br />
<em>jacarrico</em>: Hiseq for metagenomics / Miseq to characterize individual isolates #ABPH11<br />
<em>fionabrinkman</em>: MiSeq can seq P. aeruginosa LES genome accurately vs ref. which is good since large, high G+C. Using HiSeq for metagenomics though #ABPH11<br />
<em>pathogenomenick</em>: Good assembly results using Nextera sample prep on MRSA. That's good, was worried about biases forom this method. #ABPH11<br />
<em>GuillaumeMeric</em>: RT @jennifergardy: Follow @pathogenomenick and I for scintillating back-and-forth genomics gossip and commentary live from #ABPH11 at Hinxton!<br />
<em>pathogenomenick</em>: Geoff Smith thinks - like we do - that sequencing can replace the traditional clinical microbiology paradigm of culture &amp; sens. #ABPH11<br />
<em>pathogenomenick</em>: What we really want is HiSeq amounts of data in MiSeq timescales of course, for routine clinical metagenomics! #ABPH11<br />
<em>fionabrinkman</em>: @jennifergardy #ABPH11 yes, even the MiSeq name is cute. Keeps reminding me of my Wii Mii!<br />
<em>aunderwo</em>: Routine WGS for pathology: 3 colonies=&gt; extraction=&gt; library prep=&gt; overnight MiSeq sequencing=&gt; whole genome &amp; analysis next am #ABPH11<br />
<em>jacarrico</em>: Atb resistance prediction from genome sequence within a day with Miseq... Still lots to do but a goal #ABPH11<br />
<em>pathogenomenick</em>: Geoff Smith seems to imply that Velvet would be supplied with the MiSeq for de novo assembly - seems odd but we'll see #ABPH11<br />
<em>jacarrico</em>: PacBio sequencing now ! #ABPH11<br />
<em>jennifergardy</em>: AAAAAAAHHHH IT'S PACBIO TIME AT #ABPH11 YAYAYAYAYAAAAY!!!!!<br />
<em>avilella</em>: The MiSeq pipeline will run the latest version of Velvet assembler as you can find at dzerbino's website. Nothing closed and canned. #ABPH11<br />
<em>epiexperts</em>: Interesting tweets from #abph11 Applied Bioinformatics &amp; Public Health Microbiology 2011 http://search.twitter.com/search?q=%23ABPH11 #fb <a href="http://search.twitter.com/search?q=%23ABPH11">http://search.twitter.com/search?q=%23ABPH11</a><br />
<em>jennifergardy</em>: I've heard the PacBio reads are indel-y &amp; kind of crap, but the chiseled good looks of the PacBio rep make up for it #ABPH11<br />
<em>pathogenomenick</em>: Pac Bio guy is rocking some really sci-fi slides. Well done. #ABPH11<br />
<em>pathogenomenick</em>: Very cool real-time images of SMRT sequencing. Go to their site for more: http://bit.ly/l81uNH #ABPH11 <a href="http://bit.ly/l81uNH">http://bit.ly/l81uNH</a><br />
<em>HelicosUnveiled</em>: @pathogenomenick Have there been any Helicos Biosciences sightings? #ABPH11<br />
<em>aunderwo</em>: Trace coming off the PacBio machine looks noisy - I suspect quality of reads may take a while to get right #ABPH11<br />
<em>fionabrinkman</em>: Like insights that could be gained about polymerase function/pausing from PacBio seq data. Realtime data = beautiful biology. #ABPH11<br />
<em>pathogenomenick</em>: Raw error rate around 11%, mainly indels. But crucially they are constant over length of read which makes life much easier. #ABPH11<br />
<em>fionabrinkman</em>: @pathogenomenick yes! Next Star Trek movie must show shots of PacBio sequencing #ABPH11<br />
<em>pathogenomenick</em>: Hmm bit disappointed this is a pure sequencing talk so far and nothing about public health (yet, anyway) #ABPH11<br />
<em>pathogenomenick</em>: OK, going to talk about Haiti cholera outbreak now. 12-fold genome coverage achieved in 90 minutes. Wow, those bugs are in log phase #ABPH11<br />
<em>jacarrico</em>: PacBio can detect DNA methylation by the time that the polymerase pauses #ABPH11<br />
<em>kshameer</em>: RT @jacarrico: PacBio can detect DNA methylation by the time that the polymerase pauses #ABPH11<br />
<em>mgollery</em>: RT @avilella: The MiSeq pipeline will run the latest version of Velvet assembler as you can find at dzerbino's website. Nothing closed and canned. #ABPH11<br />
<em>pathogenomenick</em>: PacBio gives more even coverage of genome compared to Illumina - this is due to amplification bias. Models Poisson very well. #ABPH11<br />
<em>fionabrinkman</em>: RT @pathogenomenick: OK, going to talk about Haiti cholera outbreak now. 12-fold genome coverage achieved in 90 minutes. Wow, those bugs are in log phase #ABPH11<br />
<em>pathogenomenick</em>: PacBio cholera sequencing is kind of cool but its not a killer app - not seeing anything we can't do with 454 just yet. #ABPH11<br />
<em>aunderwo</em>: Long reads of PacBio may resolve genome gaps due to repeats #ABPH11<br />
<em>pathogenomenick</em>: The 454 read length / throughput timeline rather compresses 2008 - 2011 :( #ABPH11 (because nothing happened)<br />
<em>aunderwo</em>: So far the 454 presentation is a little underwhelming - needs to cut to the chase. We want something new and shiny! #ABPH11<br />
<em>pathogenomenick</em>: follow #ABPH11 tweeps @jennifergardy @fionabrinkman @aunderwo @jacarrico @avilella<br />
<em>TParusheva</em>: oh wow!!interesting..@kshameer @jacarrico RT PacBio can detect DNA methylation by the time that the polymerase pauses #ABPH11<br />
<em>jacarrico</em>: @pathogenomenick: follow #ABPH11 tweeps @jennifergardy @fionabrinkman @aunderwo @jacarrico @avilella&quot; @attilacsordas too ;-)<br />
<em>PacBio</em>: RT @jennifergardy: AAAAAAAHHHH IT'S PACBIO TIME AT #ABPH11 YAYAYAYAYAAAAY!!!!!<br />
<em>pathogenomenick</em>: 454 8kb PE data can produce single scaffolds for S. pneumoniae, E. coli, T. thermophilus, C. jejuni (it's true, we've done it too) #ABPH11<br />
<em>pathogenomenick</em>: Oh that's a shame, 454 rep is presenting the FDA paper which is scheduled for a full talk tomorrow. #ABPH11<br />
<em>fionabrinkman</em>: @pathogenomenick @jennifergardy  NEJM in the lead with most citations at this #ABPH11 Seq Tech session<br />
<em>aunderwo</em>: The scaffold size/number produced from large insert 454 library sequencing is still unparalleled. #ABPH11<br />
<em>pathogenomenick</em>: Pac Bio guy said the machine weighs 7 of him, and let me tell you he is a big guy  (ask @jennifergardy) #ABPH11<br />
<em>aunderwo</em>: For most of my microbial projects I would love a 454 de novo assembly followed by cheap as chips Illumina fragment lib sequencing #ABPH11<br />
<em>jacarrico</em>: 454 talk can have a slow pace but has very nice application examples published #ABPH11<br />
<em>pathogenomenick</em>: &gt;1,205 publications using 454 sequencing does show has been a very useful technology. #ABPH11<br />
<em>fionabrinkman</em>: #ABPH11 Seq tech talks reveal benefit of really coming through with working tech. HiSeq, MiSeq, 454 looks good. PacBio still exciting.<br />
<em>PacBio</em>: RT @jacarrico: PacBio can detect DNA methylation by the time that the polymerase pauses #ABPH11<br />
<em>PacBio</em>: RT @fionabrinkman: Like insights that could be gained about polymerase function/pausing from PacBio seq data. Realtime data = beautiful biology. #ABPH11<br />
<em>PacBio</em>: RT @pathogenomenick: Pac Bio guy is rocking some really sci-fi slides. Well done. #ABPH11<br />
<em>pathogenomenick</em>: Let me tell you we won't be over-running in tomorrow's session that I am chairing with Muna! #ABPH11<br />
<em>aunderwo</em>: 454 deep amplicon sequencing of HIV detects very minor variants in V3 loop which may alter viral tropism #ABPH11<br />
<em>PacBio</em>: RT @pathogenomenick: PacBio gives more even coverage of genome compared to Illumina - this is due to amplification bias. Models Poisson very well. #ABPH11<br />
<em>pathogenomenick</em>: XLR70 consensus accuracy is 99.997% versus 99.995% for FLX but this is not a meaningful measurement! #ABPH11<br />
<em>aunderwo</em>: FLX+ has a modal read length of 700bp - approaching read lengths of Sanger sequencing. Base accuracy 99.99%+ with 15x coverage #ABPH11<br />
<em>avilella</em>: 454 FLX Plus modal 700bp, 85% above 500bp, total 700MB per run, 23 hours, accuracy a couple of 10^-5 extra pc points: 99,997% #ABPH11<br />
<em>jacarrico</em>: 454 GS Flex + has 80% of reads greater than 500 bp and up to 1kbp. #ABPH11<br />
<em>kshameer</em>: RT @jacarrico: 454 GS Flex + has 80% of reads greater than 500 bp and up to 1kbp. #ABPH11<br />
<em>avilella</em>: Roche+IBM so far only hopes, as much as the 100KB Oxford nanopore mountains of reads we heard before in the twittersphere #ABPH11<br />
<em>avilella</em>: 454 guy, his 5 year old daughter thought he was behind learning the alphabet, stuck with only the A, T, C and G. #ABPH11<br />
<em>Jeylen</em>: RT @avilella  454 FLX: modal 700bp, 85% &gt;500bp, total 700MB per run, 23 hours, accuracy a couple of 10^-5 extra pc points: 99,997% #ABPH11<br />
<em>PacBio</em>: RT @aunderwo: Long reads of PacBio may resolve genome gaps due to repeats #ABPH11<br />
<em>aunderwo</em>: Julian Parkhill:WGS allows tracking of a single clone that spreads rapidly through a pop where MLST not sufficient to differentiate #ABPH11<br />
<em>fionabrinkman</em>: Julian Parkhill starting talk about seq for tracking bacterial disease. Will mention recent data but my battery is now low darnit #ABPH11<br />
<em>pathogenomenick</em>: I'm not going to tweet Julian Parkhill's talk as I did this at ASM but will jump in when/if there's new data. #ABPH11<br />
<em>aunderwo</em>: Julian Parkhill - removing SNPs in regions with clusters of SNPs (probably due to recomb) results in a more robust SNP-based tree #ABPH11<br />
<em>fionabrinkman</em>: Parkhill talking about rapid Strep pneumo evolution from this Science paper: http://goo.gl/hbREV #ABPH11 <a href="http://goo.gl/hbREV">http://goo.gl/hbREV</a><br />
<em>jacarrico</em>: In Spneumo Capsular switch is recombination of 20 kb or more , so it should be a rare event #ABPH11<br />
<em>pathogenomenick</em>: Vibrio cholera outbreak - this is new. Sequenced 136 isolates from a global collection spanning 100 years. #ABPH11<br />
<em>aunderwo</em>: Julian Parkhill:WGS of S.pneumo provides means of calculating frequency of base substitutions, recomb events and capsular switching #ABPH11<br />
<em>pathogenomenick</em>: Little evidence of recombination in core genome of El Tor tree. Strains nicely cluster by date back to 1954. #ABPH11<br />
<em>CrowdedHead</em>: RT @pathogenomenick: Vibrio cholera outbreak - this is new. Sequenced 136 isolates from a global collection spanning 100 years. #ABPH11<br />
<em>pathogenomenick</em>: &quot;Waves of transmission&quot; - quite a complex phylogeographical pattern of multiple transmissions between continents. #ABPH11<br />
<em>pathogenomenick</em>: Looks fairly compelling as a recent transmission from SE Asia to Haiti. Amazing data. #ABPH11<br />
<em>fionabrinkman</em>: Parkhill showed nice new V. cholera genomic epi data. Says errs on reducing false +ve SNPs (~1/genome) vs missing 5-10% SNPs. Good. #ABPH11<br />
<em>lltripp</em>: Reading #ABPH11 tweets and hoping to absorb some subject expertise by osmosis #imasciencegroupie<br />
<em>Chris_Evelo</em>: Wow! Retweet @kshameer RT @jacarrico: 454 GS Flex + has 80% of reads greater than 500 bp and up to 1kbp. #ABPH11<br />
<em>lexnederbragt</em>: RT @pathogenomenick: PacBio gives more even coverage of genome compared to Illumina - this is due to amplification bias. Models Poisson very well. #ABPH11<br />
<em>lexnederbragt</em>: RT @pathogenomenick: PacBio cholera sequencing is kind of cool but its not a killer app - not seeing anything we can't do with 454 just yet. #ABPH11<br />
<em>lexnederbragt</em>: RT @pathogenomenick: The 454 read length / throughput timeline rather compresses 2008 - 2011 :( #ABPH11 (because nothing happened)<br />
<em>lexnederbragt</em>: We too! MT @pathogenomenick: 454 8kb PE data can produce single scaffolds for S. pneumoniae, E. coli, (it's true, we've done it too) #ABPH11<br />
<em>lexnederbragt</em>: RT @pathogenomenick: &gt;1,205 publications using 454 sequencing does show has been a very useful technology. #ABPH11<br />
<em>norseqcenter</em>: RT @jacarrico: PacBio can detect DNA methylation by the time that the polymerase pauses #ABPH11<br />
<em>norseqcenter</em>: RT @fionabrinkman: Like insights that could be gained about polymerase function/pausing from PacBio seq data. Realtime data = beautiful biology. #ABPH11<br />
<em>norseqcenter</em>: RT @pathogenomenick: PacBio gives more even coverage of genome compared to Illumina - this is due to amplification bias. Models Poisson very well. #ABPH11<br />
<em>norseqcenter</em>: RT @aunderwo: Long reads of PacBio may resolve genome gaps due to repeats #ABPH11<br />
<em>norseqcenter</em>: RT @pathogenomenick: PacBio cholera sequencing is kind of cool but its not a killer app - not seeing anything we can't do with 454 just yet. #ABPH11<br />
<em>RebeccaGladston</em>: Great first afternoon of talks, great to have so much relevant info #ABPH11<br />
<em>4a6a5a</em>: RT @avilella: The MiSeq pipeline will run the latest version of Velvet assembler as you can find at dzerbino's website. Nothing closed and canned. #ABPH11<br />
<em>iddux</em>: @fionabrinkman @pathogenomenick #ABPH11 So anything about the EHEC outbreak?<br />
4<br />
<em>jennifergardy</em>: Beautiful sunny day here at #ABPH11 - we should move the poster session outside. #genomicsismorefuninthesun<br />
<em>pathogenomenick</em>: Oof, OK here goes for #ABPH11 day 2 - just ate a bacon sandwich in the conference room.  I am chairing, tweeting and talking this session!<br />
<em>pathogenomenick</em>: Victor Solovyev is presenting FgenesB &amp; OligoZip assembler, one of the Softberry entries in the Assemblathon http://bit.ly/lenQ3G #ABPH11 <a href="http://bit.ly/lenQ3G">http://bit.ly/lenQ3G</a><br />
<em>pathogenomenick</em>: Victor is using a gene distance measurement to cluster &quot;pathogenic&quot; and &quot;non-pathogenic&quot; E. coli. But what is a pathogen? #ABPH11<br />
<em>jennifergardy</em>: Exons at #ABPH11 !!! What are these things? What is this &quot;intron&quot; you speak of? #iprefermicrobialgenestructures<br />
<em>fionabrinkman</em>: Too hungry to properly tweet #ABPH11. Fire alarm went just as I got to breakfast. Like some depend on coffee/tea, I need a big breaky!<br />
<em>pathogenomenick</em>: Woops, next presenter started his presentation by turning off the laptop. #ABPH11<br />
<em>AntipodeanCharm</em>: When computers are in presentation mode, eg in PowerPoint, the OS should shut up and stop sending alerts to the screen #ABPH11<br />
<em>pathogenomenick</em>: RT @AntipodeanCharm: When computers are in presentation mode, eg in PowerPoint, the OS should shut up and stop sending alerts to the screen #ABPH11<br />
<em>GuillaumeMeric</em>: RT @pathogenomenick: Victor is using a gene distance measurement to cluster &quot;pathogenic&quot; and &quot;non-pathogenic&quot; E. coli. But what is a pathogen? #ABPH11<br />
<em>GuillaumeMeric</em>: Some pretty interesting talks today at the Applied Bioinformatics &amp; Public Health Microbiology conference http://goo.gl/h5lI5 follow #ABPH11 <a href="http://goo.gl/h5lI5">http://goo.gl/h5lI5</a><br />
<em>aunderwo</em>: Guillermo Lopez-Campos: Presents work that automatically extracts primers and probes from papers http://1.usa.gov/kFTCjl #ABPH11 <a href="http://1.usa.gov/kFTCjl">http://1.usa.gov/kFTCjl</a><br />
<em>aunderwo</em>: RT @pathogenomenick: Victor is using a gene distance measurement to cluster &quot;pathogenic&quot; and &quot;non-pathogenic&quot; E. coli. But what is a pathogen? #ABPH11<br />
<em>AntipodeanCharm</em>: RT @assemblathon: For great coverage of the #ABPH11 conference, follow @pathogenomenick. He's dropping some tasty morsels of NGS information today.<br />
<em>jennifergardy</em>: #ABPH11 speaker missed big issue in biomedical informatics:lab samples and epi data often belong to separate agencies who can't share easily<br />
<em>aunderwo</em>: Call for a MIAMI style standard for clinical records to make clinical informatrics more interoperable #ABPH11<br />
<em>AntipodeanCharm</em>: &quot;Doctors should try to think in an organized way&quot; G. Lopez-Campos #ABPH11<br />
<em>pathogenomenick</em>: Now up - @jacarrico is talking about linked data for microbial typing. Proliferation of sequence-based typing methods &amp; databases. #ABPH11<br />
<em>pathogenomenick</em>: For Staph we have MLST.net, ccrbTyping.net, SpaServer, MLVAplus.net - want to connect these together #ABPH11<br />
<em>aunderwo</em>: Joo Andr Carrio: Collating data from MLST, spa server, MLVA websites is for the moment manual - his dream to automate this #ABPH11<br />
<em>aunderwo</em>: RT @jennifergardy: #ABPH11 speaker missed big issue in biomedical informatics:lab samples and epi data often belong to separate agencies who can't share easily<br />
<em>fionabrinkman</em>: @jacarrico talking about molecular typing - many DBs not interconnected with little machine readable interfaces  #ABPH11<br />
<em>pathogenomenick</em>: Proposing semantic web principles for connecting molecular typing dbs. See linking open data cloud diag http://bit.ly/fTXGJP  #ABPH11 <a href="http://bit.ly/fTXGJP">http://bit.ly/fTXGJP</a><br />
<em>RebeccaGladston</em>: #abph11 so true... A plethora of standards defeats the purpose of standardisation!<br />
<em>fionabrinkman</em>: @jacarrico: 'I have a Dream' like MLK - since realized integrated molecular typing is a 'dream' about a social problem #ABPH11<br />
<em>AntipodeanCharm</em>: @jacarrico presented an amazing overview of linked data on the web. I hope he tweets that slide, hint hint. #ABPH11<br />
<em>AntipodeanCharm</em>: RT @pathogenomenick: Proposing semantic web principles for connecting molecular typing dbs. See linking open data cloud diag http://bit.ly/fTXGJP  #ABPH11 <a href="http://bit.ly/fTXGJP">http://bit.ly/fTXGJP</a><br />
<em>aunderwo</em>: When linking data the problem (even within the same organisation - speaking from experience!) is the identifiers #ABPH11<br />
<em>AntipodeanCharm</em>: Roll for surprise. @jacarrico just went from web standards for molecular typing to ontology. Good.  #ABPH11<br />
<em>pathogenomenick</em>: .@jacarrico got his students to define an ontology for sequence-based typing results 'TypOn': available here http://bit.ly/lsD2w0 #ABPH11 <a href="http://bit.ly/lsD2w0">http://bit.ly/lsD2w0</a><br />
<em>jennifergardy</em>: TyPon - an ontology for molecular epi typing methods - from @jacarrico at #ABPH11 phyloviz.net/typon Great idea and great talk.<br />
<em>avilella</em>: Anybody hear any numbers about MiSeq price per run? Or per base? #ABPH11 tweeps @jennifergardy @fionabrinkman @aunderwo @jacarrico @avilella<br />
<em>pathogenomenick</em>: You can link these data to RDF. Convert with: Triplify, Apache OODT, Virtuoso Sponger. This is cool, need to get into it more. #ABPH11<br />
<em>pathogenomenick</em>: Webservices: use RDF repositories such as Triple stores: JENA, OpenLink Virtuoso, Mulgara, Bigdata, Sesame. Query with SPARQL. #ABPH11<br />
Paper : http://bit.ly/jm7XDT Ontology: http://bit.ly/jzApXB #ABPH11 <a href="http://bit.ly/jm7XDT">http://bit.ly/jm7XDT</a><a href="http://bit.ly/jzApXB">http://bit.ly/jzApXB</a><br />
<em>aunderwo</em>: Joo Andr Carrio: An ontology and REST API for microbial typing<br />
Paper : http://bit.ly/jm7XDT Ontology: http://bit.ly/jzApXB #ABPH11 <a href="http://bit.ly/jm7XDT">http://bit.ly/jm7XDT</a><a href="http://bit.ly/jzApXB">http://bit.ly/jzApXB</a><br />
<em>pathogenomenick</em>: RT @aunderwo: Joo Andr Carrio: An ontology and REST API for microbial typing<br />
<em>pathogenomenick</em>: Has developed a RESTful MLST web interface. This is great. We just need it for next-gen now. #ABPH11<br />
#ABPH11 <a href="http://rest.phyloviz.net/webui/">http://rest.phyloviz.net/webui/</a><br />
<em>aunderwo</em>: Joo Andr Carrio: RESTful MLST web interface http://rest.phyloviz.net/webui/<br />
<em>pathogenomenick</em>: Developed data visualisation software called Phyloviz: http://bit.ly/isNJgj handles ST data, SNP data, looks pretty #ABPH11 <a href="http://bit.ly/isNJgj">http://bit.ly/isNJgj</a><br />
<em>pathogenomenick</em>: .@jacarrico makes a compelling case for open data in molecular typing. What a shame it is not embraced by wider community #ABPH11<br />
<em>marina_manrique</em>: Nick Loman @pathogenomenick starts the pipeline session. xBASE-NG A web interface for rapid analysis of bacterial genomes #ABPH11<br />
<em>aunderwo</em>: Nick Loman: Web interface for WGS analysis http://ng.xbase.ac.uk/my/ #ABPH11 <a href="http://ng.xbase.ac.uk/my/">http://ng.xbase.ac.uk/my/</a><br />
<em>pjacock</em>: RT @marina_manrique: Nick Loman @pathogenomenick starts the pipeline session. xBASE-NG A web interface for rapid analysis of bacterial genomes #ABPH11<br />
<em>aunderwo</em>: Nick Loman: Use Illumina sequence to correct homopolymeric tracts in 454 scaffolds #ABPH11<br />
<em>jacarrico</em>: Nick Loman - illumina corrected 133 putative erros in 454 assembly #ABPH11<br />
<em>marina_manrique</em>: Once more: the importance of hybrid assemblies (in this case #454 &amp; #illumina) for correcting seq errors @pathogenomenick #ABPH11 #ngs<br />
<em>fionabrinkman</em>: Now @pathogenomenick talking about Pseudomonas genome sequencing and xBASE-NG #ABPH11<br />
<em>BigSporkDaggler</em>: 'Xbase ng main selling point is not accuracy&quot; #couldhavebeenphrasedbetter #ABPH11<br />
<em>aunderwo</em>: Nick Loman: Best to aggressively 'trim' SNPs after mapping reads-better to have fewer false positives and more false negatives #ABPH11<br />
<em>jacarrico</em>: Nick Loman Alignment software can introduce SNP errors since it is optimized for speed, not accuracy #ABPH11<br />
<em>fionabrinkman</em>: .@pathogenomenick says their annotation pipeline not geared for accuracy but rather providing info - I like this when exploring data #ABPH11<br />
<em>jacarrico</em>: Keith jolley -BIGsDB - http://t.co/3yqfn2o #ABPH11 <a href="http://t.co/3yqfn2o">http://t.co/3yqfn2o</a><br />
<em>fionabrinkman</em>: .@pathogenomenick says Ion Torrent data released today re current E. coli outbreak #ABPH11 #ecoli<br />
<em>jacarrico</em>: BIGsDB offers the possibility of linking a published dataset to the DB. #ABPH11<br />
<em>jacarrico</em>: BIGsDB - can store to completely heterogeneous database and integrate it #ABPH11<br />
<em>jacarrico</em>: BIGsDB can compare several geno es and create allele profiles like a super MLST schema #ABPH11<br />
<em>pathogenomenick</em>: Keith Jolley talking BiGsDB: BLAST-based genome comparator for whole-genomes. Extends MLST idea to large numbers of genes.  #ABPH11<br />
<em>pathogenomenick</em>: 'rMLST' tree - universal typing from domain to strain. Gives good speciation within a genus. Subspeciation within S. pneumo #ABPH11<br />
<em>aunderwo</em>: Keith Jolley: BIGSdb genome comprator allows comparison of genomes by MLST but with 1000s of loci not 7 #ABPH11<br />
<em>fionabrinkman</em>: #ABPH11 @pathogenomenick discussed problem of repeats etc for SNP calling. He great to talk to re SNP analysis. See also http://XBASE.AC.UK <a href="http://XBASE.AC.UK">http://XBASE.AC.UK</a><br />
<em>pathogenomenick</em>: It's @marina_manrique talking about her NG7 genome annotation system #ABPH11<br />
<em>GigaScience</em>: Nice linking of data http://ow.ly/58mpN RT @jacarrico BIGsDB offers the possibility of linking a published dataset to the DB. #ABPH11 <a href="http://ow.ly/58mpN">http://ow.ly/58mpN</a><br />
<em>aunderwo</em>: Marina Manrique: An annotation pipeline for NGS genome data http://www.era7bioinformatics.com/en/prokaryote_genome_annotation.html #ABPH11 <a href="http://www.era7bioinformatics.com/en/prokaryote_genome_annotation.html">http://www.era7bioinformatics.com/en/prokaryote_genome_annotation.html</a><br />
<em>jacarrico</em>: www.ohnosequences.com - great name for a sequence assembler based on protein similarity #ABPH11<br />
<em>AntipodeanCharm</em>: Do we need a meta-analyzer to pool and view data from the various auto annotators? We OCD types could never just stick to one. #ABPH11<br />
<em>pablopareja</em>: RT @aunderwo: Marina Manrique: An annotation pipeline for NGS genome data http://www.era7bioinformatics.com/en/prokaryote_genome_annotation.html #ABPH11 <a href="http://www.era7bioinformatics.com/en/prokaryote_genome_annotation.html">http://www.era7bioinformatics.com/en/prokaryote_genome_annotation.html</a><br />
<em>aunderwo</em>: Marina Manrique: Using cloud computing annotate 1 genome in 2 hours, annotate 100 genomes in 2 hours! #ABPH11<br />
<em>jacarrico</em>: Cloud computing in Amazon Web services for assembly and pay-per-use approach of computing power. Very nice #ABPH11<br />
<em>pathogenomenick</em>: Marina's annotation pipeline can run on EC2 http://bit.ly/kqhrZW  #ABPH11 <a href="http://bit.ly/kqhrZW">http://bit.ly/kqhrZW</a><br />
<em>aunderwo</em>: www.ohnosequences.com nice looking website - very minimalist, low on clutter #ABPH11<br />
<em>jacarrico</em>: They developed bio4j graph database http://t.co/mxV0ldu as well! That should allow sparql queries :) #ABPH11 <a href="http://t.co/mxV0ldu">http://t.co/mxV0ldu</a><br />
<em>pathogenomenick</em>: RT @jacarrico: They developed bio4j graph database http://t.co/mxV0ldu as well! That should allow sparql queries :) #ABPH11 <a href="http://t.co/mxV0ldu">http://t.co/mxV0ldu</a><br />
<em>ldalcaraz</em>: RT @fionabrinkman: .@pathogenomenick says Ion Torrent data released today re current Europe's E. coli outbreak #ABPH11 #ecoli<br />
<em>32nm</em>: RT @jacarrico: They developed bio4j graph database http://t.co/mxV0ldu as well! That should allow sparql queries :) #ABPH11 <a href="http://t.co/mxV0ldu">http://t.co/mxV0ldu</a><br />
<em>jennifergardy</em>: This AM's #ABPH11 talks from @jacarrico @pathogenomenick and @aunderwo suggest a positive correlation between tweeting &amp; giving good talks.<br />
<em>pathogenomenick</em>: .@aunderwo is telling us about his local pipeline for orthologue prediction and reference-guided annotation. Loads into SQLdb.  #ABPH11<br />
<em>pathogenomenick</em>: .@aunderwo wants to integrate his pipeline into Galaxy. We want to do that too #usegalaxy #ABPH11<br />
<em>jacarrico</em>: galaxy.psu.edu - web Gui creation - sugestion from @aunderwo #ABPH11<br />
<em>pathogenomenick</em>: Hmm, this next talk is comparing various pipelines including FgenesB, IMG/ER, xBASE, RAST, IGS. So bit nervous. #ABPH11<br />
<em>marina_manrique</em>: Rebecca Jones comparing annotations pipelines: FgenesB IMG/ER xBASE RAST and IGS on Salmonella data #ABPH11 #ngs<br />
<em>jacarrico</em>: Lots of diferent and interestign aproaches to genome assembly and annotation pipelines so far and all full of nice features #ABPH11<br />
<em>pathogenomenick</em>: She's correctly stated our pipeline is the quickest and is easy to use, but probably won't do so well with quality #ABPH11<br />
<em>jacarrico</em>: I've quit taking notes since the tag #ABPH11 is enough to get what I want :) #gobiotwitters<br />
<em>marina_manrique</em>: Small CDS remains a challenge in bacterial genome annotation (Rebecca Jones) #ABPH11<br />
<em>aunderwo</em>: RT @jacarrico: I've quit taking notes since the tag #ABPH11 is enough to get what I want :) #gobiotwitters<br />
<em>pathogenomenick</em>: Rebecca has pointed out my pipeline can produce large overlapping genes. This is an effect of Glimmer, I should tidy it up . #ABPH11<br />
<em>1VQ9</em>: RT @jacarrico: They developed bio4j graph database http://t.co/mxV0ldu as well! That should allow sparql queries :) #ABPH11 <a href="http://t.co/mxV0ldu">http://t.co/mxV0ldu</a><br />
<em>fionabrinkman</em>: .@aunderwo mentioned how HiSeq produces so much data for microbial seq that can simply remove low quality reads &amp; still have lots #ABPH11<br />
<em>fionabrinkman</em>: Like the idea of having a pipeline roundtable discussion to get efficiently at issues re seq analysis pipelines #ABPH11<br />
<em>jacarrico</em>: All anotation is a lie, or at least an approximation to the truth - Nick thanks for saying that out loud! #ABPH11<br />
<em>eduardopareja</em>: Marina Manrique from Era7bioinformatics talking about NG7 http://bit.ly/j3wNAO in Cambridge #ABPH11 in a #bioinformatics meeting at Sanger <a href="http://bit.ly/j3wNAO">http://bit.ly/j3wNAO</a><br />
<em>AntipodeanCharm</em>: Automated annotation. Used think it was speed vs accuracy - pick one. Not so sure now. But people r reluctant 2 review gene by gene #ABPH11<br />
<em>AntipodeanCharm</em>: RT @jacarrico: All anotation is a lie, or at least an approximation to the truth - Nick thanks for saying that out loud! #ABPH11<br />
<em>AntipodeanCharm</em>: ditto. @jacarrico: I've quit taking notes since the tag #ABPH11 is enough to get what I want :) #gobiotwitters<br />
<em>aunderwo</em>: Rebecca Jones: Comparison of annotation pipeline - FgenesB annotates addition features and comparable number of CDSs #ABPH11<br />
<em>aunderwo</em>: RT @fionabrinkman: Like the idea of having a pipeline roundtable discussion to get efficiently at issues re seq analysis pipelines #ABPH11<br />
<em>jacarrico</em>: Rohlf Kass on WGS 173(?) Ecoli strains - very actual given the #EHEC outbreak #ABPH11<br />
<em>jacarrico</em>: Dynamic threshold for strain/clone differentiation - interesting approach that I support due to increasing sampling ! #ABPH11<br />
<em>jacarrico</em>: Plot of gene diversity show that MLST genes in ecoli have low variabilty...as the great majority of genes  #ABPH11<br />
<em>aunderwo</em>: Rolf Kass: Described method to find gene families in E.coli and then find those gene families that show high/low conservation #ABPH11<br />
3<br />
<em>marina_manrique</em>: @lexnederbragt Have you come to the Applied Bioinformatics &amp; Public Health conference at Cambridge? #ABPH11<br />
<em>jacarrico</em>: Selection vs Drift : the eternal question, how to/ can we effectively measure it ? #ABPH11<br />
<em>jacarrico</em>: Rebecca Gladstone - genetic diversity of S pneumo serotype 6C #ABPH11<br />
<em>jacarrico</em>: Sero 6C - prevalent in carriage, can cause diseae, not in vacc, cross protection from 6A in 13V vaccine #ABPH11<br />
<em>jacarrico</em>: WGS of 6C isolates  ( diferent STs present) strains and analysis using split decomp show some recognizable clusters #ABPH11<br />
<em>aunderwo</em>: Rebecca Gladstone: The number of genes NOT shared between strains correlates well with their clonal lineage as defined by MLST #ABPH11<br />
<em>GuillaumeMeric</em>: RT @jacarrico: Plot of gene diversity show that MLST genes in ecoli have low variabilty...as the great majority of genes  #ABPH11<br />
<em>jacarrico</em>: 6C isolates were from 5 distinct lineage..clonal expansion is contributing for spread #ABPH11<br />
<em>GuillaumeMeric</em>: RT @jacarrico: Selection vs Drift : the eternal question, how to/ can we effectively measure it ? #ABPH11<br />
<em>fionabrinkman</em>: We need genomic epi on this new E. coli outbreak strain EAEC/EHEC asap! http://goo.gl/7L1gc  http://goo.gl/a10TO #ecoli #ABPH11 <a href="http://goo.gl/7L1gc">http://goo.gl/7L1gc</a><a href="http://goo.gl/a10TO">http://goo.gl/a10TO</a><br />
<em>ldalcaraz</em>: RT @fionabrinkman: We need genomic epi on this new E. coli outbreak strain EAEC/EHEC asap! http://goo.gl/7L1gc  http://goo.gl/a10TO #ecoli #ABPH11 <a href="http://goo.gl/7L1gc">http://goo.gl/7L1gc</a><a href="http://goo.gl/a10TO">http://goo.gl/a10TO</a><br />
<em>BGI_Events</em>: RT @fionabrinkman: .@pathogenomenick says Ion Torrent data released today re current E. coli outbreak #ABPH11 #ecoli<br />
<em>sehrrot</em>: RT @fionabrinkman: .@aunderwo mentioned how HiSeq produces so much data for microbial seq that can simply remove low quality reads &amp; still have lots #ABPH11<br />
<em>sehrrot</em>: RT @fionabrinkman: We need genomic epi on this new E. coli outbreak strain EAEC/EHEC asap! http://goo.gl/7L1gc  http://goo.gl/a10TO #ecoli #ABPH11 <a href="http://goo.gl/7L1gc">http://goo.gl/7L1gc</a><a href="http://goo.gl/a10TO">http://goo.gl/a10TO</a><br />
<em>jennifergardy</em>: Fun lunch at #ABPH11 with @fionabrinkman and Martin Maiden discussing the lack of pathogenic archaea #sciencefunisdifferentfun<br />
<em>mattdotvaughn</em>: RT @aunderwo: When linking data the problem (even within the same organisation - speaking from experience!) is the identifiers #ABPH11<br />
<em>pathogenomenick</em>: Ulbrich Dobrindt is now going to talk about E. coli outbreaks - will he talk about German outbreak I wonder? #ABPH11<br />
<em>aunderwo</em>: Ulbrich Dobrindt: Talking about ExPEC E.coli - Can be asymptomatic commensals but once leave gut cause disease #ABPH11<br />
<em>fionabrinkman</em>: Ulbrich Dobrindt talking about E. coli genome plasticity - including impacts on diagnostics #ecoli #ABPH11<br />
<em>pathogenomenick</em>: Ulrich trying to classify clinical manifestations of E. coli (ExPEC, EHEC etc.) based on clustering of COGs. Works well, #ABPH11<br />
<em>aunderwo</em>: Ulrich Dobrindt: Single linkage clustering based on presence/absence of COGs groups pathotypes of E.coli #ABPH11<br />
<em>pathogenomenick</em>: Ulbrich talks about German E. coli outbreak: &gt;1400 suspected cases, 470 confirmed HUS, 15 deaths #ABPH11<br />
<em>jacarrico</em>: Ulrich now talking about the #EHEC O104:H4 outbreak...really fresh data..the hybrid was confirmed some days ago! #ABPH11<br />
<em>aunderwo</em>: Ulrich Dobrindt: ExPEC E.coli strains have clusters of virulence markers that may be markers for the ExPEC type #ABPH11<br />
<em>fionabrinkman</em>: Dobrindt: Ureopathogenic E. coli can have genes from EHEC etc. So mixed virulence repertoires occur #ecoli #ABPH11<br />
<em>pathogenomenick</em>: It's difficult to distinguish commensals and ExPEC by MLST alone. Can WGS help? #ABPH11<br />
<em>GuillaumeMeric</em>: RT @fionabrinkman: Dobrindt: Ureopathogenic E. coli can have genes from EHEC etc. So mixed virulence repertoires occur #ecoli #ABPH11<br />
<em>GuillaumeMeric</em>: RT @pathogenomenick: It's difficult to distinguish commensals and ExPEC by MLST alone. Can WGS help? #ABPH11<br />
<em>GuillaumeMeric</em>: RT @jacarrico: Ulrich now talking about the #EHEC O104:H4 outbreak...really fresh data..the hybrid was confirmed some days ago! #ABPH11<br />
<em>jacarrico</em>: Combination of MLST + MLVA to increase resolution of typing of e.coli #ABPH11<br />
<em>aunderwo</em>: Ulrich Dobrindt: Clonal complexes containing ExPEC also have non-ExPEC strains.In contract IPEC complexes contain only IPEC isolates #ABPH11<br />
<em>GuillaumeMeric</em>: Dobrindt papers are really really nice. It's nice to follow his talk via #ABPH11, thanks twitters<br />
<em>GuillaumeMeric</em>: RT @jacarrico: Combination of MLST + MLVA to increase resolution of typing of e.coli #ABPH11<br />
<em>fionabrinkman</em>: Dobrindt refers to E. coli outbreak strain as O104:H4 and confirms it contains shiga toxin and EAEC genes #ecoli #ABPH11<br />
<em>jacarrico</em>: Nice trees very PHYLOViZ like! #blatantpublicitystunt #ABPH11<br />
<em>GuillaumeMeric</em>: RT @fionabrinkman: Dobrindt refers to E. coli outbreak strain as O104:H4 and confirms it contains shiga toxin and EAEC genes #ecoli #ABPH11<br />
<em>RebeccaGladston</em>: #abph11 a great example of artificial flora for protective purposes<br />
<em>pathogenomenick</em>: Talking about long-term colonisation by asymptomatic E. coli being protective http://bit.ly/jiRPXS #ABPH11 <a href="http://bit.ly/jiRPXS">http://bit.ly/jiRPXS</a><br />
<em>GuillaumeMeric</em>: RT @pathogenomenick: Talking about long-term colonisation by asymptomatic E. coli being protective http://bit.ly/jiRPXS #ABPH11 <a href="http://bit.ly/jiRPXS">http://bit.ly/jiRPXS</a><br />
<em>pathogenomenick</em>: http://1.usa.gov/lU2Bv2 Host Imprints on Bacterial GenomesRapid, Divergent Evolution in Individual Patients #ABPH11 <a href="http://1.usa.gov/lU2Bv2">http://1.usa.gov/lU2Bv2</a><br />
<em>AntipodeanCharm</em>: Some analogies to E coli Nissle being protective against IBD, I guess. #ABPH11<br />
<em>attilacsordas</em>: RT @pathogenomenick: http://1.usa.gov/lU2Bv2 Host Imprints on Bacterial GenomesRapid, Divergent Evolution in Individual Patients #ABPH11 <a href="http://1.usa.gov/lU2Bv2">http://1.usa.gov/lU2Bv2</a><br />
<em>cgorman</em>: RT @fionabrinkman: Dobrindt refers to E. coli outbreak strain as O104:H4 and confirms it contains shiga toxin and EAEC genes #ecoli #ABPH11<br />
<em>pathogenomenick</em>: When tracking SNPs in vivo they can &quot;come and go&quot; - impact for tracking outbreaks. Homoplasy or mixed population? #ABPH11<br />
<em>jacarrico</em>: SNPs can come and go over time..lots of implications for typing. are fast clock targets usefull besides outbreak detection at all @ #ABPH11<br />
<em>jacarrico</em>: Ulrich @ the stand !  #IPadphototesting #ABPH11  http://t.co/2A6Yd6g <a href="http://t.co/2A6Yd6g">http://t.co/2A6Yd6g</a><br />
<em>aunderwo</em>: Interesting papers by Ubrindt re array based typing of Enterobacteria http://1.usa.gov/mGobnB http://1.usa.gov/l6nMUv #ABPH11 <a href="http://1.usa.gov/mGobnB">http://1.usa.gov/mGobnB</a><a href="http://1.usa.gov/l6nMUv">http://1.usa.gov/l6nMUv</a><br />
<em>jacarrico</em>: Actually it is amazing how fast we got data on the #EHEC outbreak from the first typing to sequences available. #RTgenomics #ftw #ABPH11<br />
<em>kshameer</em>: RT @pathogenomenick: When tracking SNPs in vivo they can &quot;come and go&quot; - impact for tracking outbreaks. Homoplasy or mixed population? #ABPH11<br />
<em>jacarrico</em>: Curious about next talk: whole genome sequences from clinical swabs Chlamydia trachomatis by Helena Seth-Smith #ABPH11<br />
<em>jacarrico</em>: MOMP serotyping phylogeny reconstruction misleading due to recombination #ABPH11<br />
<em>pathogenomenick</em>: Chlamydia doesn't have any significant repeats in it and so assembles very nicely #ABPH11 - but getting DNA is an issue<br />
<em>marina_manrique</em>: Helena Seth-Smith on getting complete Chlamydia trachomatis genomes from clinical swabs at #ABPH11<br />
<em>jacarrico</em>: Only 15+ MOMP serotypes ..10 already sequenced... #ABPH11<br />
<em>fionabrinkman</em>: Helena Seth-Smith on Chlamydia trachomatis genomes from clinical swabs: really needed cause so hard to culture #ABPH11<br />
<em>GuillaumeMeric</em>: Thanks for sharing tweets about #ABPH11 conference! @fionabrinkman @pathogenomenick @jacarrico<br />
<em>fionabrinkman</em>: @pathogenomenick @jennifergardy @jacarrico Helena mentioned Chamydia culturing is stochastic :) #baysianstochasticism #ABPH11<br />
<em>pathogenomenick</em>: Through immunomagnetic separation and MDA can sequence chlamydia direct from patient samples without culture #ABPH11<br />
<em>marina_manrique</em>: They capture C. trachomatis with specific antibodies to enrich the sample in C. trachomatis DNA, I like it! :) #ABPH11<br />
<em>aunderwo</em>: Helena Seth Smith: Purification of Chlamydia from clinical swabs and amplification by MDA allows genome assembly but uneven coverage #ABPH11<br />
<em>marina_manrique</em>: RT @aunderwo: Helena Seth Smith: Purification of Chlamydia from clinical swabs and amplification by MDA allows genome assembly but uneven coverage #ABPH11<br />
<em>jacarrico</em>: Only 20% success rate on the WGS from swabs: problems from transport to assembly (repeats) #ABPH11<br />
<em>jacarrico</em>: Jacqueline Chan - Phylogenetic definition of species for Acinetobacter genus #ABPH11<br />
<em>pathogenomenick</em>: Jackie Chan from our lab is talking about a phylogenomic definition of species - looking at Acinetobacter #ABPH11<br />
<em>jacarrico</em>: Single gene trees (16s and rpoO) show different clusters #ABPH11<br />
<em>aunderwo</em>: Jacqueline Chan: rpoB tree produces more robust tree than 16S for Acinetobacter. Matches the whole genome SNP tree fairly well #ABPH11<br />
<em>jacarrico</em>: Genome fluidity z- Kislyuk et al,  2011 BMC genomics. Interesting concept. #ABPH11<br />
<em>aunderwo</em>: Jacqueline Chan: Genomic fluidity gives a measure of gene diversity that can be used to draw a tree - promising http://bit.ly/mx7SJ0 #ABPH11 <a href="http://bit.ly/mx7SJ0">http://bit.ly/mx7SJ0</a><br />
<em>jacarrico</em>: Cut-off value of 20% on single linkage (?) dendrogram of genome fluidity values seems to provide a good species definition #ABPH11<br />
<em>cgorman</em>: To all #ABPH11 conference attendees: Have you heard if new #ecoli  O104:H4 outbreak has any antibiotic resistance genes as well?<br />
<em>jacarrico</em>: Little gene content difference between Acinetobacter species #ABPH11<br />
<em>fionabrinkman</em>: Re Jackie Chan's talk: RT @jacarrico: Genome fluidity z- Kislyuk et al,  2011 BMC genomics. Interesting concept. #ABPH11<br />
<em>jacarrico</em>: Angela McCann - presenting data on a Salmonella agona outbreak #ABPH11<br />
<em>marina_manrique</em>: I've uploaded the slides of our talk at #ABPH11 on BG7 a bacterial genome annotation system designed for #ngs http://slidesha.re/iX1m7Y #in <a href="http://slidesha.re/iX1m7Y">http://slidesha.re/iX1m7Y</a><br />
<em>pathogenomenick</em>: Now Angela McCann from Cork is talking about a food-borne outbreak of Salmonella agona - Illumina sequencing #ABPH11<br />
<em>jacarrico</em>: PFGE couldn't discriminate the strains - 38 strains were sequenced using Illumina paired end reads #ABPH11<br />
<em>pathogenomenick</em>: They checked their SNP set was congruent between de novo assemblies (SOAPdenovo) and mapping alignments (SSAHA2) - good approach #ABPH11<br />
<em>marina_manrique</em>: RT @pathogenomenick: They checked their SNP set was congruent between de novo assemblies (SOAPdenovo) and mapping alignments (SSAHA2) - good approach #ABPH11<br />
<em>jacarrico</em>: ML phylogeny inferred from 3928 SNPs didn' match PFGE #ABPH11<br />
<em>aunderwo</em>: Angela McCann: Differences in Salmonella PFGE profiles may be caused to some extent by phage acquisition/loss #ABPH11<br />
<em>GuillaumeMeric</em>: RT @jacarrico: ML phylogeny inferred from 3928 SNPs didn' match PFGE #ABPH11<br />
<em>fionabrinkman</em>: RT @pathogenomenick: They checked their SNP set was congruent between de novo assemblies (SOAPdenovo) and mapping alignments (SSAHA2) - good approach #ABPH11<br />
<em>jacarrico</em>: Two different and distantly related clones (by SNPs) were responsible for the different outbreaks. PFGE misled by mobile elements ? #ABPH11<br />
<em>jennifergardy</em>: Too tired to tweet from #ABPH11 right now. NEED CHOCOBISCUIT IF I AM GOING TO CARRY ON. #getmeabiscuitnownownow<br />
<em>aunderwo</em>: RT @pathogenomenick: They checked their SNP set was congruent between de novo assemblies (SOAPdenovo) and mapping alignments (SSAHA2) - good approach #ABPH11<br />
<em>fionabrinkman</em>: Angela McCann: PFGE potentially missled by mobile elements  #ABPH11<br />
<em>fionabrinkman</em>: Martin Maiden at #ABPH11: &quot;the future is now&quot; Lots of great data for bioinformaticists to get their hands on #wringinghandswithglee<br />
<em>marina_manrique</em>: I wish there were more explanations about how the data was analysed in the talks (seq tech used, assembler, &quot;annotators&quot;...) #ABPH11<br />
<em>AntipodeanCharm</em>: In 2008, there was an epidemic of testing for C. difficule. # #ABPH11<br />
<em>AntipodeanCharm</em>: .@marina_manrique: Asking about individual pipelines would be a good survey to poll #ABPH11 attendees on.<br />
<em>pathogenomenick</em>: Tim Peto from Oxford talking now .. he's got 30,000 stool samples to play with in his study. Smelly. #ABPH11<br />
<em>marina_manrique</em>: @AntipodeanCharm maybe I could ask for that in the Q&amp;A box. it'd be great if we could get some figures of used tools and pipelines #ABPH11<br />
<em>jennifergardy</em>: Tim Peto is giving the best talk ever at #ABPH11 He is my new hero. And a poo hero. 30k stool samples.<br />
<em>AntipodeanCharm</em>: How much poo can stay in a hospital ward? #ABPH11<br />
<em>pathogenomenick</em>: &quot;Genomics test is just a test&quot; - requires usual challenges of any dx test: false pos / false neg rates. #ABPH11<br />
<em>pathogenomenick</em>: Challenge is we don't have a gold standard for transmission. Need to use epi data and genomics data to create a gold standard. #ABPH11<br />
<em>AntipodeanCharm</em>: What does the toothpaste genome look like? #ABPH11<br />
<em>aunderwo</em>: Tim Peto: uses very high coverage (x85) and strict SNP filter to reduce false positives &lt;1 error/100 strains #ABPH11<br />
<em>fionabrinkman</em>: Tim Peto: MLST not descriminatory enough for C. difficile transmission study in hospital. Epi &amp; genomes needed. #yesyetagain #ABPH11<br />
<em>jacarrico</em>: I can see several twitters open from up here! You know who you are...#ABPH11<br />
2<br />
<em>fionabrinkman</em>: Tim Peto: C. difficile in their hospital: ~2.6 SNPs/yr in individuals over time #ABPH11<br />
<em>aunderwo</em>: Tim Peto: analysis of c.difficile strains from a single patient estimates 2.6 SNPs/strain/year #ABPH11<br />
<em>pathogenomenick</em>: Best slide title ever &quot;12 picks from one nose&quot; #ABPH11<br />
<em>pathogenomenick</em>: RT @aunderwo: Tim Peto: analysis of c.difficile strains from a single patient estimates 2.6 SNPs/strain/year #ABPH11<br />
<em>fionabrinkman</em>: Battery about to die again. #ABPH11<br />
<em>aunderwo</em>: Tim Peto: taking multiple Staph aureus nasal samples gives a new meaning to nose picking! #ABPH11<br />
<em>marina_manrique</em>: Tim Peto - Whole genome sequencing would be better than MLST but be careful with false positives #ABPH11<br />
<em>marina_manrique</em>: Tim Peto - Whole genome sequencing would be better than MLST but be careful with false positives #ABPH11<br />
<em>jacarrico</em>: Tim Peto deserves a picture. He gave an interesting and highly motivated talk! #ABPH11  http://t.co/5oLMJ3u <a href="http://t.co/5oLMJ3u">http://t.co/5oLMJ3u</a><br />
<em>AntipodeanCharm</em>: I am astonished that Tim Peto's bioinfs were cross when he tested their pipeline with dupes. I suggest that my colleagues do that! #ABPH11<br />
<em>jennifergardy</em>: RT @jacarrico: Tim Peto deserves a picture. He gave an interesting and highly motivated talk! #ABPH11  http://t.co/5oLMJ3u <a href="http://t.co/5oLMJ3u">http://t.co/5oLMJ3u</a><br />
<em>pathogenomenick</em>: RT @jacarrico: Tim Peto deserves a picture. He gave an interesting and highly motivated talk! #ABPH11  http://t.co/5oLMJ3u <a href="http://t.co/5oLMJ3u">http://t.co/5oLMJ3u</a><br />
<em>jennifergardy</em>: Oooh, I just learned several interesting Luxembourg facts at #ABPH11 #tinybutinteresting<br />
<em>fionabrinkman</em>: RT @jacarrico: Tim Peto deserves a picture. He gave an interesting and highly motivated talk! #ABPH11  http://t.co/5oLMJ3u <a href="http://t.co/5oLMJ3u">http://t.co/5oLMJ3u</a><br />
<em>pathogenomenick</em>: Joel Mossong &amp; MRSA SNPs. He is very keen to mention Luxembourg is a real country (by Zappa test) - airline, beer, football team #ABPH11<br />
<em>pathogenomenick</em>: Mossong: Was alarmed when he got 1Gb files per each MRSA strain sequenced, compared with 7 bytes for MLST! #ABPH11<br />
<em>marina_manrique</em>: Great! some info about the kind of technology used in Jel Mossong talk at #ABPH11 Illumina 2x100bp 80x for MSRA genomes<br />
<em>jacarrico</em>: Interesting spa type vs SNP typing max parsimony tree comparison #ABPH11<br />
<em>aunderwo</em>: Joel Mossong: using 85x coverage illumina data could extract MLST profiles from 36/40 strains. Puzzled about missing 4? #ABPH11<br />
<em>aunderwo</em>: RT @pathogenomenick: Mossong: Was alarmed when he got 1Gb files per each MRSA strain sequenced, compared with 7 bytes for MLST! #ABPH11<br />
<em>marina_manrique</em>: Another idea I've liked at Jel Mossong talk: WGS data should not be limited to SNP analysis, Mobile elements also play a role! :) #ABPH11<br />
<em>marina_manrique</em>: And once more, the need of platforms to share typing data #ABPH11<br />
<em>jacarrico</em>: &quot;Public health institutes should acquire bioinformatics expertise&quot; Joel Mossong #ABPH11 Good point !#iwillneedabetterpayedjobsoon ;)<br />
<em>AntipodeanCharm</em>: RT @jacarrico: &quot;Public health institutes should acquire bioinformatics expertise&quot; Joel Mossong #ABPH11 Good point !#iwillneedabetterpayedjobsoon ;)<br />
<em>aunderwo</em>: RT @jacarrico: &quot;Public health institutes should acquire bioinformatics expertise&quot; Joel Mossong #ABPH11 Good point !#iwillneedabetterpayedjobsoon ;)<br />
<em>jacarrico</em>: Robert Stones - Phylogenomic analysis pipelines #ABPH11<br />
<em>pathogenomenick</em>: Robert Stones who was a co-author on cool FDA Salmonella outbreak paper recently in NEJM is doing a very brave live demo of his SW #ABPH11<br />
<em>jacarrico</em>: Showing a software demo of GeneSpace... #ABPH11<br />
<em>fionabrinkman</em>: Twitter and @AntipodeanCharm at #ABPH11 is my saviour (ty!). I now have iPhone juice!<br />
<em>aunderwo</em>: Talks aren't getting less interesting. Just finding that I'm running out of tweeting juice #ABPH11<br />
<em>jacarrico</em>: RT @aunderwo: Talks aren't getting less interesting. Just finding that I'm running out of tweeting juice #ABPH11<br />
<em>AntipodeanCharm</em>: Back-to-back, fascinating talks. Intense and educational but my brain is full.  My mind is ticking over mucho information &amp; ideas. #ABPH11<br />
<em>fionabrinkman</em>: At #ABPH11 I'm struck with how far we've come with genomic epi but how hard it will be to standardize<br />
<em>aunderwo</em>: Marcus Claesson: comparing 454 and Illumina data for classifying bacteria using 16S. 454 outperforms Illumina #ABPH11<br />
<em>aunderwo</em>: Marcus Claesson: Metagenomics  - long reads of 454 give better classification , more data from Illumina =&gt; more OTUs detected #ABPH11<br />
<em>fionabrinkman</em>: Claesson: 454 better vs Illumina for 16S seq (see http://goo.gl/u4CDD) but &gt;60bp Illumina reads really helps &amp; primer choice key #ABPH11 <a href="http://goo.gl/u4CDD)">http://goo.gl/u4CDD)</a><br />
<em>torstenseemann</em>: if we had concurrent sessions at #ABPH11 we'd be &quot;stereo-typing&quot;<br />
<em>aunderwo</em>: Supang Martin: Single Genome Sequencing of full length HIV pol reveals covariance of resistance SNPs  #ABPH11<br />
<em>lexnederbragt</em>: RT @marina_manrique: Once more: the importance of hybrid assemblies (in this case #454 &amp; #illumina) for correcting seq errors @pathogenomenick #ABPH11 #ngs<br />
<em>torstenseemann</em>: the problem with conferences like #ABPH11 is that i get too many new ideas, but with no time to play with them<br />
<em>pathogenomenick</em>: RT @torstenseemann: the problem with conferences like #ABPH11 is that i get too many new ideas, but with no time to play with them<br />
<em>aunderwo</em>: RT @torstenseemann: the problem with conferences like #ABPH11 is that i get too many new ideas, but with no time to play with them<br />
<em>pathogenomenick</em>: That's the end of the day 2 session. Great set of talks, I thought. Time for BEER. #ABPH11<br />
<em>jennifergardy</em>: And another day of #ABPH11 talks ends. Now the real work begins. Beer drinking and hanging out.<br />
<em>jacarrico</em>: @jennifergardy: And another day of #ABPH11 talks ends. Now the real work begins. Beer drinking and hanging out. #FTW<br />
<em>jennifergardy</em>: @jacarrico @pathogenomenick @aunderwo I think we can also capture all #ABPH11 tagged tweets with Twapper Keeper. Mebbe Nick can blog them.<br />
<em>jennifergardy</em>: Dinner at #ABPH11 - see if you can spot @pathogenomenick @fionabrinkman and @jacarrico at our table.  http://t.co/N08OgOW <a href="http://t.co/N08OgOW">http://t.co/N08OgOW</a><br />
<em>eduardopareja</em>: RT @marina_manrique: I've uploaded the slides of our talk at #ABPH11 on BG7 a bacterial genome annotation system designed for #ngs http://slidesha.re/iX1m7Y #in <a href="http://slidesha.re/iX1m7Y">http://slidesha.re/iX1m7Y</a><br />
<em>pjacock</em>: RT @pathogenomenick: .@aunderwo wants to integrate his pipeline into Galaxy. We want to do that too #usegalaxy #ABPH11<br />
<em>rodyredo</em>: RT @AntipodeanCharm: When computers are in presentation mode, eg in PowerPoint, the OS should shut up and stop sending alerts to the screen #ABPH11<br />
<em>pathogenomenick</em>: Brendan Wren on Newsnight talking E. coli - big cheer went up in Red Lion when he came on the telly  #ABPH11 http://yfrog.com/h4wrkfwj <a href="http://yfrog.com/h4wrkfwj">http://yfrog.com/h4wrkfwj</a><br />
<em>GuillaumeMeric</em>: RT @pathogenomenick: Brendan Wren on Newsnight talking E. coli - big cheer went up in Red Lion when he came on the telly  #ABPH11 http://yfrog.com/h4wrkfwj <a href="http://yfrog.com/h4wrkfwj">http://yfrog.com/h4wrkfwj</a><br />
<em>pathogenomenick</em>: Here's an annotation of that EHEC assembly from @marina_manrique and team http://bit.ly/kXZrsu #ABPH11 <a href="http://bit.ly/kXZrsu">http://bit.ly/kXZrsu</a><br />
<em>KamounLab</em>: RT @pathogenomenick: Here's an annotation of that EHEC assembly from @marina_manrique and team http://bit.ly/kXZrsu #ABPH11 <a href="http://bit.ly/kXZrsu">http://bit.ly/kXZrsu</a><br />
<em>AntipodeanCharm</em>: RT @pathogenomenick: Here's an annotation of that EHEC assembly from @marina_manrique and team http://bit.ly/kXZrsu #ABPH11 <a href="http://bit.ly/kXZrsu">http://bit.ly/kXZrsu</a><br />
<em>jacarrico</em>: Last day of #ABPH11 . Will start with Consumption-Junction by @jennifergardy  GO JENNiFER ! bring it on!<br />
<em>pathogenomenick</em>: Final day of #ABPH11 - Jen Gardy is doing her excellent talk about TB genomic epidem. Won't tweet as did it at ASM  http://bit.ly/mEJk1V <a href="http://bit.ly/mEJk1V">http://bit.ly/mEJk1V</a><br />
<em>aunderwo</em>: #ABPH11 tweeters - what questions should we try and get raised at the Q&amp;A session later?<br />
<em>fionabrinkman</em>: .@jennifergardy on integrating TB outbreak genomes with social network analysis (see our NEJM paper http://bit.ly/gxWcF0) Go Jenn! #ABPH11 <a href="http://bit.ly/gxWcF0)">http://bit.ly/gxWcF0)</a><br />
<em>aunderwo</em>: #ABPH11 Qs: I was thinking of &quot;How close are we to (what are the barriers to) near patient NGS analysis that impacts clinical management&quot;<br />
<em>fionabrinkman</em>: .@jennifergardy re genomic epidemiology: Its tricky. It's useful. Need good epi data &amp; understand disease transmission #ABPH11<br />
<em>GuillaumeMeric</em>: RT @pathogenomenick: Here's an annotation of that EHEC assembly from @marina_manrique and team http://bit.ly/kXZrsu #ABPH11 <a href="http://bit.ly/kXZrsu">http://bit.ly/kXZrsu</a><br />
<em>jacarrico</em>: Perfect world: just a few snps available to reconstruct a outbreak spanning a few weeks, but with TB they can be sick for years #ABPH11<br />
<em>jacarrico</em>: Use of SNPs to subtype inside of MIRU type and then rely on epi data #ABPH11<br />
<em>jacarrico</em>: Since they got 2 different clades they used it to prune the social network graph by removing impossible transmission links #ABPH11<br />
<em>jacarrico</em>: Without the epi data the genomics couldn't tell us anything. Simple and Elegant demo by overlaying social graph with SNP tree #ABPH11<br />
<em>jacarrico</em>: Jennifer Talking now of Kelowna TB Outbreak  #ABPH11<br />
<em>fionabrinkman</em>: .@jennifergardy mentions movie Outbreak influence #ABPH11 &amp; did TV sci shows. She really would be a great movie consultant-hint @wbpictures!<br />
<em>aunderwo</em>: Jennifer Gardy: For TB, genome data needs to be combined with comprehensive Epi data and social network analysis to get the true pic #ABPH11<br />
<em>aunderwo</em>: In chronic infections an individual may have a diverse pop of bugs- this may confuse trying to infer transmission tree from genomics #ABPH11<br />
<em>jacarrico</em>: Now  Brendan Wren Function Junction - moving from trees to forests #ABPH11<br />
<em>aunderwo</em>: Jennifer Grady:'Deep amplicon' sequencing of TB from a patient may resolve problems where phylo tree does not seem to match epi data #ABPH11<br />
<em>marina_manrique</em>: Great! Less trees and more functions! :) Brendan Wen at #ABPH11<br />
<em>pathogenomenick</em>: Brendan Wren is taking about &quot;going beyond the trees&quot; - &quot;eco-evo&quot; perspective on virulence. See this article http://bit.ly/mzbmDv #ABPH11 <a href="http://bit.ly/mzbmDv">http://bit.ly/mzbmDv</a><br />
<em>fionabrinkman</em>: @aunderwo It'd be great to frame Ques: What should we do collectively in the field to move genomic epi into clinic more in future #ABPH11<br />
<em>jacarrico</em>: Humans became acidentally a part of Capylobacter life-cycle #ABPH11<br />
<em>fionabrinkman</em>: Unfortunately I have to taxi pool to airport now. Will miss good talks darnit. Thank you #ABPH11 tweeters for great comments!<br />
<em>GuillaumeMeric</em>: RT @jacarrico: Humans became acidentally a part of Capylobacter life-cycle #ABPH11<br />
<em>aunderwo</em>: @fionabrinkman Agreed.Only together can we make this a reality- soooo much to investigate and test b4 we can have rigorous methods #ABPH11<br />
<em>pathogenomenick</em>: Good intro slide to Oliver Pybus' talk on RNA viruses http://bit.ly/mvqjCV #ABPH11 <a href="http://bit.ly/mvqjCV">http://bit.ly/mvqjCV</a><br />
<em>pathogenomenick</em>: Virologists are ahead in epidemiology compared to bacteriologists; 10s of 1000s of sequenced strains #ABPH11<br />
<em>jacarrico</em>: Oliver Pybus promised a cucumber free talk ! #ABPH11  http://t.co/RkOMXPk <a href="http://t.co/RkOMXPk">http://t.co/RkOMXPk</a><br />
<em>jacarrico</em>: 1 year of HIV evolution is equivalent of millions of years of mammalian evolution #ABPH11<br />
<em>aunderwo</em>: Oliver Pybus: Nice slides that help to clarify what coalescent theory is about - hope they get published somewhere #ABPH11<br />
<em>jacarrico</em>: Tools in virus phylogenetics: molecular clock, coalescent theory, phylogeography and adaptation analysis #ABPH11<br />
<em>RebeccaGladston</em>: #ABPH11 public health intervention responsible for highest HCV in the world in Eygpt, oops<br />
<em>jennifergardy</em>: Ollie Pybus is giving the clearest talk on phylodynamics ever at #ABPH11 He is a supersmartie and I follow his papers like a giant dork.<br />
<em>attilacsordas</em>: RT @pathogenomenick: Virologists are ahead in epidemiology compared to bacteriologists; 10s of 1000s of sequenced strains #ABPH11<br />
<em>aunderwo</em>: Oliver Pybus: Series of seasonal fluA samples over 10+ years =&gt; each year's strain pop is derived from one strain from previous year #ABPH11<br />
<em>jacarrico</em>: Nice google earth animation to illustrate the spread oh H1N1 #ABPH11<br />
<em>ALShahib</em>: #ABPH11<br />
<em>attilacsordas</em>: #ABPH11 'Unravelling transmission trees of infectious diseases by combining genetic and epidemiological data' might check that one<br />
<em>aunderwo</em>: Oliver Pybus:Adaptive substitutions in flu occur mainly in HA + NA (surface regions). Rate of change &lt; 4 H1N1. &lt; selective pressure? #ABPH11<br />
<em>RebeccaGladston</em>: #abph11 Dig out your old isolates from the freezers to improve understanding of th epi of current disease cases<br />
<em>aunderwo</em>: @ALShahib hey your tweets are now showing in the ABPH11 timeline!<br />
<em>pathogenomenick</em>: Oliver Pybus gave an excellent overview of phylodynamics/phylogenetics. Must read his papers. http://bit.ly/jK1mAg #ABPH11 <a href="http://bit.ly/jK1mAg">http://bit.ly/jK1mAg</a><br />
<em>jacarrico</em>: Use of Effective population size  or genetic diversity : presence of selection or not #ABPH11<br />
<em>RebeccaGladston</em>: A really great set of talks @ #abph11 over all the days, looking forward to see what discussions they provoke in q&amp;a! Welldone organisers!<br />
<em>jacarrico</em>: Marijn van Ballegooijen presenting nice statistical evaluation of transmission network reconstruction #ABPH11<br />
<em>jacarrico</em>: Outbreak correlates well wind direction! Nice example for heterogeneous data integratio ! #ABPH11 </p>
