---
layout: post
status: publish
published: true
title: 'All the Tweets from #MMGC 2012'
author:
  display_name: Nick Loman
  login: nick
  email: n.j.loman@bham.ac.uk
  url: http://xbase.bham.ac.uk
author_login: nick
author_email: n.j.loman@bham.ac.uk
author_url: http://xbase.bham.ac.uk
wordpress_id: 1167
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=1167
date: '2012-03-11 12:43:18 +0000'
date_gmt: '2012-03-11 12:43:18 +0000'
categories:
- Uncategorized
- High-throughput sequencing
- Genomic Epidemiology
- E. coli O104:H4
- Genomics
- Microbiology
tags:
- mmgc
- twitter
comments: []
---
<p><em><a href="http://www.mmgc.net">Rapid Next-Generation-Sequencing Conference for Public Health and Clinical Microbiology</a></em> was held in MÃ¼nster last week. The title is pretty self-explanatory. It is probably the first meeting since last summer's <em>E. coli</em> outbreak in Germany to allow epidemiologists, microbiologists and genomicists to get together and discuss the impact the rapid release of genome data had on that occasion, and how we might handle future outbreaks.</p>
<p>The scope is similar to the conference I help organise - <a href="https://registration.hinxton.wellcome.ac.uk/display_info.asp?id=227">Applied Bioinformatics & Public Health</a> - this will return in 2013, date to be announced soon.</p>
<p>I unfortunately couldn't attend this meeting, but through Twitter I did get a good flavour of what was discussed - many thanks for those who made the effort to tweet, it's really appreciated.</p>
<p>So for posterity I have captured the tweets here, with kind permission of @TimDallman and @jacarrico:</p>
<p><REPLACE></REPLACE><br />
<strong>Most popular tweets</strong><br />
(4) <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: MiSeq testing 2x 400 bp reads. 3.4Gb of data, &gt; 70% Q&gt;30 error rates at 400 cycles 2-4% #mmgc<br />
(4) <a href="https://twitter.com/#!/raqueltobes">raqueltobes</a>: OPGEN: A great NGS-technology independent method to evaluate NGS assemblers #mmgc<br />
(1) <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: #PacBio and Optical Mapping new alternatives to Sanger sequencing for bacterial genome closure #mmgc<br />
(1) <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Smith: original spec 1.5-2gb 2x150 soon to be 7gb 2x250 reads this summer #mmgc<br />
(1) <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Geoff Smith: MRSA Methicillin-resistant Staphylococcus aureus: SNP count alone cannot predic outbreak link #mmgc</p>
<p><strong>7 - 8pm EST</strong><br />
+1m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: @marina_manrique Hello Marina! I'll try my best I'll use the hashtag #MMGC .  to Tweeps:is anyone coming here ?<br />
+56m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: #mmgc <a rel="nofollow" href="(http://t.co/e7Bih1Y5">(http://t.co/e7Bih1Y5</a>) starting keynote:  Putting the genes into genomics: from MLST to BIGSdb Martin Maiden; U.Oxford, UK<br />
+59m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: @eduardopareja @timdallman @raqueltobes @era7bioinfo hello there fellow twiiterers ! Shall we use the #mmgc  tag?<br />
+5m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Keynote - Maiden, "Genes into genomics" #mmgc<br />
+8m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Maiden : How many loci: the ultimate question? The answer is 42! ;-). But the problem is always the question ;-)#hhgg #ftw #mmgc<br />
+10m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Maiden Timescales are important. What are the questions do we want to ask? diferent questions =&gt; diferent answers #mmgc<br />
+12m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Maiden : The Planet of the Bacteria (Stephen Jay Gould) Prokaryotes are responsible for the most diversity on the tree of life #mmgc<br />
+15m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Maiden - defining core and accessory genome - moving target with bacterial diversity - what is meaningful? #mmgc<br />
+16m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Maiden You can extract Barrels of SNPs, butnyou have to figure out the meaning of those SNPs #mmgc<br />
+18m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Maiden If there was a completely clonal population then we would be here since anyone could make meaningfull trees ver easily! #mmgc<br />
+19m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Maiden: bacteria are much more "sexy" than we thought. Maynard Smith Nature 1981 #mmgc<br />
+20m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Maiden : 7 lanes x 96 isolates can be done in a Illumina run #mmgc<br />
+23m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Maiden - BigsDb MLST for the whole genome #mmgc<br />
+23m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Maiden For core genes regions Illumina has great qual and they only got 12 errors from a large collection of assembled/draft genomes #mmgc<br />
+24m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Maiden: the 12 errors were all from manual editing ! #mmgc<br />
+25m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Maiden - BigsDB, sequence bin of assembled NGS BLAST against reference alleles which can then be defined into a scheme #mmgc<br />
+26m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Maiden : BIGSdb can create MLST-like schemas for any number of locus from contig data. #mmgc<br />
+27m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Maiden: these alleles can be annotated by the community help link to function #mmgc<br />
+28m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Maiden: with BIGSdb, you can put the genes back in to genomic because you van create specific schemas and anotate  then #mmgc<br />
+29m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Maiden: associating phenotype with genotype in the Neisseria #mmgc<br />
+31m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Maiden - defining species in Neisseria, 16s rRNA doesn't work very well #mmgc<br />
+33m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Maiden: Ribosomal MLST - 53 genes high resolution, present in all domains, encode complex protein structure - coevolution of genes #mmgc<br />
+34m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Maiden - whole genome means we have whole ribosomes - we can use 53 genes for universal bacterial rMLST<br />
 #mmgc<br />
+35m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Maiden rMLST - very diferent diversity between the 53 genes. Also gives more detailed structure than standard 16s rna  #mmgc<br />
+36m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Maiden - rMLST gives good species resolution as well as genus #mmgc<br />
+38m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Maiden : rplG gene reproduces the species structure of neisseria . Great to  field diagnostics #mmgc<br />
+41m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Maiden :The number of mlst allele plots have assimptots and the ST numbers keep increasing =&gt; most variation is due to recombination #mmgc<br />
+44m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Maiden: showing how MLST clonal complex  matters for meningococcal infection likelihood from carriage #mmgc<br />
+46m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Maiden: rMLST and MLST compliment each other shows relationships between clonal complexes #mmgc<br />
+47m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Maiden: in meningococcus they are not changing the capsule due to vaccine pressure. no ideia why!v #mmgc<br />
+57m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Correction : Maiden: rplF (not rplG) reproduces the species structure of neisseria #mmgc<br />
+3m <a href="https://twitter.com/#!/aunderwo">aunderwo</a>: @TimDallman @jacarrico Keep up the good work with the #mmgc tweets. Enjoying the coverage from here in the UK!<br />
+7m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Mellmann: German EHEC outbreak #mmgc<br />
+8m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: German EHEC 2011 outbreak NGS analysis Alexander Mellmann; University Mnster, Germany #mmgc<br />
+9m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: #mmgc Alexander Mellmann University Mnster German #EHEC 2011 Outbreak #NGS talk startring now...<br />
+12m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Mellman 10% of patients develop HUS #EHEC  #mmgc<br />
+13m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Mellman: 3842 cases 855 HUS in sprout implicated outbreak in 2001 Germany #mmgc<br />
+14m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: That should be 2011 #mmgc<br />
+17m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Mellman #EHEC  subtyping SLST, MLST,... #mmgc<br />
+17m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Mellman <a rel="nofollow" href="http://t.co/9p00UOUH">http://t.co/9p00UOUH</a> #EHEC #mmgc<br />
+19m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Mellmann: O104 stereotype outbreak strain was uncommon for EHEC though not novel #mmgc<br />
+25m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Mellmann: Ecoli O104:H4 had a aggregation pattern similar to EAEC #mmgc<br />
+27m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Mellmann: MST used 1144 core genome genes Mellmann et al Plos One 2011 #mmgc<br />
+29m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Mellmann: WGS showed "mosaic" structure of EHEC and EAEC virulence factors #mmgc<br />
+33m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Mellmann: can transduce K12 with stx2 phage via outbreak strain but cannot transduce 55989 EAEC by outbreak strain #mmgc<br />
+34m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Mellmann: Suggests hypothetical Stx2 O104 is progenitor of 55989. #mmgc<br />
+36m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: #mmgc Alexander Mellmann showing interesting results of the outbreak from a PLoS ONE paper about E coli O104:H4  <a rel="nofollow" href="http://t.co/dALL4TYP">http://t.co/dALL4TYP</a><br />
+38m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Mellmann: Found that there has been carriage of this strain in patients for several months post outbreak. Loss of virulence facotors? #mmgc<br />
+47m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Haiti Cholera 2010 outbreak NGS analysis<br />
Rene S. Hendriksen; Technical University Denmark, Lyngby, Denmark #mmgc<br />
+51m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Hendriksen: outbreak of cholera in Nepal 6 months before Haiti quake #mmgc<br />
+53m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Hendriksen: half a million cases by end of November in Haiti. Nepalese troops implicated - true or false? #mmgc<br />
+55m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Hendriksen: Chin et al NEJM 2011, 5 strains sequenced by PacBio.. Used other sequenced historical strains #mmgc<br />
+56m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: #PacBio system was used to sequence 5 strains , publishe in New England..<a rel="nofollow" href="http://www.nejm.org/doi/pdf/10.1056/NEJMoa1012928">http://www.nejm.org/doi/pdf/10.1056/NEJMoa1012928</a> #mmgc<br />
+59m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Hendriksen: Reimer et al EID 2011. 23 whole genomes, also matched PFGE profiles #mmgc<br />
+1m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Hendriksen: initial papers suggest strain from Bangladesh closest match by WGS #mmgc<br />
+1m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Reimer et al 0-2 SNP difference in core genimes among the 9 Haitian outbreak isolates #mmgc<br />
+3m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Hendriksen: WGS sequences of PFGE matched strain found strains with &lt;10 SNPS to Haitian strain #mmgc<br />
+6m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Hendriksen et al MBio 2011. Interesting strain collection with PFGE, MLVA and MLST characterization #mmgc<br />
+8m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Hendriksen: Strains from Nepal sequenced, Haiti strains cluster within them. One strain differs by only one SNP #mmgc<br />
+9m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Hendriksen: Very strong evidence that cholera was trasmitted from Nepal to Haiti from the paper #mmgc<br />
+10m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Hendriksen: PCR assay based on canonical SNPs is being developed #mmgc<br />
+13m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Hendriksen: need to build global WGST databases to increase the power to identify outbreaks worlwide and track sources #mmgc<br />
+14m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: One conclusion: "There is a need to build a global WGST database to increase  the power to identify global outbreaks in real time" #mmgc<br />
+16m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Peter Gerner-Smith comment: Nepalese soldiers weren't sampled due to political reasons #mmgc<br />
+19m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Dag Harmsen comment: traditional typing methods could be used to exclude but , much harder to prove similarity #mmgc<br />
+19m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Gharbia - Assembling the biome of emerging pathogens #mmgc<br />
+20m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Assembling the biome of emerging pathogenes, a public health perspective<br />
Saheer Gharbia, HPA, Colindale, United Kingdom #mmgc<br />
+23m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Gharbia C dif in persistant outbreaks :MDR hypervirulent, high toxin producers #mmgc<br />
+25m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Saheer Gharbia  speaking about C. difficile - HPA - Clostridium difficile <a rel="nofollow" href="http://t.co/qckifT7d">http://t.co/qckifT7d</a> #mmgc<br />
+33m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Gharbia Need to integrate new tools for these new tech to get better insights #mmgc<br />
+35m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Saheer Gharbia "We use proteomics to integrate genotype (WGS) to phenotype (gene expression)" #mmgc<br />
+44m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Saheer Gharbi: "Using proteomics to select the common expressed proteins shared among pathotypes"  #mmgc<br />
+46m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Saheer Gharbi: Metagenomics: geno-proteome derived signatures to predict new outbreaks #mmgc<br />
+48m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Tom Connor from the Sanger about the PacBio - Filling the gaps #mmgc<br />
+48m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Thomas Connor: Filling the gaps: using the #PacBio RS in microbial whole genome sequencing projects #mmgc<br />
+48m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Filling in the gaps: using the PacBio RS in microbial whole genome sequencing projects Thomas Connor Wellcome Trust Sanger Institute #mmgc<br />
+52m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Connor #PacBio SMRT cell v2 read length max: 15176bp compared to max=4527bp of v1 #mmgc<br />
+52m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Connor: Version 2 SMRT cell read lengths up to 15k bp up from 5k for v1 #mmgc<br />
+52m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Thomas Connor: Version 2 of SMRT cells from #PacBio has a maximun read lenght of 15176 bp !! #mmgc<br />
+55m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Connor: PacBio needs far greater coverage then other WGS methods to recover SNPs #mmgc<br />
+56m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Connor Pacbio has about 30% of errors in snp calls in their test (190x on Pacbio) #mmgc<br />
+57m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Connor Some genomes aren't amenable to sequencing using short read technologies #mmgc<br />
+58m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Connor Pacbio useful for rapid generating reference sequences, for outbreak situations when isolates are very similar #mmgc<br />
+59m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Connor: highlighting need for good reference when investigating outbreak I.e. close comparator to best call Polymorphisms. #mmgc<br />
+59m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Thomas Connor: #PacBio could be useful to generate new reference genomes for analyzing the isolates of an outbreak #mmgc<br />
+1m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Connor PacBio strobing runs canndamage the DNA and create errors #mmgc<br />
+3m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Connor: out of box CCS and strobing only gives as good de novo as illumina #mmgc<br />
+3m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Connor PacBio standard + CCS + strobing was still worst than Illumina assembly in number of contigs #mmgc<br />
+4m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: If you include illumina to hybrid assembly much better de novo #mmgc<br />
+5m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Connor hybrid illumina and pacbio assemblies get good results as illumina reads correct Pacbio larger reads #mmgc<br />
+6m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Connor: mentions image and iCorn for hybrid assemblies #mmgc<br />
+8m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Connor: Pacbio advantages: faster and less labor intensive, great scaffolding! #mmgc<br />
+8m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Connor Every SNP is sacred! (paraphrasing Monty Python #mmgc<br />
+10m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Thomas Connor: #PacBio  is faster and less labour intensive than Sanger and useful for scaffolding when many repetitive sequences   #mmgc<br />
+10m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Connor: If 2 strains have just a couple of snps check for phages and Mobile Genetic Elements #mmgc<br />
+10m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Connor: for outbreaks long read lengths allow Polymorphisms in repetitive and phage to be detected. May be needed when diversity low #mmgc<br />
+12m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: #PacBio and Optical Mapping new alternatives to Sanger sequencing for bacterial genome closure #mmgc<br />
+18m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Geoff Smith from Illumina: The MySeq System and applications to human disease #mmgc<br />
+18m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Next Geoff Smith from Illumina talking about MiSeq #mmgc<br />
+18m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: The MiSeq sytem and applications to human disease<br />
Geoff Smith; Illiumina, Chesterford Research Park, United Kingdom #mmgc<br />
+21m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Smith MiSeq : On board clustering, fast and On-board analysis #mmgc<br />
+23m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Smith MiSeq RFID based reagent and flow cell tracker #mmgc<br />
+25m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Smith: original spec 1.5-2gb 2x150 soon to be 7gb 2x250 reads this summer #mmgc<br />
+25m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: miSeq 1.5-2 G yield 2x150 read length 75% bases Q&gt;30 . Free upgrade next year for better stats #mmgc<br />
+26m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Smith: 80% reads &gt;Q30 #mmgc<br />
+27m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: MiSeq testing 2x 400 bp reads. 3.4Gb of data, &gt; 70% Q&gt;30 error rates at 400 cycles 2-4% #mmgc<br />
+29m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Smith: Some public health examples, detection, identification, antimicrobial susceptibility and epidemiology #mmgc<br />
+29m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Smith: use in pub health: Detection, Identification, Antibiotic susceptibiliy testing , epidemiology #mmgc<br />
+29m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Geoff Smith: MiSeq extending read lenght to 400 x 2 bp ( in research) #mmgc<br />
+32m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Smith : TB example from Niemman Plos ONE paper where a SNP predicted resistance #mmgc<br />
+44m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Geoff Smith: MRSA Methicillin-resistant Staphylococcus aureus: SNP count alone cannot predic outbreak link #mmgc<br />
+48m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Geoff Smith: MRSA Methicillin-resistant Staphylococcus aureus: Sequencing  possitive clones can detect a transmission event #mmgc<br />
+28m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: John Wain - MiSeq and Typhoid #mmgc<br />
+33m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Wain: Phenotypic data still needed to understand genotype, unknown resistance mechanisms in Paratyphi A #mmgc<br />
+35m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Wain: What SNPs to use for epidemiology and transmission how to get rid of homoplasy etc #mmgc<br />
+37m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Wain: After 1993 amount of diversity in S.Typhi decreased rapidly - how does this affect epidemiology studies? #mmgc<br />
+38m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: John Wain HPA: Selectable mutations can impact on the clustering of isolates of an outbreak #mmgc<br />
+44m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Wain: Case and Carrier 0 SNPs 3 SNPs between unrelated strain. Statistically significant? #mmgc<br />
+51m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Ion Torrent Semiconductor sequencing - an overview and outlook<br />
Simone Guenther; Life Technologies; Darmstadt, Germany #mmgc<br />
+54m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Guenther :  Ion Proton - real incarnation of a sequencing centre on a box #mmgc<br />
+5m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Guenther Ion new product AmpliSeq for target enrichment #mmgc<br />
+7m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Guenther 2x100bp PE protocol for Ion torrent will be (is?) avalable #mmgc<br />
+10m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Guenther Long mate pair sequencing is also working in Ion Torrent . Good for scaffolding in de novo applications #mmgc<br />
+10m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Simone Guenther Ion Torrent: 314 chip  , paired rate &gt; 90% !! #mmgc<br />
+11m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Guenther AmpliSeq technology can start from only 10ng DNA. #mmgc<br />
+14m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Guenther: Ion torrent software has a core part and plugins that can be user-contributed. Also moving to the Cloud #mmgc<br />
+14m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Simone Guenther Ion Torrent: For RNA-seq maintains RNA orientation #mmgc<br />
+15m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Guenther: Ion Proton can be ordered right now. Arrives in the Summer or December ( noise in the com channel...) #mmgc<br />
+16m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Simone Guenther Ion Torrent: Open protocols , datasets and Source Code: community <a rel="nofollow" href="http://t.co/6l1WUfcT">http://t.co/6l1WUfcT</a>  #mmgc<br />
+17m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Whole genome mapping for de-novo assembly and typing, Richard Moore; OpGen, Gaithersburg, Maryland, USA #mmgc<br />
+18m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Moore Whole Genome Mapping = Optical Mapping ..they're changing names #mmgc<br />
+19m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Richard Moore OpGen: Whole Genome Mapping , Locates and measures distance between restriction sites in a genome  #mmgc<br />
+24m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Moore:  ~ 500 fragments in Salmonella genome #mmgc<br />
+27m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Moore WGmapsCan be used for Genome Comparison #mmgc<br />
+29m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Richard Moore OpGen: Whole Genome Maps distinguish features not detectable by PFGE  #mmgc<br />
+33m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Moore, using contigs and the MapSolver software you could simulate digestion and assemble a Sequence when doing de novo seq #mmgc<br />
+41m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Arndt von Haeseler, Vienna University: Comparison of NGS platform error rates #mmgc<br />
+28m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Alexander Goesmann, Bielefeld University Bioinformatics software tools for analyzing and comparing microbial genomes #mmgc<br />
+53m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Dirk Hper , Friedrich Loeffler Institute , Bioinformatics for rapid single read based analysis of diagnostic metagenome datasets #mmgc<br />
+4m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Dirk Hper , Friedrich Loeffler Institute , finding a needle in a haystack .... #mmgc<br />
+15m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Dirk Hper , Friedrich Loeffler Institute , 7 reads from Orthobunyavirus found in total 27420 reads led to the diagnostic #mmgc<br />
+16m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Dirk Hper , 14 sequence reads representing an Arenavirus in 103632 total reads pointed to the causative agent #mmgc<br />
+25m <a href="https://twitter.com/#!/raqueltobes">raqueltobes</a>: OPGEN: A great NGS-technology independent method to evaluate NGS assemblers #mmgc<br />
+37m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Microbe hunting: ultra deep sequencing to discover microbes in human tissue, Thomas Briese; Columbia University, New York, USA #mmgc<br />
+1m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Thomas Briese, NGS for diagnosis: no previous knowledge of target required #mmgc<br />
+21m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Design of pathogen-based diagnostic assays for the German EHEC and Dutch Klebsiella OXA-48 outbreaks 2011.C.A. Cummings Life Tech #mmgc<br />
+22m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Cummings Bioinformatics plays a large role in the design of new diagnostics #mmgc<br />
+24m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Craig Cummings, Life Technol, Design of pathogen-based diagnostic assay for the German EHEC and Dutch Klebsiella OXA-48 outbreaks 2011 #mmgc<br />
+24m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Cummings: False Positives are costly to industry (Ecoli O157 and meat industry example) #mmgc<br />
+25m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Cummings: Signature identification by genome alignment by MUMmer #mmgc<br />
+26m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Cummings: Draft genome assemblies can also be used to signature identification #mmgc<br />
+27m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Cummings : signature sequences are the diference between the intersection and union between all the test sequences #mmgc<br />
+28m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Craig Cummings, Strain - specific assays often require sequencing of more strains #mmgc<br />
+30m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Craig Cummings, Nucmer for strain - specific signature identification #mmgc<br />
+34m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Cummings: K.pneumoniae dutch outbreak: 21 persons died #mmgc<br />
+36m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Cummings: they used MIRA and CAP3 to do the PGM data assemblies #mmgc<br />
+39m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Cummings : Signatures found by comparing 12 Klebsiella genomes and 7 other Enterobacteriacea genomes #mmgc<br />
+41m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Cummings: 2009 montevideo Salmonella outbreak 1.28M pounds (weight) of Salami was recalled. 47 strains sequenced by SOLID #mmgc<br />
+45m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Cummings: Good results for Ecoli 50x coverage with Long mate pair and assembiles with MIRA. Newbler actually gave better results. #mmgc<br />
+50m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Vaccinology in the era of NGS<br />
Michele Barocchi; Novartis Vaccines and Diagnostics, Siena, Italy #mmgc<br />
+51m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Barocchi - NGS vaccinology (Novartis) #mmgc<br />
+52m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Michele Barocchi, Novartis Vaccines  and Diagnostics: Genomic reverse vaccinology #mmgc<br />
+53m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Barocchi: Pasteur idea: Isolate, Inactivate, Inject =&gt; vaccines #mmgc<br />
+54m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Barocchi: From Jenner to Pasteur 100 years passed and 2 vaccines were out... #mmgc<br />
+56m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Barocchi - vaccine success rate correlates to immune response. Antibodies easy e.g. Diphtheria, T-cell hard #mmgc<br />
+57m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Barocchi: Pneumococcus and Meningoccus have a Antigen variability period on the 10 years range. Influenza 1 year. HIV 1 day. #mmgc<br />
+57m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Barocchi - antigen variability over time plays  important role in vaccine success booster etc #mmgc<br />
+59m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Barocchi: For Reverse Vaccinology, the Bioinformatics step is where targets are identified, targets are then expresses and tested #mmgc<br />
+1m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Neisseria . 2158 ORFs - 570 surface- 350 express in E.coli - 91 confirmation of surface exposure in mouse vaccine discovery pipeline #mmgc<br />
+2m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Barocchi - 3 final Neisseria targets. 12 years to get to this point. #mmgc<br />
+6m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Barocchi - designing a vaccine one one genome difficult due to pan genome and divergence #mmgc<br />
+7m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Barocchi - how many genomes are needed? Streptococcus has 18% genome not shared by all members. #mmgc<br />
+10m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Barocchi: for GBS 3 new genes are added to the genome for each genome after 12 and core genome seems about 1800 genes. #mmgc<br />
+11m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Barocchi - difficult to use core genomes of a species as often not immunogenic, surface exposed #mmgc<br />
+13m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Barocchi : S pneumo core genome of about 1300 genes. After 100 genomes one gene in average was added to pan genome #mmgc<br />
+14m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Barocchi: Population Genetics is specially important for Vaccine Development #mmgc<br />
+14m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Barocchi - if use non-core genes for antigen have to understand pathogens population structure #mmgc<br />
+14m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Michele Barocchi, Some new vaccines need to be based on non core antigens #mmgc<br />
+15m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Barocchi: HGT has a very important role and implications for vaccine design and need to be understood #mmgc<br />
+18m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Evaluating vaccine preventability by bacterial whole genome sequencing Ulrich Vogel; University Wrzburg, Germany #mmgc<br />
+20m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Vogel - WGS evaluating vaccine prevent ability (Neisseria meningitis (again)) #mmgc<br />
+23m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: #mmgc room .Ulrich Vogel speaking <a rel="nofollow" href="http://t.co/NZhK1q9b">http://t.co/NZhK1q9b</a><br />
+30m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Vogel suggests NGS will just be a way to extract current typing rather then use whole genome. MLST, porA etc #mmgc<br />
+35m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Vogel: they used BIGSdb for the analysis of the MIRA assembled sequences. They were surprised how simple it was  #mmgc<br />
+37m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Vogel - gold standard of surrogate of protection is serum bactericidal assay #mmgc<br />
+38m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Vogel: Meningococcal antigen typing system (MATS) pnas paper #mmgc<br />
+38m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Vogel - however SEM not high throughput new surrogate ELISA assay Donnelly et al #mmgc<br />
+42m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Vogel: Clonal Complex can predict the MATS with different Accuracies but lots of strains cannot be assigned to a CC. #mmgc<br />
+45m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Vogel: Phenotypic tests continue to be necessary due to variable expression levels, gene regulation and new variants #mmgc<br />
+18m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Nubel: MRSA ST22 genome international project: 193 genome sequences #mmgc<br />
+19m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Nubel: MRSA ST22 is a measurable evolving population #mmgc<br />
+21m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Nubel: MRSA Sts 239, 225 ,22 ,5 ,121 have  2-3 *10^6 mut/nucleotide per year #mmgc<br />
+26m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Nubel: SNPs found agains a reference genome but also did de novo for the detection of Mobile Genetic Elements #mmgc<br />
+28m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Nubel: evaluating the MRSA 225 spread in the Denmark . Tracking single patients #mmgc<br />
+29m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Ulrich Nbel, comparing phenotypes and genotypes: resistance profiles can be predicted by DNA resistance analysis alone #mmgc<br />
+31m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Nubel: 6 patients 43 isolates 80something snps (?) 5 diferent STs #mmgc<br />
+34m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Nubel: the patient follow-up had samples covering 2-3 years (maybe more) and they had even treatment data #mmgc<br />
+35m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Nubel: the snp analysis could infer the direction of the transmission #mmgc<br />
+40m <a href="https://twitter.com/#!/CDK536">CDK536</a>: Nbel states,there's one clone per patient,but this is evolving as fast as on genus level #mmgc<br />
+44m <a href="https://twitter.com/#!/TimDallman">TimDallman</a>: Off to catch my flight back to London see you later #mmgc<br />
+44m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Rossello-Mora: Species is a pragmatic unit, (it is )artificial, an artifact of the mind #mmgc<br />
+45m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Ramon Rossello-Mora, Institut Mediterrani d Estudis avanc Esporles Spain: The use of NGS on the taxonomyof prokaryotes #mmgc<br />
+47m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Ramon Rossello-Mora, Taxonomy is an artifact of the mind #mmgc<br />
+48m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Rossello-Mora: Systematics doesn't need a scientific background. Taxonomy: general purpose classification, need db #mmgc<br />
+53m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Rossello-Mora:  Avoid species descriptions based on a single strain. Need to have standardized methods. Strains need to be deposited #mmgc<br />
+54m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Ramon Rossello-Mora, A taxonomical unit requires phylogenetic , genomic and phenotypic coherence #mmgc<br />
+5m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Rosselo-Mora: Jspecies software: a biologist oriented software for doing BLAST comparisons and classifying #mmgc<br />
+6m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Ramon Rossello-Mora,DDH is not working for analyzing genomic coherence. It would be substituted by in silico sequence comparison #mmgc<br />
+36m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Dag Harmsen , University Mnster Germany: Real-time Genomic surveillance of infectious disease #mmgc<br />
+37m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Dag Harmsen , In the sequencing era it is needed an error - tolerant clone definition #mmgc<br />
+44m <a href="https://twitter.com/#!/CDK536">CDK536</a>: Maiden: Naming has to be stable and backwords compatible  #mmgc<br />
+3m <a href="https://twitter.com/#!/CDK536">CDK536</a>: Stefan Niemann introducing the EU Patho-NGen-Trace Project- wanting to calibrate and validate ngs data for pathogen typing  esp on tb #mmgc<br />
+6m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Joao Carrio, Facultade de Medicina de Lisboa, Portugal : Microbial Typing Ontology #mmgc<br />
+8m <a href="https://twitter.com/#!/CDK536">CDK536</a>: jacarrico: future wgs databases have to be connected to existing typing databases #mmgc<br />
+9m <a href="https://twitter.com/#!/CDK536">CDK536</a>: jacarrico: we need a babel fish #mmgc<br />
+10m <a href="https://twitter.com/#!/CDK536">CDK536</a>: jacarrico: babel fish available on <a rel="nofollow" href="http://t.co/1fHAmB5j">http://t.co/1fHAmB5j</a> #mmgc<br />
+11m <a href="https://twitter.com/#!/rjlwillems">rjlwillems</a>: Joo Andr Carrio TyPon: prototype ontology for squence based methods #mmgc<br />
+11m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Joao Carrio, TyPon: an ontology to connect the whole genome sequences network with the results of other typing methods #mmgc<br />
+14m <a href="https://twitter.com/#!/rjlwillems">rjlwillems</a>: Joo Andr Carrio TyPon provides a common language to facilitate communication between different typing databases #mmgc<br />
+26m <a href="https://twitter.com/#!/CDK536">CDK536</a>: Aarestrup: Genetic Epidemiology has to get user friendly though everyone can use it #mmgc<br />
+29m <a href="https://twitter.com/#!/CDK536">CDK536</a>: Aarestrup: We need fast automatic one click solutions for medical stuff #mmgc<br />
+39m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: @CDK536 thanks! It is still a work in progress but already translates a bit ;-) We are preparing a proof of concept asap #mmgc<br />
+40m <a href="https://twitter.com/#!/CDK536">CDK536</a>: Aarestrup is voting for a globel database everyone is taking part and stores data as raw as possible #mmgc<br />
+46m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: Marc Struelens, ECDC Stockholm: European perspective of NGS for public health #mmgc<br />
+55m <a href="https://twitter.com/#!/lexnederbragt">lexnederbragt</a>: Pretty cool: my blog post <a rel="nofollow" href="http://t.co/8uJSkrFi">http://t.co/8uJSkrFi</a> cited in a conference presentation, + I find out through somebody tweeting @jacarrico #mmgc<br />
+58m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Marc Struelens (eCDC): Tessy 3.0 in 2012. Bionumerics for data analysis #mmgc<br />
+1m <a href="https://twitter.com/#!/rjlwillems">rjlwillems</a>: ECDC starts an European pilot project on Salmonella typing using PFGE! Difficult to understand after two days talks on NG-seq? #mmgc<br />
+2m <a href="https://twitter.com/#!/CDK536">CDK536</a>: @jacarrico Looking forward to see this proof of concept, wondering how you will rule out human errors in false naming and so on #mmgc<br />
+11m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: PeterGerner-Smidt, CDC , Atlanta , Gerogia: The sunset of culture techniques and the dawning of genomics for the diagnosis... #mmgc<br />
+13m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: PeterGerner-Smidt, Culture / Independent tests for diagnosis of infectious  #mmgc<br />
+14m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Gerner Smith: Public Healths is somebody else's private health (missed the original citation) #mmgc<br />
+16m <a href="https://twitter.com/#!/aunderwo">aunderwo</a>: @jacarrico: Gerner Smith: public health is somebody else's private health #mmgc<br />
+17m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Gerner-Smith With direct sequencing we can save 3-37 days to the current PulseNet protocols #mmgc<br />
+25m <a href="https://twitter.com/#!/eduardopareja">eduardopareja</a>: PeterGerner-Smidt,  Virulence factors from different pathogens can interact and modify virulence #mmgc<br />
+26m <a href="https://twitter.com/#!/jacarrico">jacarrico</a>: Gerner-Smith: will metagenomics the future for rapid diagnostics? He thinks thing will evolve for that #mmgc<br />
+38m <a href="https://twitter.com/#!/pathogenomenick">pathogenomenick</a>: @jacarrico @eduardopareja @TimDallman shall I archive #mmgc tweets on the blog? they are useful (wish I was there but hands are full!)<br />
+55m <a href="https://twitter.com/#!/sciencewr">sciencewr</a>: What have We Got in Common with a Gorilla? <a rel="nofollow" href="http://t.co/oTu2uVIW">http://t.co/oTu2uVIW</a> #Science #Evolution #mmgc</p>
