---
layout: post
status: publish
published: true
title: 'Midlands Sequencing and Bioinformatics Meeting: First Meeting Tuesday, 11th
  December '
author:
  display_name: Nick Loman
  login: nick
  email: n.j.loman@bham.ac.uk
  url: http://xbase.bham.ac.uk
author_login: nick
author_email: n.j.loman@bham.ac.uk
author_url: http://xbase.bham.ac.uk
wordpress_id: 1482
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=1482
date: '2012-11-19 15:06:00 +0000'
date_gmt: '2012-11-19 15:06:00 +0000'
categories:
- Academic life
tags:
- balti and bioinformatics
comments: []
---
<p><strong>Getting to the meeting</strong></p>
<p>The meeting is in the Centre for Systems Biology, 3rd Floor Haworth Building, University of Birmingham. <a href="http://www.birmingham.ac.uk/research/activity/csb/contact/index.aspx">Full details on how to get here</a>. Note you will need to ring the bell to gain entrance.</p>
<p>If you haven't made up your mind how to travel, I'd strongly suggest taking a train, the University has a campus railway station called "University (Birmingham)" and the Centre for Systems Biology is a 2 minute walk down-hill from there. The University station is 5 minutes from Birmingham New Street (take a train in the direction of Redditch, Hereford or Longbridge). </p>
<p>If you are coming by car, park in either the North or the South Car Park which are for visitors. It's pay and display but only a few pounds. Don't try to get onto campus as even if you get past security you will not likely find a space. <a href="http://www.birmingham.ac.uk/contact/directions/edgbaston-directions.aspx">Further details on parking and public transport here</a> ... </p>
<p>Lunch won't be provided, but we'll have some snacks at half-time (authentic Birmingham samosas).</p>
<p>Going for a balti afterwards: we'll take taxis to the Balti Triangle and then back to New Street. We'll aim to be back in town by 8pm so that people can get trains. Those that can stay out later may want to get a drink at the German Market with us.</p>
<p><strong>Introduction</strong></p>
<p>I posted previously about setting up a regular, informal sequencing and bioinformatics meeting in the Midlands area (open to all-comers). Things are coming together nicely with about 30 people from other institutions expressing interest, and so I'm pleased to announce the date of the first meeting will be the afternoon of <strong>Tuesday, 11th December</strong>. We have some great guest speakers lined up already. </p>
<p>Please register for the meeting (the first one is free) via the <a href="http://doodle.com/qtwgwgfwv6te8xdt">Doodle poll</a>. If you want to be notified about future meetings, or discuss how future meetings could work please <a href="https://groups.google.com/d/forum/midlands-sequencing-and-bioinformatics-meeting?hl=en">sign up to the Google group</a>.</p>
<p>The loose theme for this meeting is metagenomics and assembly (and metagenomics assembly).</p>
<p><del datetime="2012-12-03T09:35:52+00:00">Lastly, if you want to give a short talk at this meeting, please get in touch with me directly.</del> The schedule is now full!</p>
<p><strong>Agenda</strong></p>
<p>Venue: Centre for Systems Biology, 3rd Floor, Haworth Building, University of Birmingham. <a href="http://www.birmingham.ac.uk/research/activity/csb/contact/index.aspx">How to get here</a>.</p>
<p>12.45 Registration</p>
<p>13.00 Introduction (Nick Loman, Birmingham)</p>
<p>13.10 - 13.50 Sequence Analysis using Compressed Data Structures (<a href="http://people.ds.cam.ac.uk/js779/">Jared Simpson</a>, Wellcome Trust Sanger Institute. Jared is the author of the ABySS and SGA assemblers.)</p>
<p>13.50 - 14.15 Metagenomics for Diagnosis of Bacterial Infection: Bioinformatics Challenges (Nick Loman, University of Birmingham)</p>
<p>14.15 - 14.30 The Population Structure and Genomic Architecture of <em>S. aureus</em> (<a href="http://www.personal.reading.ac.uk/~gt904211/Richard_Everitts_Homepage/Welcome.html">Richard Everitt</a>, University of Reading)</p>
<p>14.30 - 15.00 Refreshments</p>
<p>15.00 - 15.40 Compressive computing for next-generation sequencing data: making assembly fast, easy, and sensitive (<a href="http://ged.msu.edu/">C. Titus Brown</a>, Michigan State University. Make sure to read <a href="http://ivory.idyll.org/">his blog</a>. Titus is responsible for an exciting new approach to genome assembly he calls 'digital normalization'.)</p>
<p>15.40 - 16.00 High-throughput microbial population genomics using cortex_var (<a href="http://www.well.ox.ac.uk/~zam/">Zamin Iqbal</a>, University of Oxford)</p>
<p>16.00 - 16.10 Hardcore MiSeq Action: run optimisation, dealing with low-diversity samples and some tasty 16S recipes (Josh Quick, University of Birmingham. Josh was one of the systems integration engineers on the MiSeq before joining our group, and there's very little he doesn't know about this machine).</p>
<p>16.10 - 16.30 The bioinformatics clinic<br />
              Format: bring a single slide introducing a problem you would like to discuss with the audience.</p>
<p>16.30 Ends - all welcome to come for a Brummy balti with the speakers.</p>
<p>20.00 (very latest) Return to Birmingham New Street</p>
