---
layout: post
status: publish
published: true
title: 'SBTM12: Sequence-based typing methods for micro-organisms'
author:
  display_name: Nick Loman
  login: nick
  email: n.j.loman@bham.ac.uk
  url: http://xbase.bham.ac.uk
author_login: nick
author_email: n.j.loman@bham.ac.uk
author_url: http://xbase.bham.ac.uk
wordpress_id: 1189
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=1189
date: '2012-07-13 18:38:47 +0100'
date_gmt: '2012-07-13 18:38:47 +0100'
categories:
- Genomic Epidemiology
tags: []
comments: []
---
<p>I'm going to be helping out with teaching the NGS-related aspects of <a href="http://gtpb.igc.gulbenkian.pt/bicourses/SBTM12">this course</a> which might be interesting to you if you are thinking about using whole-genome sequencing for bacterial typing. Keith Jolley will also be there, talking about his software <a href="http://pubmlst.org/software/database/bigsdb/">BigsDB software</a> and his <a href="http://pubmlst.org/rmlst/">new rMLST method</a> for bacterial typing.</p>
<p>The best news is that the course will be in Lisbon in September so the presence of some sunny weather - whilst not guaranteed - is significantly more likely than in the UK! The course is also about 500m from the beach! Not that this should influence your decision to come, obviously.</p>
<p>Here's the skinny:</p>
<blockquote><p>
                         ANNOUNCEMENT</p>
<p>SBTM12 -  SEQUENCE-BASED TYPING METHODS for MICROORGANISMS</p>
<p>with Keith Jolley, Nick Loman, João Carriço,<br />
Mario Ramirez, Nuno Faria and Teresa Conceição</p>
<p>    IMPORTANT DATES for SBTM12<br />
    Deadline for applications: September 10th 2012<br />
    Notification of acceptance dates:<br />
         EARLY: September 3rd 2012<br />
    (on special request, see Application)<br />
         NORMAL: September 12th 2012<br />
    Course date: September 25th - September 28th 2012</p>
<p>Course Description<br />
Overview<br />
Technological advances in DNA sequencing have led to the adoption of<br />
sequence-based typing methods as the standard techniques for bacterial<br />
identification at strain level, specially due to their portability and<br />
reproducibility. The widespread use of remotely accessible databases<br />
offering different typing data and the development of diverse data<br />
analysis techniques show the impact of Bioinformatics in this field,<br />
and the need for understanding how to operate with the databases and<br />
algorithms. Recently, the capability of sequencing whole bacterial<br />
genomes in a few days using Next Generation Sequencing (NGS)<br />
methodologies, opened a new door for the development of more<br />
sophisticated strain identification tools.</p>
<p>Objectives<br />
This training course is directed for the data analysis of sequence<br />
based typing methods, from the raw-data to the identification of a<br />
strain type by several typing methodologies, and the use of analysis<br />
algorithms to create groups of related strains. It will span from<br />
Single Locus Sequence Typing methods, such as spa typing or emm<br />
typing, to the now established MultiLocus Sequence Typing (MLST) and<br />
Multilocus Variable Number of Tandem Repeats Analysis (MLVA)<br />
methodologies. The course will also cover methods that apply to whole<br />
genome sequence data, and show how NGS data can be analyzed and made<br />
sense of, in this context.<br />
The course will be essentially hands-on. Short presentations will be<br />
interweaved by tutored assited exercises.<br />
The participants are expected to gradually gain user independence by<br />
acquiring new analytical skils in using software and online databases<br />
for specific typing methods. Special attention will be given to<br />
methods that can use NGS datasets.</p>
<p>Target Audience<br />
This course is aimed to anyone working in molecular epidemiology, that<br />
wants to develop or consolidate skills in the use and analysis of<br />
sequence-based typing methods. The course will be illustrated with<br />
examples from different bacterial species, but the concepts are<br />
applicable to any species and to other sequence-based typing methods,<br />
not explicitly referred to in the course.</p>
<p>Course Pre-requisites<br />
Basic understanding of molecular biology, namely in microbial typing<br />
methodologies and elementary computer interaction skills are expected.</p>
<p>Futher details, including application instructions available at<br />
<a href="http://gtpb.igc.gulbenkian.pt/bicourses/SBTM12">http://gtpb.igc.gulbenkian.pt/bicourses/SBTM12</a>
</p></blockquote>
