---
layout: post
status: publish
published: true
title: 'Whole-genome sequencing for MRSA epidemiology: Transmission and "clouds of
  variation"'
author:
  display_name: Nick Loman
  login: nick
  email: n.j.loman@bham.ac.uk
  url: http://xbase.bham.ac.uk
author_login: nick
author_email: n.j.loman@bham.ac.uk
author_url: http://xbase.bham.ac.uk
wordpress_id: 1447
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=1447
date: '2012-11-15 11:24:32 +0000'
date_gmt: '2012-11-15 11:24:32 +0000'
categories:
- Genomic Epidemiology
tags: []
comments:
- id: 59936
  author: lizbatty
  author_email: liz@lizbatty.co.uk
  author_url: ''
  date: '2012-11-15 12:29:10 +0000'
  date_gmt: '2012-11-15 12:29:10 +0000'
  content: 'It''s limited data, but there is at least some evidence that within-host
    diversity does not reflect length of carriage - see figure S4 <a href="http://www.pnas.org/content/suppl/2012/02/28/1113219109.DCSupplemental/pnas.201113219SI.pdf"
    rel="nofollow">in the supplementary info of this paper</a> (disclaimer: I am a
    co-author), which shows that a patient who carried S. aureus for 20 months actually
    had less diversity at the last sampled timepoint that the first. This makes me
    wary of the estimated date of colonisation for the staff member based on the diversity
    observed at the date of sampling.'
- id: 59938
  author: Julian Parkhill
  author_email: parkhill@sanger.ac.uk
  author_url: ''
  date: '2012-11-17 17:31:25 +0000'
  date_gmt: '2012-11-17 17:31:25 +0000'
  content: "Ed's issues with the analysis are centred on the question of how much
    diversity might have been present in the patients, and how much of this diversity
    might be passed on in a transmission event. These are certainly interesting questions,
    and deserve further study. However, we do not think that this uncertainty affects
    the underlying conclusions of the paper, particularly the identification of the
    healthcare worker as being potentially involved in the transmission of the strain
    over the 64-day gap.\r\n\r\nThe evidence that the strain was not present on the
    ward during the gap, but that the strains from the patients before and after the
    gap were tightly linked, was sufficient to suggest to the infection control team
    that a carrier may have been involved. The evidence that the cloud of diversity
    present in the healthcare worker encompassed diversity present in patient strains
    before and after the 64-day gap, that none of the other 154 staff tested carried
    the strain, and that no community cases unlinked to the outbreak had been identified,
    implicated that person in transmission across this gap. Indeed, after decolonisation
    of the worker, there was no further occurrence of this strain on the ward, suggesting
    that our inference was correct.\r\n\r\nAlthough analysis of the phylogenetic data
    alone suggests that it is possible that the healthcare worker was infected after
    the 64-day gap, we therefore think it unlikely, for all the reasons given above.
    \r\n\r\nDespite the cloud of diversity in the healthcare worker, there is good
    evidence for a temporal signature in the transmission chains from patients to
    mothers to partners. These are clearly linear, with sequential acquisition of
    mutations, arguing against transmission of a diverse cloud in these cases.\r\n\r\nWe
    have no conclusive evidence that this healthcare worker was involved in transmission
    across the earlier gaps, or was the original source of the outbreak, and we did
    not intend to claim this in the paper.\r\n\r\nThe question of whether the variation
    present in the healthcare worker represents time since colonisation is fascinating,
    and the coincidence of the last common ancestor of this diversity with the origin
    of the outbreak is intriguing, but this is clearly an area that needs a lot more
    study."
- id: 59939
  author: Links 11/17/12 | Mike the Mad Biologist
  author_email: ''
  author_url: http://mikethemadbiologist.com/2012/11/17/links-111712/
  date: '2012-11-17 21:45:23 +0000'
  date_gmt: '2012-11-17 21:45:23 +0000'
  content: '[...] Threat to research funding prompts outcry from Bay State university
    presidents and hospital executives Glass Frogs (very, very cool) New Infection,
    Not Relapse, Brings Back Symptoms of Lyme Disease, Study Finds The Persistence
    of Resistance And Some Reasons Why Whole-genome sequencing for MRSA epidemiology:
    Transmission and “clouds of variation” [...]'
- id: 59974
  author: '[Researcher Comment] Whole-genome sequencing helps halt MRSA bacterial
    outbreak - 2 Minute Medicine2 Minute Medicine'
  author_email: ''
  author_url: http://www.2minutemedicine.com/researcher-comment-whole-genome-sequencing-helps-halt-mrsa-outbreak/
  date: '2013-02-09 20:33:12 +0000'
  date_gmt: '2013-02-09 20:33:12 +0000'
  content: '[...] Blog post by Prof. Ed Feil critiquing the study, with response by
    study author Julian Parkhill in th... [...]'
---
<p>It's an unusual sensation to wake up in the morning and hear Moira Stewart on the Radio 2 breakfast show talking about <em>bacterial genomics and whole-genome sequencing</em>. But it wasn't a lucid dream, <a href="http://www.thelancet.com/journals/laninf/article/PIIS1473-3099(12)70268-2/fulltext">the publication of a new paper from Simon Harris and Sharon Peacock</a> (of Cambridge University and the Sanger Centre, respectively) in Lancet Infectious Diseases yesterday triggered a wave of great publicity proclaiming the power of sequencing bacterial genomes to perform hospital epidemiology of Meticillin-resistant <em>Staphylococcus aureus</em> (MRSA). Indeed my parents were listening to Radio 4 and excitedly emailed to say "they are talking about the kind of stuff you do <a href="http://news.bbc.co.uk/today/hi/today/newsid_9769000/9769436.stm">on the Today programme!</a>". With parents, somehow your work is only really valid when it is featured on the news.</p>
<p>If you haven't read about this study, Ewan Callaway at <a href="http://www.nature.com/news/dna-sequencers-stymie-superbug-spread-1.11808 ">Nature News</a> provides a neat overview. In fact, I'd actually heard about this story before as Sharon Peacock came and gave our departmental seminar to a packed audience last week. We were enthralled with the level of discrimination shown, and translational benefits of such a study, that the results of WGS combined with "shoe-leather epidemiology" could actually end an ongoing outbreak, and change hospital practice and policy. I tweeted:</p>
<blockquote class="twitter-tweet"><p>An impressive aspect of the Peacock MRSA study today is how they used WGS data to help change practice, including swabbing of clinical staff</p>
<p>— Nick Loman (@pathogenomenick) <a href="https://twitter.com/pathogenomenick/status/268648473090486273" data-datetime="2012-11-14T09:36:01+00:00">November 14, 2012</a></p></blockquote>
<p><script charset="utf-8" type="text/javascript" src="//platform.twitter.com/widgets.js"></script>I then didn't check Twitter for a few hours, and came back to a flurry of tweets, triggered by some comments from Ed Feil:</p>
<blockquote class="twitter-tweet" data-in-reply-to="268674568938455040"><p>@<a href="https://twitter.com/alanmcn1">alanmcn1</a> @<a href="https://twitter.com/pathogenomenick">pathogenomenick</a> Regarding the peacock study. Can you look at the tree (Fig F) and explain to me how the staff member could</p>
<p>— Ed Feil (@Number49ivy) <a href="https://twitter.com/Number49ivy/status/268698585787400192" data-datetime="2012-11-14T12:55:09+00:00">November 14, 2012</a></p></blockquote>
<blockquote class="twitter-tweet"><p>@<a href="https://twitter.com/alanmcn1">alanmcn1</a> @<a href="https://twitter.com/pathogenomenick">pathogenomenick</a> possibly have started and maintained the outbreak given that all the variation from this individual is</p>
<p>— Ed Feil (@Number49ivy) <a href="https://twitter.com/Number49ivy/status/268699432734818304" data-datetime="2012-11-14T12:58:31+00:00">November 14, 2012</a></p></blockquote>
<blockquote class="twitter-tweet"><p>@<a href="https://twitter.com/alanmcn1">alanmcn1</a> @<a href="https://twitter.com/pathogenomenick">pathogenomenick</a> limited to one branch? Sure, this person might have infected the final baby (patient 15), but its equally</p>
<p>— Ed Feil (@Number49ivy) <a href="https://twitter.com/Number49ivy/status/268699651249676290" data-datetime="2012-11-14T12:59:23+00:00">November 14, 2012</a></p></blockquote>
<blockquote class="twitter-tweet"><p>@<a href="https://twitter.com/alanmcn1">alanmcn1</a> @<a href="https://twitter.com/pathogenomenick">pathogenomenick</a> likely that this baby infected the patient. Bear in mind they only sequenced multiple colonies from one host</p>
<p>— Ed Feil (@Number49ivy) <a href="https://twitter.com/Number49ivy/status/268699861459816448" data-datetime="2012-11-14T13:00:13+00:00">November 14, 2012</a></p></blockquote>
<blockquote class="twitter-tweet"><p>@<a href="https://twitter.com/alanmcn1">alanmcn1</a> @<a href="https://twitter.com/pathogenomenick">pathogenomenick</a> but only one colony from each baby.</p>
<p>— Ed Feil (@Number49ivy) <a href="https://twitter.com/Number49ivy/status/268700467733880832" data-datetime="2012-11-14T13:02:38+00:00">November 14, 2012</a></p></blockquote>
<p>This led to a long and hard to follow discussion on Twitter debating the finer points of this paper. </p>
<p>Now, working on the basis that the Twitter conversation was hard to follow, but this was an important subject, I asked Ed to summarise his thoughts in a blog post. Ed is another pioneer in the field of bacterial genomic epidemiology and has co-authored a <a href="http://www.ncbi.nlm.nih.gov/pubmed/20093474">seminal paper on intercontinental spread of the ST239 clone of MRSA</a> with some of the same team responsible for the Lancet study.</p>
<p>Personally I am extremely interested in his thoughts as we are now at the point where WGS studies might start to be applied routinely in clinical practice and any remaining issues in the practical usage deserve a decent airing. Ed was a little reluctant as he'd only written one blog post before (about football) but I encouraged him to give it a try anyway ... here's what he said.</p>
<h2>Ed Feil guest post</h2>
<p>Harris et al (LID 14 Nov 2012) <a href="http://www.thelancet.com/journals/laninf/article/PIIS1473-3099(12)70268-2/fulltext">describe the use of rapid new sequencing technologies to investigate an outbreak of MRSA</a> on a special care baby unit (SCBU). There is much that is very impressive about this work. The authors very powerfully demonstrate how this technology is set to revolutionise real time outbreak investigation and epidemiological surveillance. The data identify epidemiologically linked cases (belonging to a single outbreak), broadly confirming inferences from antibiotic resistance profiling; at least in those cases where the antibiotic resistance data was correct. The data also reveal this outbreak to be due to a novel clone, closely related to EMRSA-15 but with the addition of a PVL toxin gene. The presence of this gene can lead to skin and soft tissue infections, and an epidemiological shift from the hospital to the community. This study demonstrates that this technology is fast enough to realistically generate informative data during the course of an outbreak (i.e. within 24 hours). This is at least as quick as current methods and allows real-time interventions. It is also cheap, and getting cheaper. This is a good thing.</p>
<p>Notably, the study also implicates the role of a single health care individual in maintaining the outbreak, and perhaps even as the ultimate source:</p>
<blockquote><p>"Moreover, use of bacterial whole-genome sequencing in real time was able to identify the potential source of an ongoing MRSA outbreak".</p></blockquote>
<p>This individual was taken off the ward and decolonised - this almost certainly prevented onward infections. This is clearly all good.</p>
<p>[caption id="attachment_1458" align="aligncenter" width="483"]<a href="http://pathogenomics.bham.ac.uk/blog/wp-content/uploads/lid2.png"><img class=" wp-image-1458 " title="lid2" src="http://pathogenomics.bham.ac.uk/blog/wp-content/uploads/lid2.png" alt="" width="483" height="398" /></a> Part of one of the brilliant figures from this paper, showing the patients affected by MRSA and possible transmission events. The heathcare worker is marked in orange.[/caption]</p>
<p>There is no doubt in my mind that this technology will be used routinely for outbreak investigation within hospitals in the near to mid-term future. It is inevitable (and indeed desirable) that single individuals will be implicated as a major source of infection in this way. However, this places an extra burden of responsibility to ensure inferences concerning transmission are as accurate as possible, or at least that a clear measure of uncertainty is provided. The identification of single individuals in health care settings as "potential sources" of outbreaks has personal (and indeed legal) implications which have barely begun to be addressed. Pioneering high profile papers such as this need to provide clear signposts and templates for the medical microbiology and infection control communities to follow and develop.</p>
<p>In fact, the role of this individual in maintaining the outbreak is far from clear, and it seems very unlikely to me that the authors have correctly identified this individual as the source or even "a potential source".</p>
<p>The basic problem stems from the fact that the staff member in question and the cases (infected babies) were not treated the same way. Whereas only a single colony was sequenced from the infected babies on the outbreak ward, 20 colonies were sequenced from the member of staff. These 20 isolates from the staff member corresponded to a cloud of variation. This was interpreted to reflect prolonged carriage, and thus to be consistent with a key role of this individual in spreading this MRSA clone over a reasonable length of time, possibly the entire outbreak. Although there is no data to support this, it is argued in the paper that had 20 colonies also been sequenced from each infected baby, the level of diversity in each set of 20 would have been considerably lower than that observed in the staff member lower. That work really needs to be done.</p>
<p>[caption id="attachment_1457" align="aligncenter" width="965"]<a href="http://pathogenomics.bham.ac.uk/blog/wp-content/uploads/eid1.png"><img class="size-full wp-image-1457" title="eid1" src="http://pathogenomics.bham.ac.uk/blog/wp-content/uploads/eid1.png" alt="" width="965" height="415" /></a> Phylogeny of the outbreak using awesome "bullseye" diagrams. On the right the isolates from the healthcare worker are plotted in orange.[/caption]</p>
<p>There are two concerns here:</p>
<p>1. Detectable within-host variation does not necessarily reflect prolonged carriage. The uncomfortable truth is that the ultra-high resolution in the data means we can no longer think in terms of transmission resulting in the movement of one "strain" from patient A to B. Whereas variation can, and will, accumulate within a single individual by de novo mutation over time, it seems highly unlikely that the starting point will be a single pure founding genotype with no variation. Instead, transmission by droplet infection or skin contact may well mean that these clouds of variation are transmitted <em>en masse</em> in a single event. Currently we simply don't have the data to quantify this. However, the upshot is that it is entirely possible that the staff member may in fact have been colonised immediately prior to being sampled from an infected baby on the ward rather than the other way around. If so, this individual may have had absolutely no role at all in the infections that occurred earlier on in the outbreak.</p>
<p>2. The "cloud of variation" present within the staff member does not encompass the whole variation observed within the outbreak isolates, but is restricted to a single branch of the tree. If this individual was indeed the source of infection throughout the course of the outbreak, then an explanation is required as to why the variation within this individual is not scattered throughout the tree. Similarly, the authors argue that the most recent common ancestor of the isolates recovered from the staff member existed prior to the start of the outbreak, thus supporting the possibility that this individual was the source of infections from the start. The clear problem with this logic is that the estimated time to the most recent common ancestor of all the genomes isolated during the outbreak (from infected babies), which is a much more meaningful estimate of the time the outbreak really did emerge, would be much earlier than this.</p>
<p>It is quite clear that we need a great deal more data on within-host variation, and the proportion of this variation that is likely to be transmitted to new hosts to get a firmer grip on the transmission dynamics of this pathogen.</p>
<p>How about the "transmissome"? you heard it here first ...</p>
<p><span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.jtitle=The+Lancet+Infectious+Diseases&rft_id=info%3Adoi%2F10.1016%2FS1473-3099%2812%2970268-2&rfr_id=info%3Asid%2Fresearchblogging.org&rft.atitle=Whole-genome+sequencing+for+analysis+of+an+outbreak+of+meticillin-resistant+Staphylococcus+aureus%3A+a+descriptive+study&rft.issn=14733099&rft.date=2012&rft.volume=&rft.issue=&rft.spage=&rft.epage=&rft.artnum=http%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1473309912702682&rft.au=Harris%2C+S.&rft.au=Cartwright%2C+E.&rft.au=T%C3%B6r%C3%B6k%2C+M.&rft.au=Holden%2C+M.&rft.au=Brown%2C+N.&rft.au=Ogilvy-Stuart%2C+A.&rft.au=Ellington%2C+M.&rft.au=Quail%2C+M.&rft.au=Bentley%2C+S.&rft.au=Parkhill%2C+J.&rft.au=Peacock%2C+S.&rfe_dat=bpr3.included=1;bpr3.tags=Biology%2CMicrobiology%2C+Genetics%2C+Systems+Biology%2C+Bioinformatics%2C+Biotechnology%2C+Computational+Biology%2C+Molecular+Biology">Harris, S., Cartwright, E., Török, M., Holden, M., Brown, N., Ogilvy-Stuart, A., Ellington, M., Quail, M., Bentley, S., Parkhill, J., & Peacock, S. (2012). Whole-genome sequencing for analysis of an outbreak of meticillin-resistant Staphylococcus aureus: a descriptive study <span style="font-style: italic;">The Lancet Infectious Diseases</span> DOI: <a rev="review" href="http://dx.doi.org/10.1016/S1473-3099(12)70268-2">10.1016/S1473-3099(12)70268-2</a></span></p>
