---
layout: post
status: publish
published: true
title: 'Oxford Nanopore megaton announcement: "Why do you need a machine?" - exclusive
  interview for this blog!'
author:
  display_name: Nick Loman
  login: nick
  email: n.j.loman@bham.ac.uk
  url: http://xbase.bham.ac.uk
author_login: nick
author_email: n.j.loman@bham.ac.uk
author_url: http://xbase.bham.ac.uk
wordpress_id: 1057
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=1057
date: '2012-02-17 17:06:28 +0000'
date_gmt: '2012-02-17 17:06:28 +0000'
categories:
- Uncategorized
tags: []
comments:
- id: 59830
  author: AGBT 2012 Blog Post Round-up
  author_email: ''
  author_url: http://pathogenomics.bham.ac.uk/blog/2012/02/agbt-2012-blog-post-round-up/
  date: '2012-02-17 17:17:09 +0000'
  date_gmt: '2012-02-17 17:17:09 +0000'
  content: '[...] Oxford Nanopore megaton announcement: “Why do you need a machine?”
    – exclusive interview for t... [...]'
- id: 59831
  author: anonymous1
  author_email: amy.bouck@gmail.com
  author_url: ''
  date: '2012-02-17 18:38:42 +0000'
  date_gmt: '2012-02-17 18:38:42 +0000'
  content: "Dude... \"You can imagine a sexy Kate Winslet / Marion Cotillard style
    epidemiologist armed with one of these and a laptop.\" \r\n\r\nSeriously?"
- id: 59832
  author: Nick Loman
  author_email: n.j.loman@bham.ac.uk
  author_url: http://xbase.bham.ac.uk
  date: '2012-02-17 18:40:49 +0000'
  date_gmt: '2012-02-17 18:40:49 +0000'
  content: Oh come on, just a bit of fun :)
- id: 59833
  author: rglover
  author_email: rachel.glover@fera.gsi.gov.uk
  author_url: ''
  date: '2012-02-17 18:54:07 +0000'
  date_gmt: '2012-02-17 18:54:07 +0000'
  content: "We do a lot of field-based pathogen diagnostics R&amp;D and I just keep
    thinking of the many applications we could throw this at. I think the DNA barcoding
    community will jump on this if the cost drops - they'll finally get something
    resembling the fabled 'barcorder'. \r\n\r\nanonymous1, we already have scientists
    in 'the field' (literally) with portable real-time PCR machines, now we can give
    them  sequencers...very exciting stuff."
- id: 59834
  author: anonymous1
  author_email: amy.bouck@gmail.com
  author_url: ''
  date: '2012-02-17 18:56:24 +0000'
  date_gmt: '2012-02-17 18:56:24 +0000'
  content: ok... ok...  "You can imagine a sexy Daniel Craig, dressed in those oh-so-hard-to-forget
    baby blue skivvies, armed with one of these and laptop, toiling away in the Caribbean,
    elicidating the metagenomics of coral reef communities." Better? :)
- id: 59835
  author: Nick Loman
  author_email: n.j.loman@bham.ac.uk
  author_url: http://xbase.bham.ac.uk
  date: '2012-02-17 18:58:54 +0000'
  date_gmt: '2012-02-17 18:58:54 +0000'
  content: Hah, that image is not doing it for me but thank you for adding some much-needed
    balance :)
- id: 59836
  author: rglover
  author_email: rachel.glover@fera.gsi.gov.uk
  author_url: ''
  date: '2012-02-17 18:59:42 +0000'
  date_gmt: '2012-02-17 18:59:42 +0000'
  content: sounds better to me!! :)
- id: 59837
  author: All the Oxford Nanopore Videos
  author_email: ''
  author_url: http://pathogenomics.bham.ac.uk/blog/2012/02/all-the-oxford-nanopore-videos/
  date: '2012-02-17 19:26:09 +0000'
  date_gmt: '2012-02-17 19:26:09 +0000'
  content: '[...] Oxford Nanopore website is struggling a bit due to the huge interest
    generated by their announcement earlier today, so I&#8217;ve embedded their new
    videos in this post for you to [...]'
- id: 59838
  author: The Next DNA Disruptor? - Forbes
  author_email: ''
  author_url: http://www.forbes.com/sites/matthewherper/2012/02/17/the-next-dna-disruptor/
  date: '2012-02-17 19:29:39 +0000'
  date_gmt: '2012-02-17 19:29:39 +0000'
  content: '[...] on his fantastic Pathogens: Genes And Genomes blog, biologist Nick
    Loman calls this &#8220;the megaton sequencing announcement we’ve been waiting
    for... Bloomberg has slightly less breathless coverage. There is also a wonderful
    post by the genomicist [...]'
- id: 59844
  author: AGBT 2012 Day 4 Tweets
  author_email: ''
  author_url: http://pathogenomics.bham.ac.uk/blog/2012/02/agbt-2012-day-4-tweets/
  date: '2012-02-18 10:26:26 +0000'
  date_gmt: '2012-02-18 10:26:26 +0000'
  content: '[...] yesterday&#8217;s megaton announcement from Oxford Nanopore certainly
    made up for what had been a slightly subdued AGBT 2012 up to that point in the
    way of [...]'
- id: 59845
  author: JamesH
  author_email: j.hereward@uq.edu.au
  author_url: ''
  date: '2012-02-18 11:53:57 +0000'
  date_gmt: '2012-02-18 11:53:57 +0000'
  content: Any news on whether the Gridion will take samples from 96 well plates at
    the conference, as was on their website previously? For population genomics in
    non model organisms this would be the killer, no barcoding, no MID tags, run 96
    samples, wash and load the next 96, run 100's to 1000's of samples on the one
    cartridge. I think meeting demand for the minion (as if it isn't going to be called
    a minyon) is going to be the biggest struggle for them! I have a bunch of Illumina
    reads for a weed, and this might solve all the mapping/scaffolding issues for
    $900, and having it sticking out of my laptop is going to be hard to beat for
    cool factor! Still taking in all the news. Bring on the revolution!
- id: 59846
  author: Proteo.me.uk &raquo; Blog Archive &raquo; Should you buy a nanopore sequencer?
  author_email: ''
  author_url: http://proteo.me.uk/2012/02/should-you-buy-a-nanopore-sequencer/
  date: '2012-02-18 13:02:39 +0000'
  date_gmt: '2012-02-18 13:02:39 +0000'
  content: '[...] of the two sequencers announced &#8211; others have done a good
    job of listing the specs here and here, but basically you have machines with either
    512, or 2000 nanopores capable of sequencing fragments [...]'
- id: 59847
  author: This so small&#8230;and so HUGE &laquo; genomeek
  author_email: ''
  author_url: http://genomeek.wordpress.com/2012/02/18/this-so-small-and-so-huge/
  date: '2012-02-18 15:30:26 +0000'
  date_gmt: '2012-02-18 15:30:26 +0000'
  content: '[...] Which of course has already been commented here or here and here,
    with some extra informations here [...]'
- id: 59856
  author: GroovyGeek
  author_email: biboyanov@gmail.com
  author_url: ''
  date: '2012-02-19 21:03:37 +0000'
  date_gmt: '2012-02-19 21:03:37 +0000'
  content: "[...]how will “run until” change your research? If you are looking for
    something specific, you just wait until you have got that particular bit of chromosome
    7 running through the pore and stop – no need to oversample the whole genome to
    100x.[...]\r\n\r\nWith a 4% single-pass error rate you still have to sample 100X.
    \ And if that bit of chromosome 7  happens to be at the end of  a fragment you
    have not gained much.\r\n\r\nOverall seems to be a lot better received than the
    previous SMS contender (PacBio) but with no data out there (yet) it is difficult
    to separate the hype from reality.  If only 75% of what they claimed is true it
    would be impressive."
- id: 59857
  author: jsavill
  author_email: John_savill@health.qld.gov.au
  author_url: ''
  date: '2012-02-20 01:49:44 +0000'
  date_gmt: '2012-02-20 01:49:44 +0000'
  content: "well done Nick!  great summary and excellent addition to your Blog.  Next
    time I will go directly to this Blog.\r\n to think i wasted so much time updating
    my research on this technology over the weekend and this morning.  \r\nJohn \r\nfrom
    downunder"
- id: 59858
  author: johansosa
  author_email: johansosa@gmail.com
  author_url: ''
  date: '2012-02-20 05:32:11 +0000'
  date_gmt: '2012-02-20 05:32:11 +0000'
  content: Two questions that are very important to me. 1. --&gt;  Can it do single
    cell sequencing?    2. Can it detect DNA methylation?
- id: 59859
  author: Nick Loman
  author_email: n.j.loman@bham.ac.uk
  author_url: http://xbase.bham.ac.uk
  date: '2012-02-20 10:32:14 +0000'
  date_gmt: '2012-02-20 10:32:14 +0000'
  content: |-
    @johansosa
    1) it does single molecule sequencing without amplification so I guess it can do single cell sequencing! I guess it will be more efficient if you can enrich your sample for the cells of interest.
    2) don't think we know the details about what base modifications are detectable yet
- id: 59860
  author: Nick Loman
  author_email: n.j.loman@bham.ac.uk
  author_url: http://xbase.bham.ac.uk
  date: '2012-02-20 10:33:58 +0000'
  date_gmt: '2012-02-20 10:33:58 +0000'
  content: '@GroovyGeek - you are right you will need to see the molecule a few times
    to get an accuracte consensus sequence, but the very long read lengths mean unambiguous
    alignments, there is no amplification stage (so no PCR biases e.g. under-representation
    of high GC regions) and the error model is supposed to be mainly deletions, so
    you won''t need anything like 100x oversampling if you want to find a SNP (for
    example).'
- id: 59862
  author: Beyond the Hype &#8211; The Oxford Nano Announcement at AGBT &laquo; Omically
    Speaking
  author_email: ''
  author_url: http://www.omespeak.com/blog/?p=507
  date: '2012-02-20 17:59:22 +0000'
  date_gmt: '2012-02-20 17:59:22 +0000'
  content: '[...]  [...]'
- id: 59865
  author: jsavill
  author_email: John_savill@health.qld.gov.au
  author_url: ''
  date: '2012-02-21 07:33:37 +0000'
  date_gmt: '2012-02-21 07:33:37 +0000'
  content: "Nick\r\nI enjoyed yours and others  past analyses of Ion torrent vs Miseq
    sequencer and data and assembly.  \r\n\r\nWe are all now very interested to see
    the Beta testing of the GridIon using E coli  DH108  also, to compare for accuracy
    as they move forward in development.  \r\n\r\nWere there any hints of nominated
    labs for Beta testing eg Broad or the cost of each of the Gridion versions? \r\nWas
    there a mention of whether we should wait until Q3  or before?\r\njohn\r\nfrom
    downunder"
- id: 59866
  author: So What Could Nanopore Mean for Bacterial Genomics? (And the Pelham 123
    Problem) | Mike the Mad Biologist
  author_email: ''
  author_url: http://mikethemadbiologist.com/2012/02/21/so-what-could-nanopore-mean-for-bacterial-genomics-and-the-pelham-123-problem/
  date: '2012-02-21 15:05:53 +0000'
  date_gmt: '2012-02-21 15:05:53 +0000'
  content: '[...] now, you might have heard about Oxford Nanopore&#8216;s first preview
    of their Gridion and Minion technologies at the recent [...]'
- id: 59867
  author: USB-Sized DNA Sequencer by Oxford Nanopore | Andrew Hessel
  author_email: ''
  author_url: http://andrewhessel.com/?p=412
  date: '2012-02-21 15:17:36 +0000'
  date_gmt: '2012-02-21 15:17:36 +0000'
  content: '[...] to retail at about $900.  For more information, see the company
    website or Nick Loman&#8217;s terrific blog post on the device.  Tags: MinION,
    Nanopore, Oxford Nanopore, sequencing, USB   Previous postTomorrow [...]'
- id: 59869
  author: fink123
  author_email: sindhu_kurup@hotmail.com
  author_url: ''
  date: '2012-02-22 08:16:00 +0000'
  date_gmt: '2012-02-22 08:16:00 +0000'
  content: what will be the coverage attained by this method?
- id: 59870
  author: The Three Musketeers of Genome Sciences and Bioinformatics | TalkBiotech
  author_email: ''
  author_url: http://talkbiotech.org/?p=257
  date: '2012-02-22 13:19:50 +0000'
  date_gmt: '2012-02-22 13:19:50 +0000'
  content: '[...] it will further sky-rocket the sequence data volume into an unimaginable
    height (Blogs on MinION: Pathogens: Genes and Genomes; genomes [...]'
- id: 59871
  author: The Three Musketeers of Genome Sciences and Bioinformatics &laquo; Aakhayan
  author_email: ''
  author_url: http://raunakms.wordpress.com/2012/02/23/the-three-musketeers-of-genome-sciences-and-bioinformatics/
  date: '2012-02-23 00:32:08 +0000'
  date_gmt: '2012-02-23 00:32:08 +0000'
  content: '[...] it will further sky-rocket the sequence data volume into an unimaginable
    height (Blogs on MiniON: Pathogens: Genes and Genomes; genomes [...]'
- id: 59872
  author: 'Dispositivo USB Sequenciador de DNA : Makarispo'
  author_email: ''
  author_url: http://makarispo.com/index.php/dispositivo-usb-sequenciador-de-dna/
  date: '2012-02-23 12:11:02 +0000'
  date_gmt: '2012-02-23 12:11:02 +0000'
  content: '[...] deve ser vendido em cerca de US $ 900. Para mais informações, consulte
    o site da empresa ou o post do blog de Nick Loman sobre o dispositivo. Compartilhe!           [...]'
- id: 59878
  author: Making sequencing simpler with nanopores | Biomedical Genomics &amp; Evolution
    Lab
  author_email: ''
  author_url: http://hdopazolab.com/2012/03/10/making-sequencing-simpler-with-nanopores/
  date: '2012-03-10 18:56:38 +0000'
  date_gmt: '2012-03-10 18:56:38 +0000'
  content: '[...] Scientist, GenomeWeb, BioIT World and Forbes all have reported on
    it, and bloggers Nick Loman and Keith Robison have also had a chance to talk to
    some of the Oxford Nanopore peeps about [...]'
- id: 59881
  author: 2012 Update to the NGS Field Guide | The Molecular Ecologist
  author_email: ''
  author_url: http://www.molecularecologist.com/2012/04/2012-update-to-the-ngs-field-guide/
  date: '2012-04-22 18:13:38 +0100'
  date_gmt: '2012-04-22 18:13:38 +0100'
  content: '[...] was followed by the &#8220;megaton announcement&#8221; of the gridION
    and minION nanopore sequencers at AGBT in February. Now, we&#8217;ll have to [...]'
- id: 59962
  author: Oxford Nanopore sequencer annoucements &laquo; To strive, to seek, to find,
    &#8230;
  author_email: ''
  author_url: http://chl82.wordpress.com/2012/02/18/oxford-nanopore-sequencer-annoucements/
  date: '2013-01-22 00:47:32 +0000'
  date_gmt: '2013-01-22 00:47:32 +0000'
  content: '[...] and MinION systems yesterday, which both use their nanopore sequencing
    technology. Luke Jostins, Nick Loman, and Keith Robison have all provided detailed
    commentary about this release, so I&#8217;ll only [...]'
---
<p>Sometimes this genome blogging lark really pays off. Yesterday was one of those days as I got a sneak preview of the big announcement at AGBT, and 20 minutes to speak with Oxford Nanopore's Dan Turner (Director of Applications), Clive Brown (Chief Technical Officer) and Zoe McDougall (Director of Comms). The downside of course was that I couldn't tell anyone what they said until the embargo was lifted at 5pm today!</p>
<p>I asked as many questions as I could without knowing the contents of the AGBT talk. I probably should have asked a bunch more. I do remember saying "wow" quite a lot.</p>
<p>First, go and read the <a href="http://pathogenomics.bham.ac.uk/blog/2012/02/oxford-nanopore-introduces-dna-strand-sequencing-on-the-high-throughput-gridion-platform-and-presents-minion-a-sequencer-the-size-of-a-usb-memory-stick/">press release</a>!</p>
<p>Executive Summary</p>
<ul>
<li>Nanopore have announced a strand sequencing method, made possible by a heavily modified biological nanopore and an industrially-fabricated polymer
<li>DNA passes through the nanopore and tri-nucleotides in contact with the pore are detected through electrochemistry
<li>Demonstrated 2x50kb sense & anti-sense of same molecules (lambda phage) - no theoretical read length limit
<li>Can sequence direct from blood without need for sample preparation
<li>Two products announced:
<ul>
<li>MinIon - USB disposable sequencer for ~ $900 has 512 nanopores - target 150mb/hour
<li>MinIon can run at 120-1000 bases/minute per pore for up to 6 hours
<li>GridIon - two versions of rack-mountable sequencer with 2000 nanopores (2nd half 2012), 8000 nanopores (2013)
<li>GridIons can be racked in parallel, 20 could do a whole human genome in 15 minutes
<li>Each GridIon can do "tens of gigabases" over 24 hours
</ul>
<li>Both machines commercially available 2nd half 2012
<li>Sequencing can be paused, sample recovered, replaced, started again
<li>Accuracy is 96%, errors are deletions, error profile will improve through software
</ul>
<p><strong>The MinIon</strong></p>
<p>Firstly you need to know this is pronounced "min, ion" rather than "min-yon". The MinIon is an array of nanopores - 512 to be precise - and circuitry housed in a USB stick. Why a USB stick? "The form factor is determined by the requirements" - as there are no fluidics you don't need a big machine. There are <strong>no fluidics</strong>. "Your fluidics is a Gilson", said Brown. The prototype version has an ugly battery pack attached to it but it will eventually use USB power. The USB stick is <strong>disposable</strong>. "Why do you need an instrument?" he says. We wander into the realms of sci-fi at this point. DNA molecules pass through the nanopore and nucleotide sequence is detected by the electronics. Bases are streamed - live - to your laptop as FASTQ (bases with qualities). This is where the "run until" makes sense, if you are interested in a particular gene just wait until the sequence comes out and shut it down to preserve the circuitry.</p>
<p><strong>"Hungry Hippo" sequencing</strong></p>
<p><img alt="" src="http://blogs.westword.com/showandtell/hippos1.jpg" title="Hungry Hippos!" class="alignnone" width="500" height="311" /><br />
The sample lives in solution and the DNA floats around, there is no attachment to beads or solid surfaces. Brown says the nanopore functions "like a Hungry Hippo", grabbing DNA as it bumps up against the pore. Only about 0.001% of the starting DNA is processed and the sample is unadulterated and can be recovered at the end. Do some sequencing, then literally "pause" it, remove the sample - mess around with it - replace it and continue sequencing. This is an iterative approach - how will it change our way of doing sequencing experiments? Can you run forever? "Right now this is limited by the circuits which burn out after about 6 hours."</p>
<p><strong>Fast-Slow Sequencing</strong></p>
<p>Nanopore can ratchet down the speed of the DNA passing through the pore. Right now they are running "about 120-150 bases per minute", but this can be sped up to give 150 megabases per hour of sequence from the MinION. It can be sped further, even as far as 1000 bases per minute per pore, although running above 500 bases per minute starts to impact the accuracy.</p>
<p><strong>Read lengths</strong></p>
<p>Read length reflects the fragment size in the sample. They've sequenced sense and antisense of lambda phage - giving a 100kb read from same molecule. "We've tried 5, 10, 15, 20kb etc. without problems" says Dan Turner. You could bung an Illumina library in there should you wish.</p>
<p><strong>Accuracy</strong></p>
<p>A huge question. They are talking about a 4% raw read error rate. The errors are mainly deletions. "We know what the problem is and we can fix it" said Turner. He explained the signal comes from the interaction of DNA with the pore barrel. The barrel is in contact with three nucleotide bases at one time, then the strand is moved along one base by a "special processive enzyme". Each tri-nucleotide gives a specific signal. A regular Viterbi algorithm turns that into bases, and the Markov model can be improved through better training. Notably the quality does not drop off over the length of the read and substitution errors are infrequent.</p>
<p><strong>The Breakthrough</strong></p>
<p>A major problem with turning biological nanopores into DNA sequencers is that they are often embedded in fragile lipid bilayers which are unstable to pH and temperature (see <a href="http://core-genomics.blogspot.com/2012/01/how-does-nanopore-sequencer-work.html">James Hadfield's great nanopore primer</a> for more information). The major break-through is that Oxford Nanopore can make this membrane in an industrial process in a factory and the result remains stable, meaning it can be shipped to the customer at room temperature. The nanopore itself has over 300 modifications to make it work efficiently, these were found through random mutation during a series of high-throughput screening experiments.</p>
<p><strong>Bioinformatics</strong></p>
<p>With the MinIon - bases are streamed off in FASTQ straight to your laptop. You can set up your workflow to "run until ..." a particular sequence is read, and then shut it off to protect the circuitry.</p>
<p><strong>The GridIon</strong></p>
<p>The GridIon has already been announced and is a rackable sequencer which takes disposable cartridges for massively parallel sequencing. It will be available in 2000 and 8000 nanopore versions. There are four wells to one circuit.</p>
<p><strong>Sample Preparation</strong></p>
<p>The example given at the presentation was sequencing of whole blood. "You add blood, and some buffer and some enzymes", presumably to nick the DNA strands which don't need to be denatured. That's it. But you could process your sample in a million different ways first, as long as dsDNA is presented to the pore. "You could even put an Illumina library in". No amplification required. At the presentation Brown said they "had a look at RNA" and found they could read it just like DNA, no reverse complementation necessary.</p>
<p><strong>The Illumina Connection</strong></p>
<p>This is an Oxford Nanopore product, being marketed by Oxford Nanopore. Illumina have a stake in the company.</p>
<p><strong>Is this a game changer?</strong></p>
<p>This is a very different product to the existing suite of sequencers. It's also rather dissimilar to PacBio despite being single molecule (but apparently superior in all respects).</p>
<p>For those chasing the goal of doing tens of thousands of $1,000 human genomes, this may not seem to be the obvious solution. The cost per base is competitive with existing systems. But I think we need to think differently about this machine:</p>
<ul>
<li>how will access to a disposable sequencer change the way we do biology and medicine? With no capital costs, this certainly has the power to democratise sequencing even further. But does the $900 price point make it a tough sell for near-patient testing or field applications, or will clever use of the technology make it economical? Immediately you imagine that you could load a sample, sequence for 10 minutes, remove, put another sample in to get a kind of barcode-free multiplexing
<li>how will 100kb+ reads change your research? It seems to me this will solve the very gritty problem of trying to reconstruct microbial communities through assembly of short-reads. Large chromosomal variations should be detectable with ease. Certainly <em>de novo</em> assembly will become trivially easy for many organisms. Should be amazing for 'unculturable' microbes too.
<li>how will "run until" change your research? If you are looking for something specific, you just wait until you have got that particular bit of chromosome 7 running through the pore and stop - no need to oversample the whole genome to 100x.
<li>we can now take the sequencer out into the field. How will this transform microbial ecology? You can go and pipette in some sewage water and be a 21st-century John Snow and detect cholera. Does this help with the "disease weather map" ? You can imagine a sexy Kate Winslet / Marion Cotillard style epidemiologist armed with one of these and a laptop.
<li>what experiments can you do with iterative sequencing? What are the benefits of taking a sample on and off and manipulating it? Will this change the way we do RNA & ChIP-Seq type research?
</ul>
<p>I'm still processing this announcement and would love to hear your thoughts below.</p>
<p>But to sum up, this is the megaton sequencing announcement we've been waiting for.</p>
