---
layout: post
status: publish
published: true
title: 'COST training school: Bioinformatics for microbial community analysis'
author:
  display_name: Nick Loman
  login: nick
  email: n.j.loman@bham.ac.uk
  url: http://xbase.bham.ac.uk
author_login: nick
author_email: n.j.loman@bham.ac.uk
author_url: http://xbase.bham.ac.uk
wordpress_id: 1496
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=1496
date: '2012-11-22 10:09:08 +0000'
date_gmt: '2012-11-22 10:09:08 +0000'
categories:
- Uncategorized
- High-throughput sequencing
- Metagenomics
- 16S
- Ion Torrent
- Genomics
tags: []
comments: []
---
<p>I will be helping teach one day of this course which is fully-funded if you are a student from a COST-participating country... see below for details:</p>
<p>Deadline is next week, so hurry if you are interested.</p>
<blockquote><p>COST training school ES1103: Bioinformatics for microbial community analysis</p>
<p>Dates: December 11th- 14th (the school will begin at 1pm on Tuesday and finish at 5pm on Friday)</p>
<p>Location: University of Liverpool, Centre for Genomic Research (CGR), Life Sciences Building, Liverpool L69 7ZB</p>
<p>Organisers: Dr Christopher Quince (Christopher.quince@glasgow.ac.uk), Dr Christiane Hertz-Fowler (chf@liv.ac.uk)</p>
<p>Lecturers: Dr Christopher Quince (Christopher.quince@glasgow.ac.uk), Dr Nick Loman (n.j.loman@bham.ac.uk) and Dr Martin Hartmann (martin.hartmann@microbiome.ch)</p>
<p>Description: The aim of this three and a half day workshop will be to give students an overview of the tools and bioinformatics techniques available for the analysis of next generation sequence data from microbial communities. The emphasis during the first three days will be on the analysis of amplicon sequences, for example 16S rRNA or fungal ITS, generated using next generation sequencing platforms: principally 454 but also Illumina and Ion Torrent. The entire process from initial sequence data through filtering, noise removal, OTU generation and taxonomic classification will be addressed during the first day and a half. Detailed instruction on performing noise-removal, using AmpliconNoise, and the use of the Mothur pipeline will be provided. The emphasis on the third day will be on multivariate statistics for the analysis and ecological interpretation of the resulting data sets using R. The final day of the workshop will consist of an introduction to microbial genomics, covering de novo sequence assembly, and gene annotation. Metagenome analysis will be discussed but not in great detail. The format will comprise a mixture of lectures and hands-on tutorials where students will process example data sets in real-time.  Students will also be encouraged to bring their own data for analysis.</p>
<p>Application procedure: Funded places exist through the COST action ES1103 for students from participating member states (see http://www.cost.eu/domains_actions/essem/Actions/ES1103?parties).  Applications should be sent by e-mail to Dr Christopher Quince (Christopher.quince@glasgow.ac.uk). Applications should consist of one paragraph describing the studentâ€™s motivation for attending the course and level of bioinformatics experience. The latter is purely to allow the correct pitching of content and all levels of prior knowledge will be catered for. However, basic Linux and sequence analysis skills would be helpful. Applications received before Wednesday November 28th will be considered and students selected who will benefit most from the training. Students from any career level from under-graduate to professorial can attend but this is a hands-on workshop and preference will be given to people who will analyse their own data at some point in the future.</p>
<p>Funding: COST will reimburse each student up to 1000 EUs to cover registration, travel, hotel and food. Hotel reservations have been made in a block booking at the nearby Liner Hotel (75 EU per night). If you do not want to stay at this hotel please notify us in your application. There will be an 80EU registration fee payable in advance by the student. This will cover lunches during the course.</p></blockquote>
