---
layout: post
status: publish
published: true
title: 'A new sequencing technology enters the ring: SHTseq(TM)'
author:
  display_name: Nick Loman
  login: nick
  email: n.j.loman@bham.ac.uk
  url: http://xbase.bham.ac.uk
author_login: nick
author_email: n.j.loman@bham.ac.uk
author_url: http://xbase.bham.ac.uk
wordpress_id: 1003
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=1003
date: '2012-02-15 16:33:04 +0000'
date_gmt: '2012-02-15 16:33:04 +0000'
categories:
- High-throughput sequencing
tags:
- crapbio
- amazing new sequencing technology
- shtseq
comments:
- id: 59804
  author: CrapBio brings you SHTseq(TM) – sequencing on the cloud « Biochemistry,
    Molecular, Cellular &amp; Developmental Biology
  author_email: ''
  author_url: http://bmcdb.wordpress.com/2012/02/15/crapbio-brings-you-shtseqtm-sequencing-on-the-cloud/
  date: '2012-02-15 23:37:28 +0000'
  date_gmt: '2012-02-15 23:37:28 +0000'
  content: ''
- id: 59816
  author: '#NotAGBT Breaking News: TCAK announce legal challenge against CrapBio'
  author_email: ''
  author_url: http://pathogenomics.bham.ac.uk/blog/2012/02/notagbt-breaking-news-tcak-announce-legal-challenge-against-crapbio/
  date: '2012-02-16 14:27:57 +0000'
  date_gmt: '2012-02-16 14:27:57 +0000'
  content: '[...] This press release just in from Chad Bower at TCAK suggesting CrapBio&#8217;s
    machine &#8211; announced yesterday &#8211; might have hit serious problems before
    it&#8217;s even released! Release follows: Today the [...]'
- id: 59817
  author: AGBT 2012 Day 2 Tweets
  author_email: ''
  author_url: http://pathogenomics.bham.ac.uk/blog/2012/02/agbt-2012-day-2-tweets/
  date: '2012-02-16 14:40:50 +0000'
  date_gmt: '2012-02-16 14:40:50 +0000'
  content: '[...] other news was of course the announcement of the incredible sounding
    SHTseq(TM) from CrapBio which can produce 100Mb reads at 25% accuracy. This was
    announced at the parallel #notAGBT [...]'
- id: 59820
  author: AGBT 2012 Blog Post Round-up
  author_email: ''
  author_url: http://pathogenomics.bham.ac.uk/blog/2012/02/agbt-2012-blog-post-round-up/
  date: '2012-02-17 09:57:54 +0000'
  date_gmt: '2012-02-17 09:57:54 +0000'
  content: '[...] A new sequencing technology enters the ring: SHTseq(TM) [...]'
- id: 59864
  author: Links 2/20/12 | Mike the Mad Biologist
  author_email: ''
  author_url: http://mikethemadbiologist.com/2012/02/20/links-22012/
  date: '2012-02-20 21:45:03 +0000'
  date_gmt: '2012-02-20 21:45:03 +0000'
  content: '[...] A new sequencing technology enters the ring: SHTseq(TM) A tale of
    two analysts Divided Auckland: Overcrowding a hotbed for infections More Doctors
    &#8216;Fire&#8217; Vaccine Refusers: Families Who Reject Inoculations Told to
    Find a New Physician; Contagion in Waiting Room Is a Fear Wright is wrong about
    sociology as well as basketball [...]'
- id: 59889
  author: Humor &#8211; Announcement of a New Sequencing Technology &laquo; Homologus
  author_email: ''
  author_url: http://www.homolog.us/blogs/2012/06/08/humor-announcement-of-a-new-sequencing-technology/
  date: '2012-06-08 19:41:09 +0100'
  date_gmt: '2012-06-08 19:41:09 +0100'
  content: '[...] Please follow the rest of the press release at this link. [...]'
- id: 59890
  author: UK Next Gen Sequencing Meeting 2012 &#8211; Loads of good sessions
  author_email: ''
  author_url: http://pathogenomics.bham.ac.uk/blog/2012/07/uk-next-gen-sequencing-meeting-2012-loads-of-good-sessions/
  date: '2012-07-30 08:09:56 +0100'
  date_gmt: '2012-07-30 08:09:56 +0100'
  content: '[...] including &#8220;England&#8217;s oldest pub&#8221;, Ye Olde Trip
    to Jerusalem. Neil Hall, the CEO of SHTseq, has said the first round is on [...]'
- id: 59991
  author: The Secondary Problem With the Culture and Brain IQ Paper | Mike the Mad
    Biologist
  author_email: ''
  author_url: http://mikethemadbiologist.com/2013/03/30/the-secondary-problem-with-the-culture-and-brain-iq-paper/
  date: '2013-03-30 14:01:16 +0000'
  date_gmt: '2013-03-30 14:01:16 +0000'
  content: '[...] There&#8217;s a recent and ridiculous Culture and Brain paper on
    the genetics of intelligence in Chinese college students. Thankfully, Jeremy Yoder
    knocks down the ridiculous statistics, and as he notes we&#8217;ll probably see
    more SHTstat (which is the analytical module for SHTseq). [...]'
- id: 60173
  author: Quora
  author_email: ''
  author_url: http://www.quora.com/What-is-CrapBio-and-how-does-its-SHiT-Sequencing-Technology-work#ans6610271
  date: '2014-08-20 08:18:08 +0100'
  date_gmt: '2014-08-20 08:18:08 +0100'
  content: |-
    <strong>What is CrapBio and how does its SHiT Sequencing Technology work?</strong>

    "There's a question that I've been asking myself for nearly three decades now and trying to get a research handle on, and that is why is the quality of public debate so low and why is it that the quality often seems to deteriorate the more important&#8230;
- id: 60175
  author: 'Nick Loman Presents Nanopore Data, #NotsoNickLoman Mocks Him &laquo; Homolog.us
    &#8211; Bioinformatics'
  author_email: ''
  author_url: http://www.homolog.us/blogs/blog/2014/09/03/nick-loman-presents-nanopore-data-notsonickloman-mocks-him/
  date: '2014-09-03 22:15:49 +0100'
  date_gmt: '2014-09-03 22:15:49 +0100'
  content: '[&#8230;] A new sequencing technology enters the ring: SHTseq(TM) [&#8230;]'
---
<p>Just received this press release from Neil Hall, CEO of a new sequencing company called CrapBio who's launching at this year's <a href="https://twitter.com/#!/search/%23notAGBT">#notAGBT</a>.. sounds very interesting!<br />
<a href="http://pathogenomics.bham.ac.uk/blog/wp-content/uploads/crapbio_logo.jpg"><img src="http://pathogenomics.bham.ac.uk/blog/wp-content/uploads/crapbio_logo-300x226.jpg" alt="" title="crapbio_logo" width="300" height="226" class="alignleft size-medium wp-image-1011" /></a><br />
<strong>CrapBio </strong><br />
CrapBio is excited to announce the release of its new SHTseq&trade; whole genome sequencing platform which is going to revolutionize the genomics industry and soon the healthcare industry. </p>
<p><strong>SHTseq</strong><br />
SHTseq or Super High-throughput Template sequencing is a totally new paradigm for DNA analysis.  In the era of second and third generation sequencing, scientists would take days to extract DNA and prepare it for analysis on slow cumbersome analysis machines.  CrapBio is able to get around this using AngstromRealtimeSensors&trade;  that can accurately read the DNA while still inside cells. Better than that, you don’t even need to take the DNA to the analysis  machine you can simply take an image of the organism you wish to sequence and upload it to the SHTcloud and have the DNA information  extracted directly by out highly trained SHTtechnicians.</p>
<p>With SHTseq you can simply send the sample from you iphone and have the sequencing data returned to your email address in a matter of minutes. THATS RIGHT!,  you can have a SHT experiment run in minutes! Not hours or days.  Download your iSHT app now and start sequencing. </p>
<p><strong>Longer Reads-Better Data – noSHT (What would you do with 100Mb reads?).</strong><br />
We are able to generate super-long reads with our ARSesnsors. Using CrapBio-SHTseq technology we regularly get 10Mb reads and we have even seen reads of 100Mb which completely sequenced E. coli 20 times in a single read.  Our base calling accuracy is 25%, but with genomes with extreme AT/GC bias it reaches 40%.  Although this is lower than other platforms the longer reads allow you to extract much more information from our reads than old-fashioned 2nd generation sequencers. Also this error is totally randomly distributed (unlike homopolymer errors in other technologies!)  and there is no decline in base calling accuracy toward the ends of reads. The last base in a read is just as good as the first base. </p>
<p><strong>Cleaning up SHT&trade;  with Illumina data</strong><br />
If, for whatever reason, you need accurate sequence data we have developed hybrid assembler that can incorporate Illumina error correcting reads.  With our HybridAssemblyReadDenoisingSHT data you can simply upload you 100x illumina data with your sample and get reads returned to you will 99.999% accuracy*   </p>
<p><strong>CompleteSHT&trade; our human genome analysis service</strong><br />
Finally we are pleased to announce our CompleteSHT human genome sequencing service. We have optimized our laboratory and informatics pipelines for human DNA sequencing and are able generate a DNA sequence de novo, without HARDSHT, with 99.9% accuracy**.  Simply send the name, ethnicity and starsign of the subject and we can return a SNP file to your email address INSTENTAINIOUSLY!***. </p>
<p><strong>How much does SHT cost?</strong><br />
Due to the way our technology works we have very low reagent costs, therefore SHTseq is the cheapest technology on the market. Speak to a CrapBio representative to find out more.  </p>
<p><strong>More SHT</strong><br />
If you would like to speak to one of our product specialists to find out how SHT will change the way you do science please email SHT@CRP.COM </p>
<p>*there may be a high chimera rate with this product.<br />
** this works best with samples of European decent<br />
*** for research use only </p>
