---
layout: post
status: publish
published: true
title: I want to learn bioinformatics! A guide for complete beginners.
author:
  display_name: Nick Loman
  login: nick
  email: n.j.loman@bham.ac.uk
  url: http://xbase.bham.ac.uk
author_login: nick
author_email: n.j.loman@bham.ac.uk
author_url: http://xbase.bham.ac.uk
wordpress_id: 1718
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=1718
date: '2013-07-18 10:27:30 +0100'
date_gmt: '2013-07-18 10:27:30 +0100'
categories:
- Uncategorized
tags: []
comments:
- id: 60039
  author: marcowanger
  author_email: marcowanger@gmail.com
  author_url: ''
  date: '2013-07-18 16:34:34 +0100'
  date_gmt: '2013-07-18 16:34:34 +0100'
  content: Tell him to try Rosalind, browse through the threads in Seqanswers. Also
    The NGS Wikibook.
- id: 60040
  author: I want to learn bioinformatics! A guide for complete beginners. | Roberts
    Lab
  author_email: ''
  author_url: http://faculty.washington.edu/sr320/?p=3834
  date: '2013-07-18 18:16:49 +0100'
  date_gmt: '2013-07-18 18:16:49 +0100'
  content: '[...] I want to learn bioinformatics! A guide for complete beginners.
    [...]'
- id: 60041
  author: lzwright
  author_email: lzwright@gmail.com
  author_url: http://ngsnewbie.blogspot.com/
  date: '2013-07-18 19:02:43 +0100'
  date_gmt: '2013-07-18 19:02:43 +0100'
  content: Ah, my pal Bio Mick Watson.  That's where I started w/ Linux.  maybe 4,
    5, 6 mos ago?  Once I got ubuntu installed (and I tried out a few other distros
    as well) you could not prevent me from putting it on every computer within striking
    distance.  I virtually knew nothing about anything when I started.  I got a couple
    of linux books (latest acquisition is Sobell) and I also recently acquired Practical
    Computing for Biologists.  I am now deep in the land of de novo assembly of venom
    duct transcriptome of the marine snail Terebra anilis (terebrids are sister family
    to the better known cone snails).  I was lucky enough to attend Titus Brown's
    NGS 2013 but was also doing his tutorials way before I got there, and did not
    find them so difficult, even as a newbie, once I had some linux under my belt.   Now
    I *really* need to learn some programming.  I am following an MIT course on python
    programming http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-00sc-introduction-to-computer-science-and-programming-spring-2011/Syllabus/
    and using something called code academy http://www.codecademy.com/tracks/python
    to pave the way.  Have not yet explored Rosalind.  It's hard to find the time
    for this when I spending my life at the terminal, but I kinda sorta hafta.
- id: 60042
  author: Nick Loman
  author_email: n.j.loman@bham.ac.uk
  author_url: http://xbase.bham.ac.uk
  date: '2013-07-19 08:12:50 +0100'
  date_gmt: '2013-07-19 08:12:50 +0100'
  content: "Good suggestion re: the NGS Wikibook, link here:\r\nhttp://en.wikibooks.org/wiki/Next_Generation_Sequencing_(NGS)"
- id: 60043
  author: prm36
  author_email: paul.mcadam@sky.com
  author_url: ''
  date: '2013-07-19 10:46:45 +0100'
  date_gmt: '2013-07-19 10:46:45 +0100'
  content: A more time consuming route, but the way I learned was through an MRes
    in computational bio.  Got a solid grounding in many fields of bioinformatics,
    hands on research experience, and a publication out of it.  If you've got the
    time and plan to use a lot of bioinformatics in your future research, then it's
    well worth the investment.
- id: 60057
  author: Bioinformatics is not something you are taught, it&#8217;s a way of life
    | opiniomics
  author_email: ''
  author_url: http://biomickwatson.wordpress.com/2013/08/06/bioinformatics-is-not-something-you-are-taught-its-a-way-of-life/
  date: '2013-08-06 08:05:32 +0100'
  date_gmt: '2013-08-06 08:05:32 +0100'
  content: '[...] his subsequent blog post, which details some of the [...]'
- id: 60100
  author: So you want to be a computational biologist? | opiniomics
  author_email: ''
  author_url: http://biomickwatson.wordpress.com/2013/11/08/so-you-want-to-be-a-computational-biologist/
  date: '2013-11-08 17:46:05 +0000'
  date_gmt: '2013-11-08 17:46:05 +0000'
  content: '[...] Nick Loman and I were approached by Nature Biotech to write a commentary
    based on our blog posts, here and here respectively, and if you&#8217;re reading
    this then the commentary is out [...]'
- id: 60177
  author: Things to learn to do bioinformatics. | bench to bioinformatics
  author_email: ''
  author_url: http://benchtobioinformatics.wordpress.com/2014/10/18/things-to-learn-to-do-bioinformatics/
  date: '2014-10-18 02:52:53 +0100'
  date_gmt: '2014-10-18 02:52:53 +0100'
  content: '[&#8230;] of which are a very good sources for technical learning/examples.
     Also, Nick Loman&#8217;s blog had a post dealing with learning bioinformatics.
     These and many others are great sources of [&#8230;]'
---
<p>I <a href="https://twitter.com/pathogenomenick/status/357421902895132672">asked a question on Twitter</a> yesterday:</p>
<p><em>What is the correct way to handle a request to “help me learn bioinformatics” from a non-computer-literate person?</em></p>
<p>This is a frequent request I encounter, and although I have various stock answers, I was curious to find out what you guys would say. Further, I wanted a resource with jumping off links, which I hope this blog post can serve as.</p>
<p>And, as with many times before, I was blown away by the use of Twitter for the diversity and quality of opinion and information generated from such a seemingly innocuous question. I know I am lucky in having &gt;3,500 eager followers hanging on my every question, but still ;)</p>
<p>Luis Pedro Coelho initially took issue with the question and said it is important to define what a user really wants, as "learning bioinformatics" "is not a goal per se". Is the user driven by intellectual curiosity, do they want to learn to code, or simply develop enough skills to "analyse my RNA-Seq data?". Russell Neches takes the extreme view which is that bioinformatics "is a fiction. Biologists just use computers for certain things". Manoj Semanta likes to stratify bioinformatics into <a href="http://www.homolog.us/blogs/blog/2011/07/22/a-beginners-guide-to-bioinformatics-part-i/">five</a> <a href="http://www.homolog.us/blogs/blog/2011/07/22/a-beginners-guide-to-bioinformatics-part-i/">layers</a> of increasing complexity, with using web interfaces and running command-line programmes being the easiest and developing new algorithms being the hardest..</p>
<p>Some suggested learning R or Python and pointed to helpful online tutorials, such as <a href="http://learnpythonthehardway.org/">http://learnpythonthehardway.org/</a> [Phil Ashton]. Another useful resource was the excellent Software Carpentry website which is aimed specifically at scientists wishing to learn best practices, for example the use of version control and Makefiles for reproducible research (<a href="http://software-carpentry.org">software-carpentry.org</a>) [suggested by Rob Davey, Deanna Church]. Software Carpentry plan to run more bootcamps for complete novices in the coming year, so keep an eye on their website.</p>
<p>Casey Bergman suggested using <a href="http://www.galaxyproject.org/">Galaxy</a>, a web-based bioinformatics engine particularly heavily used for NGS/genomics analysis (www.galaxy-project.org). Although Rob Davey qualified this advice pointing out that "using a workflow UI and being a bioinformatician may not be the same thing. Chris Cole agrees, "Galaxy != bioinformatics. It's a great and powerful system, tho."</p>
<p>The <a href="http://biomickwatson.wordpress.com/">always opinionated</a> Mick Watson suggested more of a 'sink or swim' approach, specifically to go away and install <a href="http://www.ubuntu.com/">Ubuntu</a> on a PC or laptop, because "because to "learn bioinformatics" you need commitment and time and effort". Mick also pointed to some online resources hosted at ARK Genomics which expanded on this idea of bioinformatics being tough, but worth investing time in learning (<a href="http://www.ark-genomics.org/events-online-training/eu-training-course">http://www.ark-genomics.org/events-online-training/eu-training-course</a>).</p>
<p>The theme of "learning by doing" is probably the one that I suggest most to people, also suggested by Aylwyn Scally. I tell people that learning programming through reading a book and doing simple exercises is demotivating if you don't have a problem in mind. So pick a problem that you think can be solved with scripting or bioinformatics tools, perhaps a biological question, and attempt to do it "by all means necessary". Being driven by a goal will help you keep motivated. Mario Caccamo says "Learning by doing is not ideal but that's the reality".</p>
<p>Some suggested attempting some simple tasks. One suggestion was to take data from a laboratory evolution paper (e.g. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21940899">http://www.ncbi.nlm.nih.gov/pubmed/21940899</a>) and try to reproduce it, in this case by detecting a small set of mutations [@contaminatedscience]. Another was to use the intriguing ROSALIND platform  to attempt bioinformatics problem solving (<a href="http://rosalind.info/problems/locations/">http://rosalind.info/problems/locations/</a>) [Robert Lanfae, Adam Kiezun].</p>
<p>Bastien Chevreux suggested the popular "Dummies Guide …" series including Bioinformatics for Dummies (<a href="http://www.dummies.com/store/product/Bioinformatics-For-Dummies-2nd-Edition.productCd-0470089857.html">http://www.dummies.com/store/product/Bioinformatics-For-Dummies-2nd-Edition.productCd-0470089857.html</a>) although C. Titus Brown  takes issue with the name of these books suggesting that "the culture of "I'm too stupid" inhibits learning". Good point. They look rubbish on your bookshelf too.</p>
<p>Pete @drosophilic suggested enrolling in local training courses, and a list is kept maintained by Stephen Turner at <a href="http://stephenturner.us/p/edu">http://stephenturner.us/p/edu</a>. I also note the newly launched iAnn Events platform (<a href="http://iann.pro/iannviewer">http://iann.pro/iannviewer</a>).</p>
<p>Another useful resource is BioStars, see the thread <a href="http://www.biostars.org/p/75925/">Advice for newcomers to the bioioinformatics field</a> [Pierre Lindenbaum]</p>
<p>C. Titus Brown has a workshop with online course materials for <a href="http://ged.msu.edu/angus/tutorials-2013/">next-generation sequence analysis</a>.</p>
<p>Alan McNally suggested it is possible for a newbie to learn bioinformatics successfully, citing himself as a case study: "I was the requester 4 years ago. Was told to switch to linux and start reading user guides. Haven't looked back".</p>
<p>And finally, Aylwyn Scally remarks that "the first thing I tell them is close MS Excel".</p>
<p>Thanks to all those who took part in the discussion!</p>
<p>Do you have anything to add?</p>
<p>Update: 31st July 2013 - added some links to the Homolog.us blog.</p>
