---
layout: post
status: publish
published: true
title: Beatles and Bioinformatics! 27th November 2013
author:
  display_name: Nick Loman
  login: nick
  email: n.j.loman@bham.ac.uk
  url: http://xbase.bham.ac.uk
author_login: nick
author_email: n.j.loman@bham.ac.uk
author_url: http://xbase.bham.ac.uk
wordpress_id: 1878
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=1878
date: '2013-10-21 08:05:37 +0100'
date_gmt: '2013-10-21 08:05:37 +0100'
categories:
- Genomics
tags:
- baltis and bioinformatics
- beatles and bioinformatics
comments: []
---
<p>So far we have held three meetings in the <a href="http://pathogenomics.bham.ac.uk/blog/tag/balti-and-bioinformatics/">Baltis and Bioinformatics</a> series, which has had some great talks and resulted in some great connections being made. The meetings are based around the linked topics of genome sequencing and bioinformatics, with the idea to take leaders in the bioinformatics field, and put them together with students, post-docs, 'service' bioinformaticians and others at the coal-face of data analysis. The meetings should stimulate discussion, collaboration opportunities and learning. We traditionally finish off with a bioinformatics clinic, where you can ask the hardest (or stupidest) questions you can think of. There is a heavy emphasis on method at this meeting - 'how did you do a particular analysis?', 'What software is best to try for this particular problem?', 'What issues did you encounter along the way?', rather than glossy re-hashing of already published work.</p>
<p>For the fourth meeting in this series, I am delighted and in fact bowled-over that the Center for Genomic Research at the University of Liverpool have decided not only to host but to spend a bit of money on the meeting. Not only that, they have stumped up a bit of cash to invite some international speakers, and they have subsidised lunch and the venue fee so that we can keep this meeting free. They are great people, and I should also thank the BBSRC and NERC for their generosity in sponsoring the CGR workshops. Many thanks for Neil Hall and Christiane Hertz-Fowler for their kind support of this meeting. This does also mean we need plenty of attendees, so please spread this announcement far and wide so we can make this the largest meeting yet.</p>
<p>[caption id="attachment_1885" align="alignright" width="300"]<a href="http://pathogenomics.bham.ac.uk/blog/wp-content/uploads/beetles.jpeg"><img class="size-medium wp-image-1885" title="beetles" src="http://pathogenomics.bham.ac.uk/blog/wp-content/uploads/beetles-300x225.jpeg" alt="" width="300" height="225" /></a> BEETLES[/caption]</p>
<p>[caption id="attachment_1886" align="alignright" width="300"]<a href="http://pathogenomics.bham.ac.uk/blog/wp-content/uploads/beatles.jpg"><img class="size-medium wp-image-1886" title="beatles" src="http://pathogenomics.bham.ac.uk/blog/wp-content/uploads/beatles-300x225.jpg" alt="" width="300" height="225" /></a> BEATLES[/caption]</p>
<p>Finally, the loose theme for the meeting is 'metagenomics', although this meeting will also be of interest to those doing 'just genomics'. And of course, being in Liverpool we just had to have a BEATLES (not BEETLES) theme.</p>
<p>A 16S &amp; metagenomics workshop will be held on the 28th and 29th of November, further details on this to be announced shortly.</p>
<p>It is free to attend, but <a href="https://docs.google.com/forms/d/1GFTKDaNequMz7P-lowRCMjyNtUinzQqaK8NaWLvkxQQ/viewform">please register through this link</a>! And whilst you are waiting for the meeting, please check the highly entertaining <a href="https://twitter.com/search?q=%23BeatlesandBioinformatics&amp;src=typd">#BeatlesAndBioinformatics</a> hashtag.</p>
<p>Due to high demand, registration for this meeting will close on <strong>Thursday, 31st October 2013 at 5pm (GMT)</strong>.</p>
<p>AGENDA</p>
<p>Venue: The Chapel, The Foresight Centre, University of Liverpool, Brownlow Street, Liverpool, L69 3GL. http://www.foresightcentre.co.uk</p>
<p>12.00 - Registration &amp; Lunch</p>
<p>13.00 - KEYNOTE: Sebastien Boisvert (<a href="http://twitter.com/sebhtml">@sebhtml</a>), Université Laval, Québec, Canada - "Ray and Ray Cloud Browser for Metagenomics"</p>
<p>13.50 - Chris Stewart (<a href="https://twitter.com/CJStewart7">@CJStewart7</a>), University of Northumbria at Newcastle - "Development of the Gut Microbiome in Preterm Infants at Risk of Necrotising Enterocolitis and Sepsis"</p>
<p>14.10 - Chris Quince, University of Glasgow - "CONCOCT: Clustering cONtigs on COverage and ComposiTion"</p>
<p>14.30 - Susannah Salter (<a href="https://twitter.com/Zannah_Du">@Zannah_Du</a>), Wellcome Trust Sanger Institute - "What's lurking in your kits?"</p>
<p>14.50 - Refreshment break</p>
<p>15.10 - KEYNOTE: Prof. Daniel Huson - Center for Bioinformatics, University of Tübingen - "Identifying Organisms from a Stream of DNA Sequences"</p>
<p>16.00 - Sujai Kumar (<a href="https://twitter.com/SujaiK">@SujaiK</a>), University of Oxford - "Blobology: exploring raw (meta)genome data for contaminants, symbionts and parasites using taxon-annotated GC-coverage plots"</p>
<p>16.20 - Mike Cox (<a href="https://twitter.com/MikeyJ">@MikeyJ</a>), Imperial College - "Copy number correction in 16S analysis"</p>
<p>16.40 - Rebecca Gladstone (<a href="https://twitter.com/RAGladstone">@RAGladstone</a>), University of Southampton / Wellcome Trust Sanger Institute - "Managing hundreds and thousands of bacterial genome sequences"</p>
<p>17.00 - Chris Hayward, Amazon Web Services - "Getting started with genomics in the cloud"</p>
<p>17.30 - The Magical Mystery Tour (dinner , followed by a visit to Matthew Street and the Cavern Club)</p>
