---
layout: post
status: publish
published: true
title: 'Course Advert: NERC Workshop on Population and Metagenomics Analysis'
author:
  display_name: Nick Loman
  login: nick
  email: n.j.loman@bham.ac.uk
  url: http://xbase.bham.ac.uk
author_login: nick
author_email: n.j.loman@bham.ac.uk
author_url: http://xbase.bham.ac.uk
wordpress_id: 1958
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=1958
date: '2013-12-02 16:54:31 +0000'
date_gmt: '2013-12-02 16:54:31 +0000'
categories:
- Bioinformatics
tags: []
comments: []
---
<p>I help out with a fair few workshops on genomic analysis, but most are limited by being restricted to just a day or two, hardly enough time to cover more than the basics. So the <strong>10 day long</strong><strong> NERC Population and Metagenomics Analysis course</strong>, organised by the fantastic Konrad Paszkiewicz from the University of Exeter is a quite amazing opportunity. The teaching will be from some of the best people in the field (and me!) and of the highest quality. Incredibly, if you are a UK-funded researcher the costs of the course are <strong>entirely waived</strong>, with accommodation and meals also included! Spaces are limited but I would definitely urge you to sign-up. Full details below,  and please <a href="http://www.eventbrite.co.uk/e/nerc-workshop-on-population-and-metagenomics-analysis-tickets-8628888237">head over to the course workshop page to register</a>. Hurry!</p>
<p><span lang="EN-US" style="font-size: large;"><strong>Workshop Overview</strong></span></p>
<p><span style="font-size: small;"><span lang="EN-US">A ten-day workshop taking place between 25 February - 6<strong> </strong>March 2014 </span>providing detailed hands-on training for population and meta-genomics analysis for researchers with little or no background in mathematics or computing. </span></p>
<p><span style="font-size: small;">Venue: <a href="http://www.dartingtonhall.com">Dartington Hall</a>, Totnes, Devon (nearest train station - Totnes)<br />
</span></p>
<p><span style="font-size: small;">Times: 25 February - 6th March 2014. </span></p>
<p><span style="font-size: small;">Arrival evening of Tuesday 25 February 2014. Departure morning of 6th March 2014. The course itself will take place 9am-12pm, 2pm-5pm and on some evenings 7pm-10pm everyday 26 February-5th March. Students are expected to attend the entire course.<br />
</span></p>
<p><span style="font-size: small;">Contact: <a href="mailto:%20research-events@exeter.ac.uk">research-events@exeter.ac.uk</a></span></p>
<p><span style="font-size: large;"><strong>Registration</strong></span></p>
<p><span style="font-size: small;">The course itself is free of charged and is funded by a Professional Postgraduate Development Award from NERC. </span></p>
<p><span style="font-size: small;">A total of 30 funded places are available which cover the costs of accommodation and food, but not the cost of transportation to/from the venue. </span></p>
<p><span style="font-size: small;">An additional 10 places are available for participants from industry. The cost of accommodation and meals will need to be covered by the participants.<br />
</span></p>
<p><span style="font-size: small;">You should register your interest by 31 December 2013. Participants will be informed by 10th January 2014 as to whether they have been selected. Please note that preference will be given to researchers funded by NERC.</span></p>
<p><span style="font-size: large;"><strong>Accomodation and Transport:<br />
</strong></span></p>
<p><span style="font-size: small;"><strong>For UK-based academic researchers:</strong> </span></p>
<p><span style="font-size: small;">The course is free of charge for up to 30 academic researchers working at recognised UK HEIs and research institutes. Accommodation at Dartington Hall is included and includes breakfast, lunch and dinner. Transportation to/from Dartington hall is NOT included.</span></p>
<p><span style="font-size: small;"><strong>For all other participants: </strong><br />
</span></p>
<p>Whilst course fees will be waived, the cost of accommodation and meals will NOT be included. If selected, you will need to book accommodation with Dartington Hall separately. A special rate of £102 per night plus VAT has been negotiated (this includes breakfast, lunch and dinner).</p>
<p><strong style="font-size: medium;">Requirements:</strong></p>
<p><strong style="font-size: medium;"><br />
</strong>Selected participants <strong>must bring</strong> their own laptops to the course. This needs to be a modern laptop (Windows, Mac OSX or Linux) with a full-size screen and wireless (please do not bring netbooks etc). Ethernet sockets may not be available during the course so please plan accordingly.</p>
<p><strong style="font-size: medium;"> </strong></p>
<p><strong style="font-size: medium;">Draft Programme</strong></p>
<p><span style="font-size: 10pt;">A</span><span style="font-size: 10pt;">n outline of the short course is given below with lead instructors in brackets. Please note that this is subject to change.<br />
</span></p>
<p><strong>Tuesday 25 February 2014:</strong></p>
<p>Arrival at Dartington Hall</p>
<p>Evening welcome buffet</p>
<p><strong>Wednesday 26 February 2014:</strong></p>
<p>Breakfast</p>
<p>Morning session 9am-12pm:</p>
<p>Introduction to the course</p>
<p>Hands on-workshop: Introduction to Amazon EC2 cloud (Konrad Paszkiewicz)</p>
<p><span style="font-size: 10pt;"> Lunch<br />
</span></p>
<p><span style="font-size: 10pt;"> Afternoon session 2pm-5pm:</span></p>
<p><span style="font-size: 10pt;"> Hands-on workshop: Introduction to Linux (Konrad Paszkiewicz and Julian Catchen)</span></p>
<p>Dinner</p>
<p>Evening session 7pm-10pm:</p>
<p><span style="font-size: 10pt;"> Hands-on workshop:</span> Introduction to Linux <span style="font-size: 10pt;">(Konrad Paszkiewicz and Julian Catchen)</span></p>
<p><span style="font-size: 10pt;"><strong>Thursday 27 February 2014:</strong></span></p>
<p><span style="font-size: 10pt;"> Breakfast</span></p>
<p>Morning session 9am-12pm:</p>
<p><span style="font-size: 10pt;"> Hands-on workshop</span><span style="line-height: 1.6em;">: Introduction to Linux </span><span style="font-size: 10pt;">(Konrad Paszkiewicz and Julian Catchen)</span></p>
<p><span style="font-size: 10pt; line-height: 1.6em;"> Lunch </span></p>
<p><span style="font-size: 10pt; line-height: 1.6em;"> Afternoon session 2pm-5pm:</span></p>
<p><span style="font-size: 10pt; line-height: 1.6em;"> Lecture: Introduction to genomics and bioinformatics (David Studholme)</span></p>
<p>Dinner</p>
<p><span style="line-height: 1.6em;"> Evening session 7pm-10pm:</span></p>
<p><span style="font-size: 10pt;"> <span style="font-size: 10pt;"> Hands-on workshop:</span> Short read genomics <span style="font-size: 10pt;">(Konrad Paszkiewicz and David Studholme)</span><br />
</span></p>
<p><span style="font-size: 10pt;"><strong>Friday 28 February 2014:</strong></span></p>
<p><span style="font-size: 10pt; line-height: 1.6em;"> Breakfast</span></p>
<p><span style="line-height: 1.6em;"> Morning session 9am-12pm:</span></p>
<p><span style="line-height: 1.6em;"> Lecture: Introduction to RAD-seq </span><span style="font-size: 10pt;">(William Cresko and Julian Catchen)</span></p>
<p><span style="font-size: 10pt;"> Lunch </span></p>
<p><span style="font-size: 10pt;"> Afternoon session 2pm-5pm:</span></p>
<p><span style="font-size: 10pt;"> <span style="font-size: 10pt;"> Hands-on workshop</span>: RAD-seq <span style="font-size: 10pt;">(William Cresko and Julian Catchen)</span></span></p>
<p><span style="line-height: 1.6em;">Dinner</span></p>
<p><span style="line-height: 1.6em;"> Evening session 7pm-10pm:</span></p>
<p><span style="font-size: 10pt;"><span style="font-size: 10pt;"> <span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"> Hands-on workshop</span>: RAD-seq <span style="font-size: 10pt;">(William Cresko and Julian Catchen)</span></span></span></span><br />
</span></p>
<p><span style="font-size: 10pt;"><strong>Saturday 1 March 2014:</strong></span></p>
<p><span style="font-size: 10pt;"> Breakfast</span></p>
<p>Morning session 9am-12pm:</p>
<p><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;">Hands-on workshop</span>: RAD-seq <span style="font-size: 10pt;">(William Cresko and Julian Catchen)</span></span></span></p>
<p><span style="font-size: 10pt;"> Lunch<br />
</span></p>
<p><span style="font-size: 10pt;"> Afternoon session 2pm-5pm:</span></p>
<p><span style="font-size: 10pt;"> Participant presentations</span></p>
<p><span style="line-height: 1.6em;">Dinner</span></p>
<p>Evening session 7pm-10pm:</p>
<p>&nbsp;</p>
<p><span style="font-size: 10pt;"><strong>Sunday 2 March 2014:</strong></span></p>
<p><span style="font-size: 10pt;"> Free day with organised activities<br />
</span></p>
<p><span style="font-size: 10pt;"><strong>Monday 3 March 2014:</strong></span></p>
<p><span style="font-size: 10pt; line-height: 1.6em;"> Breakfast</span></p>
<p><span style="line-height: 1.6em;"> Morning session 9am-12pm:</span></p>
<p><span style="line-height: 1.6em;"> Lecture: </span><span style="font-size: 10pt; line-height: 1.6em;">Marker-based metagenomics introduction </span><span style="font-size: 10pt;">(Jose Clemente)</span></p>
<p><span style="font-size: 10pt;"> Lecture: </span><span style="font-size: 10pt;">Statistical challenges in metagenomics analysis </span><span style="font-size: 10pt;">(Chris Quince)</span></p>
<div><span style="font-size: 10pt;"> Lunch </span></div>
<p><span style="font-size: 10pt;"> Afternoon session 2pm-5pm:</span></p>
<p><span style="font-size: 10pt;"> <span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;">Hands-on workshop</span>: Introduction to QIIME <span style="font-size: 10pt;">(Jose Clemente, Daniel McDonald)</span></span></span></span></span></span></span></span></span></p>
<p>Dinner</p>
<p><span style="line-height: 1.6em;"> Evening session 7pm-10pm:</span></p>
<p><span style="font-size: 10pt; line-height: 1.6em;"> <span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;">Hands-on workshop</span>: Introduction to QIIME <span style="font-size: 10pt;">(Jose Clemente, Daniel McDonald)</span></span></span></span></span></span></span></span></span></p>
<div><span style="font-size: 10pt; line-height: 1.6em;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"> </span></span></span></span></span></span></span></span></span></div>
<p><span style="font-size: 10pt;"><strong>Tuesday 4 March 2014:</strong></span></p>
<p><span style="font-size: 10pt;"> Breakfast</span></p>
<p>Morning session 9am-12pm:</p>
<p><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"> <span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;">Lecture: Whole genome metagenomics introduction <span style="font-size: 10pt;">(Nick Loman)</span></span></span></span></span></span></span><br />
</span></p>
<div><span style="font-size: 10pt;"> Lunch </span></div>
<p><span style="font-size: 10pt;"> Afternoon session 2pm-5pm:</span></p>
<p><span style="font-size: 10pt;"> <span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"> <span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"> <span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"> Hands-on workshop</span>: Whole genome metagenomics <span style="font-size: 10pt;">(Nick Loman, Chris Quince)</span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></p>
<p>Dinner</p>
<p>Evening session 7pm-10pm:</p>
<p><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"> <span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"> <span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"> <span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"> Hands-on workshop</span>: Whole genome metagenomics <span style="font-size: 10pt;">(Nick Loman, Chris Quince)</span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span><span style="font-size: 10pt;"><br />
</span></span></span></span></span></span></span></span></p>
<p><strong style="font-size: 10pt; line-height: 1.6em;">Wednesday 5 March 2014:</strong></p>
<p><span style="font-size: 10pt;"> Breakfast</span></p>
<p><span style="line-height: 1.6em;"> Morning session 9am-12pm:</span></p>
<p><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"> <span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"> Hands-on workshop</span>: Free session</span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span> (bring your own data)</span></span></span></span></span></span></span></span></p>
<p><span style="font-size: 10pt;"> Lunch </span></p>
<p><span style="font-size: 10pt;"> Afternoon session 2pm-5pm:</span></p>
<p><span style="font-size: 10pt;"> <span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"> <span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"> <span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;">Hands-on workshop</span>: Free session</span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span> (bring your own data)</span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></p>
<p><span style="line-height: 1.6em;">Dinner</span></p>
<p><span style="line-height: 1.6em;"> Evening session 7pm-10pm:</span></p>
<p><span lang="EN-US" style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"><span style="font-size: 10pt;"> End of workshop party</span></span></span><br />
</span></p>
<p><span lang="EN-US" style="font-size: 10pt;"><span style="font-size: 10pt;"><strong>Thursday 6 March 2014:</strong></span></span></p>
<p>Breakfast</p>
<p>Departure</p>
<p><span style="font-size: large;"><strong>Full event information</strong></span></p>
<p><span style="font-size: 10pt;"><span style="font-size: small;">Population and metagenomics analysis </span></span><span style="font-size: 10pt;">are fields which have developed rapidly over the recent years and have opened up new methodologies to researchers in ecology, systematics, evolutionary development and ecotoxicology. However, the software which has been developed to analyse these types of data are typically non-graphical and complex to master for researchers in biological sciences who have not been specifically trained in bioinformatics. In this short course the Amazon EC2 cloud will be used for training using laptops.</span></p>
<p><span style="font-size: 10pt;">One of the most significant recent developments in population genomics is Restriction-site Associated DNA sequencing (RAD-seq)</span><sup>1</sup><span style="font-size: 10pt;">. This technique uses high-throughput sequencing to simultaneously sequence and genotype organisms at tens of thousands of loci. The number of markers generated makes analysis much more sensitive than traditional microsatellite-based approaches, enabling resolution between very closely related individuals who belong to the same microsatellite type. It also requires comparatively less development and optimization time since the number of markers is proportional to the number of fragments digested. It can be applied even in the absence of a reference genome and can assist with genome assembly as well as provide functional information</span><sup>1,2</sup><span style="font-size: 10pt;">.</span></p>
<p><span lang="EN-US" style="font-size: 10.0pt;">Metagenomics involves the study of communities of microbial organisms in particular environments. The combination of uncultured-based techniques and high-throughput sequencing technology has made possible a comprehensive characterization of whole communities for a fraction of the cost. This enables studies of particular environmental niches over time or changing conditions at different resolution levels<sup>3,8</sup>. </span></p>
<p><span style="font-size: 10pt;">We have arranged for leading population genomic and metagenomic experts in the US and the UK to serve as instructors for this short-course. The US instructors below have either developed the molecular methods, the theory behind the analysis, or have actively developed the relevant software to perform the analysis.</span></p>
<p><span style="font-size: medium;"><strong> </strong></span></p>
<p><span style="font-size: medium;"><strong>Workshop instructors</strong></span></p>
<ol>
<li><span lang="EN-US" style="font-size: 10.0pt;">Professor William Cresko is a Principle Investigator and Director of the Institute of Ecology and Evolution at the University of Oregon. He is a pioneer of the RAD-seq technique, and has used the approach extensively to perform genetic mapping of stickleback fish phenotypic variation<sup>1</sup> as well as the evolutionary genomics of pipefishes and seahorses (</span><span lang="EN-US"><a href="http://creskolab.uoregon.edu/"><span style="font-size: 10.0pt; color: windowtext; text-decoration: none;">http://creskolab.uoregon.edu/</span></a></span><span lang="EN-US" style="font-size: 10.0pt;">).<br />
</span></li>
<li><span lang="EN-US" style="font-size: 10.0pt;">Dr Julian Catchen is a Postdoctoral Research Fellow at the University of Oregon Institute of Ecology and Evolution. He is the author of Stacks – the most popular software package designed to process and analyse RAD-seq data.<br />
</span></li>
<li><span lang="EN-US" style="font-size: 10.0pt;">Prof. Jose Clemente is based at the Icahn School of Medicine at Mount Sinai, New York. He is a contributing author to the QIIME (Quantitative Insights Into Microbial Ecology)<sup>4</sup> software package which is one of the most popular software tools for performing metagenomic analysis. His lab at Mount Sinai is particularly focused on characterizing the mechanisms of action of the microbiome in IBD.<br />
</span></li>
<li><span lang="EN-US" style="font-size: 10.0pt;">Dr Nick Loman is an MRC Special Training Research Fellow currently working at the University of Birmingham. His research program focuses on the genomic and metagenomic analysis of microbial sequence data in a clinical context.<br />
</span></li>
<li><span lang="EN-US" style="font-size: 10.0pt;">Daniel McDonald is a graduate student in the Interdisciplinary Quantitative Biology program in the BioFrontiers Institute at the University of Colorado, and a part of Prof. Rob Knight's lab, a recognized leader in microbiome research. Daniel is a contributing author of QIIME and a core software developer on the project.<br />
</span></li>
<li><span lang="EN-US" style="font-size: 10.0pt;">Dr David Studholme is a Senior Lecturer in Bioinformatics at the University of Exeter. His research interests encompass applications of genomics, transcriptomics and metagenomics to plant-pathogen interactions. His recent projects have focussed on tree-pathogens <em>Chalara fraxinea</em> and <em>Phytophthora ramorum</em> as well as bacterial pathogens of banana, enset, tomato and other crops.<br />
</span></li>
<li><span lang="EN-US" style="font-size: 10.0pt;">Dr Konrad Paszkiewicz is the Director of the Wellcome Trust Biomedical Informatics hub. He is responsible for the provision of training for PhD students and researchers as well as bioinformatics facilities and capabilities within the University of Exeter.<br />
</span></li>
<li><span lang="EN-US" style="font-size: 10.0pt;">Prof. Peter Kille is the director of Bio-Initiatives at the University of Cardiff. His primary research expertise lies in the application of molecular techniques such as proteomics and genomics to eco-toxicology. His research interests encompass the effect on biological systems of the release of heavy metals into the environment.<br />
</span></li>
<li><span lang="EN-US" style="font-size: 10.0pt;">Dr Christopher Quince is a Reader at the University of Glasgow. He leads the Computational Microbial genomics group which focuses on the development of novel algorithms to aid the analysis of microbial community structures. The group also develop engineering systems using microbial communities including microbial fuel cells and filtration systems. He is the author of PyroNoise and AmpliconNoise which are integral to the analysis of many high-throughput metagenomic datasets.</span></li>
</ol>
<p><span lang="EN-US" style="font-size: 10.0pt;">All of the instructors have extensive experience teaching in a short-course/workshop environment. These include </span><span lang="EN-US"><a href="https://academy.nescent.org/wiki/Main_Page"><span style="font-size: 10.0pt;">the National Evolutionary Synthesis Centre Workshops</span></a></span><span lang="EN-US" style="font-size: 10.0pt;"> (NESCent) in Next Generation Sequencing 2011 and 2012 () held at Duke University, North Carolina, USA and the </span><span lang="EN-US"><a href="http://evomics.org/workshops/workshop-on-genomics/2013-workshop-on-genomics-cesky-krumlov/"><span style="font-size: 10.0pt;">Evomics</span></a></span><span lang="EN-US" style="font-size: 10.0pt;"> workshop held every January in </span><span lang="EN-US" style="font-size: 10pt;">Č</span><span lang="EN-US" style="font-size: 10.0pt;">esk</span><strong><span lang="EN-US" style="font-size: 10pt;">ý</span></strong><span lang="EN-US" style="font-size: 10.0pt;"> Krumlov, Czech Republic. The QIIME group hold regular workshops in the US and worldwide. Much of the teaching material for the proposed short course has already been produced, delivered and tested to a large number of audiences. Amazon cloud images for each section of the short course have already been produced by the instructors and have been extensively tested in previous workshops.</span></p>
<p><span style="font-size: 10pt;">The Amazon and Linux training will be based on a modified version of the ‘Unix &amp; Perl for Biologists’</span><sup>7</sup><span style="font-size: 10pt;"> course adapted for use on the cloud with an extensive EC2 tutorial. Pre-existing in-house workshop materials will be used to teach the basics of remapping, assembly and variant calling. The Stacks software suite will be used in conjunction with R to teach RAD-seq analysis. We will use QIIME</span><sup>4</sup><span style="font-size: 10pt;">, MEGAN</span><sup>5</sup><span style="font-size: 10pt;"> and MetaPhlAn</span><sup>6</sup><span style="font-size: 10pt;"> software packages to teach various aspects of marker based and shotgun metagenomics.</span></p>
<p><span style="font-size: 10pt;"><span style="font-size: medium;"><strong> </strong></span></span></p>
<p><span lang="EN-US" style="font-size: 10.0pt;"><strong>References:</strong><br />
</span></p>
<ol>
<li><span style="font-size: 6.5pt; font-family: 'inherit','serif'; color: #222222;">Hohenlohe PA, Bassham S, Etter PD, Stiffler, N. Cresko, W.A. <em>Population Genomics of Parallel Adaptation in Threespine Stickleback using Sequenced RAD Tags</em>. PLoS Genetics 2010;6.</span></li>
<li><span style="font-size: 6.5pt; font-family: 'inherit','serif'; color: #222222;">J. Catchen, P. Hohenlohe, S. Bassham, A. Amores, and W. Cresko. <em>Stacks: an analysis tool set for population genomics</em>. Molecular Ecology. 2013.</span></li>
<li><span style="font-size: 6.5pt; font-family: 'inherit','serif'; color: #222222;">J Rousk, E Bååth, PC Brookes, CL Lauber, C Lozupone, JG Caporaso, R Knight </span><span lang="EN-US"><a href="http://scholar.google.co.uk/citations?view_op=view_citation&amp;hl=en&amp;user=8wv9sLkAAAAJ&amp;citation_for_view=8wv9sLkAAAAJ:9yKSN-GCB0IC"><em><span lang="EN-GB" style="font-size: 6.5pt; font-family: 'inherit','serif'; color: #222222; text-decoration: none;">Soil bacterial and fungal communities across a pH gradient in an arable soil</span></em></a></span><em></em><span style="font-size: 6.5pt; font-family: 'inherit','serif'; color: #222222;">The ISME journal 4 (10), 1340-1351</span></li>
<li><span style="font-size: 6.5pt; font-family: 'inherit','serif'; color: #222222;">J. Caporaso et al. <em>QIIME allows analysis of high-throughput community sequencing data. </em>Nature Methods 7, 335-336 (2010)</span></li>
<li><span style="font-size: 6.5pt; font-family: 'inherit','serif'; color: #222222;">Huson, DH, Mitra, S, Weber, N, Ruscheweyh, H, and Schuster, SC (2011). </span><span lang="EN-US"><a href="http://ab.inf.uni-tuebingen.de/publications/articlereference.2011-07-08.1919071627"><em><span lang="EN-GB" style="font-size: 6.5pt; font-family: 'inherit','serif'; color: #222222; text-decoration: none;">Integrative analysis of environmental sequences using MEGAN4.</span></em></a></span><span style="font-size: 6.5pt; font-family: 'inherit','serif'; color: #222222;"> Genome Research, 21:1552-1560</span></li>
<li><span style="font-size: 6.5pt; font-family: 'inherit','serif'; color: #222222;">Segata, N et al. <em>Metagenomic microbial community profiling using unique clade-specific marker genes.</em> Nature Methods 9, 811-814 (2012).</span></li>
<li><span lang="EN-US"><span lang="EN-GB" style="font-size: 6.5pt; font-family: 'inherit','serif'; color: #222222; text-decoration: none;"><span style="font-size: 6.5pt; font-family: 'inherit','serif'; color: #222222;">Bates, S. et al. <em>Global biogeography of highly diverse protisan communities in soil</em>. </span><span style="font-size: 4.0pt; font-family: 'inherit','serif'; color: #222222;">ISME J 7: 652-659; Dec. 2012</span><span style="font-size: 5.5pt; font-family: 'inherit','serif'; color: #222222;">. </span><a href="http://korflab.ucdavis.edu/Unix_and_Perl/">http://korflab.ucdavis.edu/Unix_and_Perl/</a></span></span></li>
</ol>
