---
layout: post
status: publish
published: true
title: A chat with Oxford Nanopore's Clive Brown at AGBT 2013
author:
  display_name: Nick Loman
  login: nick
  email: n.j.loman@bham.ac.uk
  url: http://xbase.bham.ac.uk
author_login: nick
author_email: n.j.loman@bham.ac.uk
author_url: http://xbase.bham.ac.uk
wordpress_id: 1559
wordpress_url: http://pathogenomics.bham.ac.uk/blog/?p=1559
date: '2013-03-06 10:20:11 +0000'
date_gmt: '2013-03-06 10:20:11 +0000'
categories:
- Uncategorized
tags: []
comments:
- id: 59982
  author: paultmorrison
  author_email: paul_morrison@dfci.harvard.edu
  author_url: ''
  date: '2013-03-07 19:02:19 +0000'
  date_gmt: '2013-03-07 19:02:19 +0000'
  content: "An enjoyable read and I now have renewed optimism for Oxford Nanopore.
    It was a surprise after the meeting to be sorting the attendees by affiliation
    and finding Clive and two others from ONT. I wish they had gotten some of this
    message out there (and I had spent less time freezing in the dark and more time
    at the bar at the pool.)\r\n\r\nAnd my message to ONT and Clive:  I still love
    the MinION, I have a proposal for 12 sexy cancer related projects that would be
    perfect for it so a shoebox full of these USB dongles would find a loving home
    if I was on the \"Clive's Favorites\" list.\r\nXXXOOO -Paul"
- id: 59983
  author: Brian K
  author_email: support@labspaces.net
  author_url: http://www.labspaces.net
  date: '2013-03-08 05:15:02 +0000'
  date_gmt: '2013-03-08 05:15:02 +0000'
  content: How'd I miss this post??  Great interview, Nick!  I'm with you, I want
    to believe but it's so hard when so many promises were made at AGBT12.  Clive
    can't be mad at other people.  ONT made unbelievable claims that they didn't deliver
    on when they said they'd deliver.  Don't say a product is going to be out at year's
    end and then disappear into the ether for a year and a half.  I think a level
    of disappointment can only be expected.  Clive's response should be, "Yep, we
    made a mistake.  Apologies. Hopefully if we put in some long nights, we'll be
    able to blow your minds in the near future."  Clive, I want my mind blown, but
    please spare us your "do not distribute" list.  That's a bit school yard.
- id: 59984
  author: GenomicsIO
  author_email: skurscheid+wordpress@gmail.com
  author_url: ''
  date: '2013-03-08 09:30:44 +0000'
  date_gmt: '2013-03-08 09:30:44 +0000'
  content: "Quote: “We didn’t even know that long reads were so important to people
    until after that AGBT presentation.”\r\n\r\nAs excited as I am about the technology,
    I have to say that a statement like this is rather surprising. Shouldn't a company
    like Oxford Nanopore _know_ their market and targeted audience? To me it seems
    a bit naive to make a major announcement at a conference like AGBT without presenting
    any actual data, and then be surprised about the \"backlash\" - genome scientist
    breath data, after all..."
- id: 59985
  author: Links 3/10/13 | Mike the Mad Biologist
  author_email: ''
  author_url: http://mikethemadbiologist.com/2013/03/10/links-31013/
  date: '2013-03-10 20:53:11 +0000'
  date_gmt: '2013-03-10 20:53:11 +0000'
  content: '[...] A chat with Oxford Nanopore’s Clive Brown at AGBT 2013 (I find this
    very remarkable; taken at face value, they really don&#8217;t understand their
    customers) Asking Questions and the “Finkbeiner Test” Oxford Nanopore chip announced!
    (Almost&#8230;) The sequester and HIV/AIDS Take me as I am, and my paper as it
    is? [...]'
- id: 59987
  author: Tania Fernandez
  author_email: tfernandez@b-c.com
  author_url: ''
  date: '2013-03-24 08:30:44 +0000'
  date_gmt: '2013-03-24 08:30:44 +0000'
  content: "Good interview Nick. \r\n\r\nGo easy guys on ONT. Lets not hold them to
    timelines and just hope that the wait has been worth it. Its easier to do catch
    up than be a first mover. ONT has blazed the nanopore sequencing  field for a
    while. \r\n\r\nEvery bit of data that is released will help further this field
    and God knows that we need that ! There are  a bunch of companies in stealth mode
    that are banking on some of the fundamentals of ONT technology. So every bit of
    new information  helps. \r\n\r\nWill keep the faith for now till data proves otherwise
    !"
- id: 59993
  author: igorcarron
  author_email: igorcarron@gmail.com
  author_url: http://nuit-blanche.blogspot.com
  date: '2013-04-05 17:31:38 +0100'
  date_gmt: '2013-04-05 17:31:38 +0100'
  content: "Nick,\r\n\r\nThank you for the interviews. I deal with machine learning
    and compressive sensing and wrote two blog entries on the subject of the Nanopore
    technology as a result of your blog entries and interviews:\r\n\r\nOf Well Logging
    and Nanopore Sequencing\r\nhttp://nuit-blanche.blogspot.com/2013/03/of-well-logging-and-nanopores.html\r\n\r\nStructural
    Information in Nanopore Sequencing ?\r\nhttp://nuit-blanche.blogspot.com/2013/04/structural-information-in-nanopore.html\r\n\r\nIgor."
- id: 59995
  author: singlemoleculesceptic
  author_email: stephen.sowerby@otago.ac.nz
  author_url: ''
  date: '2013-04-15 09:59:36 +0100'
  date_gmt: '2013-04-15 09:59:36 +0100'
  content: Sadly, I suspect it's all going to custard.  Any hint of success would
    be accompanied by some data, but nothing.  All they have is blockade data from
    a mixture of free nucleotides showing appropriate blockade levels.  That is not
    sequencing.    The exonuclease approach requires free cleaved nucleotides to pass
    through the nanopore, in sequence.  If they can get exonuclease onto the nanopore
    edge, and bind DNA, and start processing, so that all cleaved nucleotides enter
    the pore, each sequential nucleotide must be distinguished from any other competing
    blockade event unless they will introduce insertion errors.  Whole molecule sequencing
    is no closer than it was when DNA blockades were first published in 1996. CEO's
    evasive comments in the Bloomberg article scream volumes.  Fingers crossed, but
    don't hold your breath.
- id: 60015
  author: Oxford Nanopore's Exciting Sequencing News | Clark Executive Search
  author_email: ''
  author_url: http://clarksearch.com/oxford-nanopores-exciting-sequencing-news/
  date: '2013-05-13 21:25:36 +0100'
  date_gmt: '2013-05-13 21:25:36 +0100'
  content: '[...] 05/13/13 When will Nanopore come out? [...]'
- id: 60053
  author: Upcoming meetings of interest
  author_email: ''
  author_url: http://pathogenomics.bham.ac.uk/blog/2013/08/upcoming-meetings-of-interest/
  date: '2013-08-05 07:38:25 +0100'
  date_gmt: '2013-08-05 07:38:25 +0100'
  content: '[...] CTO of Oxford Nanopore will be giving the keynote, and hopefully
    updating us on the status of their eagerly anticipated single-molecule sequencer.
    There are sessions on bioinformatics, metagenomics, clinical application [...]'
- id: 60089
  author: Oxford Nanopore Raises $64 Million
  author_email: ''
  author_url: http://nextgenseek.com/2013/10/oxford-nanopore-raises-64-million/
  date: '2013-10-09 19:52:25 +0100'
  date_gmt: '2013-10-09 19:52:25 +0100'
  content: '[...] get to see more from Oxford Nanopore this year? There is a good
    chance for that. Earlier this year, Nick Loman blogged from AGBT 2013 that Oxford
    Nanopore got delayed due to unexpected problems and expect to start [...]'
- id: 60103
  author: PaulHDear
  author_email: phd@mrc-lmb.cam.ac.uk
  author_url: ''
  date: '2013-11-15 21:30:28 +0000'
  date_gmt: '2013-11-15 21:30:28 +0000'
  content: They didn't know that people would be interested in long reads?  That's
    incredible.  It's a bit like claiming you've developed a teleporter and then basing
    your business model on using it to deliver pizzas.
- id: 60158
  author: And So Begins Oxford Nanopore MinION Sequencing
  author_email: ''
  author_url: http://nextgenseek.com/2014/04/and-so-begins-oxford-nanopore-minion-sequencing/
  date: '2014-04-11 12:20:53 +0100'
  date_gmt: '2014-04-11 12:20:53 +0100'
  content: '[&#8230;] February 2013: Oxford Nanopore Skips AGBT 2014, but privatley
    talks about why it is behind the schedule.  [&#8230;]'
- id: 60168
  author: Behold. Oxford Nanopore Reads Are Here
  author_email: ''
  author_url: http://nextgenseek.com/2014/06/behold-oxford-nanopore-reads-are-here/
  date: '2014-06-11 12:15:01 +0100'
  date_gmt: '2014-06-11 12:15:01 +0100'
  content: '[&#8230;] February 2013: Oxford Nanopore Skips AGBT 2014, but privatley
    talks about why it is behind the schedule. [&#8230;]'
---
<p>Don't judge me, reader, because I'd skipped a session at AGBT to go and have a swim in the sea. A man can only spend so much time in dimly-lit, low-ceilinged hotel conference rooms, popping low-sugar sweets, before the will to live ebbs away.</p>
<p>On returning to the conference, passing the bar I spotted a distinctive bald head. Wait. I recognise that guy. Was it him?</p>
<p>I reversed and took another peek. Yes, it was Clive Brown, deep in a meeting. "Hello Clive!". He looked up, slightly grumpy to be interrupted mid-flow. "I'm Nick Loman, wasn't expecting to see you here!". Oh hello Nick. I catch a glimpse of a prototype MinIon on the table. "Hah, yeah, I've buried three of those on the beach. Tell everyone!"</p>
<p>We meet in the bar the next day. Clive talks at machine-gun pace, whilst fiddling with a prototype MinIon which is on the desk, repeatedly taking it apart and reassembling it, like a soldier checking his gun before battle. It feels weighty, substantial, larger than the version announced previously. It's got a mini-connector for USB3. "Feels a bit too expensive for a disposable sequencer, needs to be more plastic-y", I venture. Clive agrees.</p>
<p><a href="http://pathogenomics.bham.ac.uk/blog/wp-content/uploads/IMG_16522.jpg"><img class="alignright size-large wp-image-1562" title="IMG_1652" src="http://pathogenomics.bham.ac.uk/blog/wp-content/uploads/IMG_16522-1024x768.jpg" alt="" width="1024" height="768" /></a></p>
<p>Clive is angry. He feels he's been treated unreasonably by the community, and the press, since AGBT 2012's electrifying announcements. "I'm bloody sequencing single molecules directly on this little device here!". The implication is that no-one should be surprised it's taken longer than expected to be released. He is unapologetic.</p>
<p>Clive is angry. He's angry with the guy that marched to the nanopore suite at AGBT, banging and shouting through the door: "Where is your data? Show us the data!!".</p>
<p>Clive is angry with reporters who keep asking him the same questions: why they haven't released data, why they haven't fulfilled their promise of commercialisation by the end of 2012.</p>
<p>Clive has a list. A list of people that says he'll see to it won't get a MinIon when it comes out. I can't tell if he's joking.</p>
<p>So why didn't you release some data, Clive? He tells me that the raw signal data is commercially valuable, that someone in the business could take the traces and reverse engineer details of their customised nanopore this way. The idea that other parties could steal information to further their own nanopore projects is a recurring theme in our chat.</p>
<p>So why didn't the MinIon come out in 2012? Technically, he lists several setbacks. The custom sensor microchip (ASIC) wasn't performing as they wanted, necessitating a redesign from scratch. "That put us back about 5 months, but it was the right thing to do". There have also been problems stabilising the lipid bilayer, and so over days and weeks it degrades. He set his team a new accuracy target of 1%, a major improvement from the 4% error rate announced at AGBT.</p>
<p>I venture the idea that even if the MinIon is a year, two years late, if it's half as good as he says it is, all will be forgotten. Like waiting for the next version of Quake or Grand Theft Auto.</p>
<p>"It's not going to be that long, we're going to start announcing stuff this year, including data from our early-access programme."</p>
<p>Why don't you engage with the community better? I suggest that no tweets and no web updates isn't a good look for a company with so many eyes on. He says that they have to be careful about putting any information out there right now, in case it is used against them. He suggests that now Zoe McDougall, their communications director is back from maternity leave, they will improve their communication with the community.</p>
<p>Technical breakthroughs. They've found that error rates can be improved by having multiple nanopores on the chip with different properties, and then merging the data. Some nanopores are better at recognising certain nucleotide signatures than others, and so they can be complementary. This is a hint that consensus accuracy might ultimately be important, a la Pacific Biosciences. ** see footnote</p>
<p>He's keen on the idea of nanopore as a disruptive technology for proteomics, citing the <a href="http://www.nature.com/nbt/journal/vaop/ncurrent/full/nbt.2503.html">unfoldase</a> that should permit proteins to pass through the pore.</p>
<p>Clive is a man under pressure. I genuinely got the impression that the company were caught off-guard by all the attention and had no idea they would be under such scrutiny.</p>
<p>"We didn't even know that long reads were so important to people until after that AGBT presentation." He explains that he set his team a technical challenge to go from 20kb to 50kb to 100kb, simply because he likes pushing them further than they think they can go. His focus on getting error rate down to 1% results from similar pushing, sometimes to the chagrin of the commercial side of the operation.</p>
<p>Clive is guarded, and regularly checks himself, ensuring he doesn't say anything that would "get him in trouble".</p>
<p>"You know what, I hated doing that presentation at AGBT. I had to hide in my room for two days afterwards."</p>
<p>"I'm not Jonathan Rothberg".</p>
<p>What do I think? I find it hard to simply write-off nanopore as vapourware, as some seem happy to do. There is a great group of people in this company, and frankly it just wouldn't be cricket to promise so much without delivering. I will wait and see. I feel sure the conversation will have moved on by AGBT 2014.</p>
<p>"I want to believe" as they might say on the X-Files.</p>
<p>Plus, I don't want to end up on Clive's list.</p>
<p>&nbsp;</p>
<p>** Clive has written to clarify this point: <em>I didnt mean that as an alternative to raw read, but it came up repeatedly during the conference that a number of early access groups are trying to do major projects to "improve the reference" of their given organism. They are currently mixing a number of short reads from different technologies and without the long reads, they have difficulty assembling (a major use of PB data). I have noticed that with two pores (or more) we effectively have two orthogonal error modes, which means this kind of improved reference, with assembly, can be done economically on one platform – which should be a lot easier.</em></p>
