---
layout: post
title: "Why use Nextera over Nextera XT?"
description: ""
category:  
tags: []
---


My reply to an email I was sent about whether Nextera or Nextera XT a better choice for a whole-genome shotgun metagenomics project:

> The theoretical reason you might use Nextera over Nextera XT is that as the starting amount of DNA is higher, the library might have increased diversity and this might impact the sequencing.
>
> Nextera starts with 50ng whereas XT starts with 2ng. 
>
> However, if you actually do the maths this doesn't seem to be a real issue. A single copy of a haploid human genome (3Gb) is about 3.5pg. Therefore, you have about 1713 gigabases of information in 2ng. As we are only sequencing about 10 gigabases per sample, the sampling should still be essentially random. You can get a metric of this by looking at things like PCR duplication rates, and we haven't seen a significant problem with XT. 
>
> Although Illumina originally said XT was for 'small genomes' they are recommending it for human and cancer genomes now. I think it's more of a marketing thing than a science thing.
