---
layout: post
title: "BIOM25: Metagenomics practical"
description: ""
category: 
tags: []
---

## E. coli outbreak

Our paper describing the outbreak:

<http://www.nejm.org/doi/full/10.1056/NEJMoa1107643>

Our paper describing use of whole-genome shotgun metagenomics to diagnose the outbreak:

<http://jama.jamanetwork.com/article.aspx?articleid=1677374>

The data website:

<https://www.ebi.ac.uk/metagenomics/projects/ERP001956>

Q. Pick 10 samples at random. Look at the taxonomic distributions. What is the most dominant taxon at order level and species level for each sample? Does this seem normal?

Q. Are any toxins present? Which ones? What is the significance of this toxin and how might it cause disease?

Q. Generally, what genes are responsible for antibiotic resistance in E. coli? Can you find any of those genes in the dataset?

Q. Now compare your findings from this dataset with a healthy population from the MetaHIT paper:

<https://www.ebi.ac.uk/metagenomics/projects/ERP000108>

How do the German samples compare to the “healthy” population?

## Non-human environment

Now, choose a non-human environment to study, and to present to the group:

Soil:
<https://www.ebi.ac.uk/metagenomics/projects/doSearch?search=Search&biome=SOIL>

Water:
<https://www.ebi.ac.uk/metagenomics/projects/doSearch?search=Search&biome=MARINE>

Animal:
<https://www.ebi.ac.uk/metagenomics/projects/doSearch?search=Search&biome=NON_HUMAN_HOST>

Artificial:
<https://www.ebi.ac.uk/metagenomics/projects/doSearch?search=Search&biome=ENGINEERED>

Q. What did the study set out to find?

Q. How did they sample their environment? How many samples did they look at?

Q. How does this environment compare taxonomically with the human gut?

Q. How does this environment compare functionally with the human gut?


