---
layout: post
title: "BIOM25: Metagenomics practical"
description: ""
category: 
tags: []
---

Scratchpad for session:

<https://docs.google.com/document/d/1DY8efRRM2MO7N2DuFTKN-QEmwOm9JuZF_UZKAFme9RU/edit?usp=sharing>

## Normal human microbiome

The MetaHIT project studied healthy volunteers, as well as people with diabetes and inflammatory
bowel disease to characterise their microbiomes:

<https://www.ebi.ac.uk/metagenomics/projects/ERP000108>

Have a read of how the study was designed:

<http://www.nature.com/nature/journal/v464/n7285/full/nature08821.html>

Q: Take 10 samples at random and look at their taxonomic distribution. Tabulate the top 3 phyla present and their relative abundances.

## E. coli outbreak

Our paper describing the outbreak:

<http://www.nejm.org/doi/full/10.1056/NEJMoa1107643>

Our paper describing use of whole-genome shotgun metagenomics to diagnose the outbreak:

<http://jama.jamanetwork.com/article.aspx?articleid=1677374>

The data website:

<https://www.ebi.ac.uk/metagenomics/projects/ERP001956>

Q. Pick some samples at random. For example sample, look at the taxonomic distributions.

Q. Do any samples look abnormal, compared to the 'normal' microbiome?

Q. Are any toxins present? Which ones? What is the significance of this toxin and how might it cause disease?

## Antibiotic resistance

Here is a report from one of the outbreak metagenomes using a different analysis pipeline:

<https://app.onecodex.com/analysis/public/94b2f5f421a841be>

Q. What antibiotic resistance genes are present? (Hint: check the AMR report)

Q. What antibiotics might the outbreak strain be resistant to?

Q. How could we prove that the outbreak strain is resistant to these antibiotics?

## Non-human environment

Now, choose a non-human environment to study, and to present to the group:

Soil:
<https://www.ebi.ac.uk/metagenomics/projects/doSearch?search=Search&biome=SOIL>

Water:
<https://www.ebi.ac.uk/metagenomics/projects/doSearch?search=Search&biome=MARINE>

Animal:
<https://www.ebi.ac.uk/metagenomics/projects/doSearch?search=Search&biome=NON_HUMAN_HOST>

Artificial:
<https://www.ebi.ac.uk/metagenomics/projects/doSearch?search=Search&biome=ENGINEERED>

Q. What did the study set out to find?

Q. How did they sample their environment? How many samples did they look at?

Q. How does this environment compare taxonomically with the human gut? Is it more or less diverse? Are the set of organisms present similar or different?

Q. How does this environment compare functionally with the human gut? Can you explain these findings in the context of the environment?



